---
title: Comparative analyses of two primate species diverged by more than 60 million years show different rates but similar distribution of genome-wide UV repair events
keywords:
- DNA repair
- Nucleotide excision repair
- UV damage
- Gray Mouse Lemur
- Fibroblasts
lang: en-US
date-meta: '2021-05-01'
author-meta:
- Umit Akkose
- Veysel Ogulcan Kaya
- Laura Lindsey-Boltz
- Zeynep Karagoz
- Adam D. Brown
- Peter A. Larsen
- Anne D. Yoder
- Aziz Sancar
- Ogun Adebali
header-includes: |-
  <!--
  Manubot generated metadata rendered from header-includes-template.html.
  Suggest improvements at https://github.com/manubot/manubot/blob/main/manubot/process/header-includes-template.html
  -->
  <meta name="dc.format" content="text/html" />
  <meta name="dc.title" content="Comparative analyses of two primate species diverged by more than 60 million years show different rates but similar distribution of genome-wide UV repair events" />
  <meta name="citation_title" content="Comparative analyses of two primate species diverged by more than 60 million years show different rates but similar distribution of genome-wide UV repair events" />
  <meta property="og:title" content="Comparative analyses of two primate species diverged by more than 60 million years show different rates but similar distribution of genome-wide UV repair events" />
  <meta property="twitter:title" content="Comparative analyses of two primate species diverged by more than 60 million years show different rates but similar distribution of genome-wide UV repair events" />
  <meta name="dc.date" content="2021-05-01" />
  <meta name="citation_publication_date" content="2021-05-01" />
  <meta name="dc.language" content="en-US" />
  <meta name="citation_language" content="en-US" />
  <meta name="dc.relation.ispartof" content="Manubot" />
  <meta name="dc.publisher" content="Manubot" />
  <meta name="citation_journal_title" content="Manubot" />
  <meta name="citation_technical_report_institution" content="Manubot" />
  <meta name="citation_author" content="Umit Akkose" />
  <meta name="citation_author_institution" content="Molecular Biology, Genetics &amp; Bioengineering Program, Faculty of Engineering and Natural Sciences, Sabanci University, Istanbul, 34956, Turkey" />
  <meta name="citation_author_orcid" content="0000-0002-9731-3702" />
  <meta name="citation_author" content="Veysel Ogulcan Kaya" />
  <meta name="citation_author_institution" content="Molecular Biology, Genetics &amp; Bioengineering Program, Faculty of Engineering and Natural Sciences, Sabanci University, Istanbul, 34956, Turkey" />
  <meta name="citation_author_orcid" content="0000-0002-3447-0735" />
  <meta name="citation_author" content="Laura Lindsey-Boltz" />
  <meta name="citation_author_institution" content="Department of Biochemistry and Biophysics, School of Medicine, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina, 27599, USA" />
  <meta name="citation_author_orcid" content="0000-0001-6493-8194" />
  <meta name="citation_author" content="Zeynep Karagoz" />
  <meta name="citation_author_institution" content="Molecular Biology, Genetics &amp; Bioengineering Program, Faculty of Engineering and Natural Sciences, Sabanci University, Istanbul, 34956, Turkey" />
  <meta name="citation_author_orcid" content="0000-0002-7627-9218" />
  <meta name="citation_author" content="Adam D. Brown" />
  <meta name="citation_author_institution" content="Department of Pharmacology and Cancer Biology, Duke University, Durham, North Carolina, 27708, USA" />
  <meta name="citation_author_orcid" content="0000-0002-9122-4960" />
  <meta name="citation_author" content="Peter A. Larsen" />
  <meta name="citation_author_institution" content="Department of Biology, Duke University, Durham, North Carolina, 27708, USA" />
  <meta name="citation_author_orcid" content="0000-0002-3634-3625" />
  <meta name="citation_author" content="Anne D. Yoder" />
  <meta name="citation_author_institution" content="Department of Biology, Duke University, Durham, North Carolina, 27708, USA" />
  <meta name="citation_author_orcid" content="0000-0002-1781-9552" />
  <meta name="citation_author" content="Aziz Sancar" />
  <meta name="citation_author_institution" content="Department of Biochemistry and Biophysics, School of Medicine, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina, 27599, USA" />
  <meta name="citation_author_orcid" content="0000-0001-6469-4900" />
  <meta name="citation_author" content="Ogun Adebali" />
  <meta name="citation_author_institution" content="Molecular Biology, Genetics &amp; Bioengineering Program, Faculty of Engineering and Natural Sciences, Sabanci University, Istanbul, 34956, Turkey" />
  <meta name="citation_author_orcid" content="0000-0001-9213-4070" />
  <meta name="twitter:creator" content="@compGenomics" />
  <link rel="canonical" href="https://CompGenomeLab.github.io/lemur-manu/" />
  <meta property="og:url" content="https://CompGenomeLab.github.io/lemur-manu/" />
  <meta property="twitter:url" content="https://CompGenomeLab.github.io/lemur-manu/" />
  <meta name="citation_fulltext_html_url" content="https://CompGenomeLab.github.io/lemur-manu/" />
  <meta name="citation_pdf_url" content="https://CompGenomeLab.github.io/lemur-manu/manuscript.pdf" />
  <link rel="alternate" type="application/pdf" href="https://CompGenomeLab.github.io/lemur-manu/manuscript.pdf" />
  <link rel="alternate" type="text/html" href="https://CompGenomeLab.github.io/lemur-manu/v/44f5bc48b6869d288b107da645614c3991d2bcb5/" />
  <meta name="manubot_html_url_versioned" content="https://CompGenomeLab.github.io/lemur-manu/v/44f5bc48b6869d288b107da645614c3991d2bcb5/" />
  <meta name="manubot_pdf_url_versioned" content="https://CompGenomeLab.github.io/lemur-manu/v/44f5bc48b6869d288b107da645614c3991d2bcb5/manuscript.pdf" />
  <meta property="og:type" content="article" />
  <meta property="twitter:card" content="summary_large_image" />
  <link rel="icon" type="image/png" sizes="192x192" href="https://manubot.org/favicon-192x192.png" />
  <link rel="mask-icon" href="https://manubot.org/safari-pinned-tab.svg" color="#ad1457" />
  <meta name="theme-color" content="#ad1457" />
  <!-- end Manubot generated metadata -->
bibliography:
- content/manual-references.json
manubot-output-bibliography: output/references.json
manubot-output-citekeys: output/citations.tsv
manubot-requests-cache-path: ci/cache/requests-cache
manubot-clear-requests-cache: false
...






<!-- [![ORCID icon](images/orcid.svg){height="11px" width="11px"}](https://orcid.org/0000-0002-9731-3702) -->
Umit Akkose^1,†^,
<!-- [![ORCID icon](images/orcid.svg){height="11px" width="11px"}](https://orcid.org/0000-0002-3447-0735) -->
Veysel Ogulcan Kaya^1,†^,
<!-- [![ORCID icon](images/orcid.svg){height="11px" width="11px"}](https://orcid.org/0000-0001-6493-8194) -->
Laura Lindsey-Boltz^2,†^,
<!-- [![ORCID icon](images/orcid.svg){height="11px" width="11px"}](https://orcid.org/0000-0002-7627-9218) -->
Zeynep Karagoz^1^,
<!-- [![ORCID icon](images/orcid.svg){height="11px" width="11px"}](https://orcid.org/0000-0002-9122-4960) -->
Adam D. Brown^3^,
<!-- [![ORCID icon](images/orcid.svg){height="11px" width="11px"}](https://orcid.org/0000-0002-3634-3625) -->
Peter A. Larsen^4^,
<!-- [![ORCID icon](images/orcid.svg){height="11px" width="11px"}](https://orcid.org/0000-0002-1781-9552) -->
Anne D. Yoder^4^,
<!-- [![ORCID icon](images/orcid.svg){height="11px" width="11px"}](https://orcid.org/0000-0001-6469-4900) -->
Aziz Sancar^2^,
<!-- [![ORCID icon](images/orcid.svg){height="11px" width="11px"}](https://orcid.org/0000-0001-9213-4070) -->
Ogun Adebali^1^,*

<small>

1. Molecular Biology, Genetics & Bioengineering Program, Faculty of Engineering and Natural Sciences, Sabanci University, Istanbul, 34956, Turkey
2. Department of Biochemistry and Biophysics, School of Medicine, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina, 27599, USA
3. Department of Pharmacology and Cancer Biology, Duke University, Durham, North Carolina, 27708, USA
4. Department of Biology, Duke University, Durham, North Carolina, 27708, USA

<!-- ^#^ Current Address: Department of Veterinary and Biomedical Sciences, University of Minnesota, St. Paul, Minnesota, 55112, USA <br> -->
<!-- <br> -->
^†^ Joint Authors

\* To whom correspondence should be addressed. Tel: +90 216 568 7043; Fax: +90 216 483 9550; E-mail: oadebali@sabanciuniv.edu

Present Address: [Peter A. Larsen] Department of Veterinary and Biomedical Sciences, University of Minnesota, St. Paul, Minnesota, 55112, USA

</small>

<small>
<!-- **Running title**: Nucleotide excision repair profiles in primates -->
</small>

<small>
<!-- **Keywords:** Nucleotide excision repair, UV damage, XR-seq, Mouse Lemur, (6-4)PP, CPD, Primate -->
</small>

<!-- ## Abstract {.page_break_before} -->
## Abstract


<!-- 150 words Nat Comm -->

<!-- Nucleotide excision repair is the primary DNA repair mechanism that removes bulky DNA adducts such as UV-induced pyrimidine dimers. 
Although human repair profiles were extensively studied, how repair maps vary between primates is yet to be investigated.
Here, we characterized the genome-wide UV-induced damage repair in gray mouse lemur, *Microcebus murinus*, in comparison to human.
Mouse lemurs are strictly nocturnal, are the world's smallest living primates, and last shared a common ancestor with humans at least 60 million years ago. 
We derived fibroblast cell lines from mouse lemur, exposed them to UV irradiation and captured excised oligomers through the XR-seq protocol. 
Overall, UV sensitivity, repair efficiency, and transcription-coupled repair levels differed between the two primates.
Despite this, genome-wide repair profiles of the homologous regions are highly correlated. 
We suggest mouse lemurs and humans, and possibly primates in general, share a homologous repair mechanism as well as genomic variance distribution, albeit with their variable repair efficiency.
Our results also emphasize the deep homologies of individual tissue types across the eukaryotic phylogeny. -->

<!-- 200 words NAR -->

<!-- Nucleotide excision repair is the primary DNA repair mechanism that removes bulky DNA adducts such as UV-induced pyrimidine dimers. 
Correspondingly, genome-wide mapping of nucleotide excision repair with eXcision Repair sequencing (XR-seq), provides comprehensive profiling of DNA damage repair. 
Although human repair profiles were extensively studied, how repair maps vary between primates is yet to be investigated.
Here, we characterized the genome-wide UV-induced damage repair in gray mouse lemur, *Microcebus murinus*, in comparison to human.
Mouse lemurs are strictly nocturnal, are the world's smallest living primates, and last shared a common ancestor with humans at least 60 million years ago. 
We derived fibroblast cell lines from mouse lemur, exposed them to UV irradiation and captured excised oligomers through the XR-seq protocol. 
Mouse lemur repair profiles were analyzed in comparison to the human fibroblasts.
Overall, UV sensitivity, repair efficiency, and transcription-coupled repair levels differed between the two primates.
Despite this, genome-wide repair profiles of the homologous regions are highly correlated. 
We suggest mouse lemurs and humans, and possibly primates in general, share a homologous repair mechanism as well as genomic variance distribution, albeit with their variable repair efficiency.
Our results also emphasize the deep homologies of individual tissue types across the eukaryotic phylogeny. -->

<!-- Key points
* Nucleotide excision repair mechanisms in mouse lemur and human are homologous.
* UV repair rates between human and lemur differ.
* Genome-wide UV repair profiles are similar between the two primates.
 -->


<!-- Nucleotide excision repair is the primary DNA repair mechanism that removes bulky DNA adducts such as UV-induced pyrimidine dimers. 
Correspondingly, genome-wide mapping of nucleotide excision repair with eXcision Repair sequencing (XR-seq), provides comprehensive profiling of DNA damage repair. 
A number of XR-seq experiments at a variety of conditions for different damage types revealed heterogenous repair in the human genome.
Although human repair profiles were extensively studied, how repair maps vary between primates is yet to be investigated.
Here, we characterized the genome-wide UV-induced damage repair in gray mouse lemur,*Microcebus murinus*, in comparison to human.
Mouse lemurs are strictly nocturnal, are the world's smallest living primates, and last shared a common ancestor with humans at least 60 million years ago. 
We derived fibroblast cell lines from mouse lemur, exposed them to UV irradiation. 
The following repair events were captured genome-wide through the XR-seq protocol. 
Mouse lemur repair profiles were analyzed in comparison to the equivalent human fibroblast datasets.
We found that overall UV sensitivity, repair efficiency, and transcription-coupled repair levels differ between the two primates.
Despite this, comparative analysis of human and mouse lemur fibroblasts revealed that genome-wide repair profiles of the homologous regions are highly correlated. 
This correlation is stronger for the highly expressed genes.
With the inclusion of an additional XR-seq sample derived from another human cell line in the analysis, we found that fibroblasts of the two primates repair UV-induced DNA lesions in a more similar pattern than two distinct human cell lines do.
Our results suggest that mouse lemurs and humans, and possibly primates in general, share a homologous repair mechanism as well as genomic variance distribution, albeit with their variable repair efficiency.
This result also emphasizes the deep homologies of individual tissue types across the eukaryotic phylogeny. -->


<!-- 250 words Genome Research-->

### Background

Nucleotide excision repair is the primary DNA repair mechanism that removes bulky DNA adducts such as UV-induced pyrimidine dimers. 
Correspondingly, genome-wide mapping of nucleotide excision repair with eXcision Repair sequencing (XR-seq), provides comprehensive profiling of DNA damage repair. 
A number of XR-seq experiments at a variety of conditions for different damage types revealed heterogenous repair in the human genome.
Although human repair profiles were extensively studied, how repair maps vary between primates is yet to be investigated.
Here, we characterized the genome-wide UV-induced damage repair in gray mouse lemur,*Microcebus murinus*, in comparison to human.

### Results

We derived fibroblast cell lines from mouse lemur, exposed them to UV irradiation, and analyzed the repair events genome-wide using the XR-seq protocol.
Mouse lemur repair profiles were analyzed in comparison to the equivalent human fibroblast datasets.
We found that overall UV sensitivity, repair efficiency, and transcription-coupled repair levels differ between the two primates.
Despite this, comparative analysis of human and mouse lemur fibroblasts revealed that genome-wide repair profiles of the homologous regions are highly correlated, and this correlation is stronger for highly expressed genes.
With the inclusion of an additional XR-seq sample derived from another human cell line in the analysis, we found that fibroblasts of the two primates repair UV-induced DNA lesions in a more similar pattern than two distinct human cell lines do.

### Conclusion

Our results suggest that mouse lemurs and humans, and possibly primates in general, share a homologous repair mechanism as well as genomic variance distribution, albeit with their variable repair efficiency.
This result also emphasizes the deep homologies of individual tissue types across the eukaryotic phylogeny.

## Keywords

Nucleotide excision repair, UV damage, XR-seq, Mouse Lemur, (6-4)PP, CPD, Primate

<!-- ## Introduction  -->
## Background 

Nucleotide excision repair is an essential mechanism to remove bulky DNA adducts, including UV-induced DNA lesions [@doi:10.1074/jbc.R117.807453]. 
As in other repair systems, excision repair starts with damage recognition. 
Two subpathways based on damage recognition lead to two repair mechanisms: global repair (GR) and transcription-coupled repair (TCR). 
GR is active throughout the genome, whereas TCR is only active on the transcribed strands as it is initiated by damage recognition through stalled RNA polymerase II (RNAPII). 
To date, many techniques have been developed to detect DNA damage and repair [@doi:10.1002/em.22365].
The approach of some methodologies has been NGS-based, which allows answering genome-wide questions.
To reveal genome-wide excision repair dynamics, heterogeneity and associations, eXcsion Repair sequencing (XR-seq) was developed.
The XR-seq technique directly measures the repair events by capturing excised DNA oligomers containing the lesion.
It was found that TCR is more efficient, particularly for slowly-repaired DNA lesions. 
For instance, among UV photoproducts 6-4 pyrimidine-pyrimidone ([6-4]PP) and cyclobutane pyrimidine dimer (CPD), CPD is more prone to TCR as GR less efficiently repairs it. 
Although DNA lesions might preferentially form at certain local sites in the genome, the overall heterogeneity of repair is mainly due to uneven repair efficiency throughout the genome [@doi:10.1073/pnas.1706522114; @doi:10.1038/s41467-018-05064-0]. 
Genome-wide heterogenous repair distribution is mostly caused by transcription and chromatin structure [@doi:10.1101/gad.261271.115; @doi:10.1073/pnas.1614430113; @doi:10.1073/pnas.1603388113; @doi:10.1038/s41467-019-08290-2].

To date, genome-wide repair maps were generated for model organisms, including *Escherichia coli* [@doi:10.1073/pnas.1700230114; @doi:10.1074/jbc.C117.818807], *Saccharomyces cerevisiae* [@doi:10.1073/pnas.1801687115], *Drosophila melanogaster* [@doi:10.1074/jbc.AC119.011448], *Arabidopsis thaliana* [@doi:10.1038/s41467-018-03922-5], *Mus musculus* [@doi:10.1073/pnas.1804493115], and *Homo sapiens* [@doi:10.1101/gad.261271.115; @doi:10.1073/pnas.1706522114]. 
TCR presence was verified for each of these species. 
For eukaryotic genomes, the consistent finding was the efficient repair in open chromatin regions.
Heterochromatin regions were found to be repaired at later time points.
Human repair profiles were extensively studied with respect to damage formation and chromatin states.
Whether regions in the human genome that are efficiently repaired are organism-specific is yet to be investigated.

To study whether repair patterns are unique to the organism of interest, we aimed to compare human and a deeply diverged non-human primate. 
Gray mouse lemur (*Microcebus murinus*) stands out as a promising model organism candidate because of its small body size, short gestation time (2 months) and fast sexual maturation (6-8 months) [@doi:10.1002/ajp.22423;@doi:10.1534/genetics.116.199448]. 
A near chromosome level reference genome for the gray mouse lemur was recently sequenced and assembled [@doi:10.1186/s12915-017-0439-6]. 
With no surprise, it was shown that mouse lemur and human orthologs share ~91% identity. 
Although a robust genome assembly is available, we lack an in-depth understanding of this species' genomic features such as epigenetic maps, transcriptomes and methylomes.

Given that mouse lemurs and humans last shared a common ancestor at the base of the primate clade [@doi:10.1016/j.tig.2007.02.007], the same cell types from human and mouse lemur should behave similarly in response to DNA damage as a reflection of their deep homology.
With this motivation, we carried out a comparative study between these two primates to understand similarities and differences between their repair profiles.
We derived primary fibroblasts from mouse lemur and immortalized them.
We performed survival assays in response to UV stress, immunoslot blot assays and in vivo excision assays for both cell lines. 
From mouse lemur fibroblasts, we obtained transcriptomes, exposed cells to UV and performed XR-seq.
XR-seq captured the excised oligomers as repair products for two main UV-induced damage types: (6-4)PP and CPD.
We compared lemur and human fibroblast XR-seq datasets for their genomic repair distribution.


## Results

### Excised oligomer characteristics

The in vivo excision assay was used to analyze the excised oligomers containing UV-induced lesions (Fig {@fig:fig1}A). 
Excised oligomers were captured for the two distinct damage types with specific anti-damage antibodies.
The oligomer length distribution varies from 16 to 30nt. 
Two intense bands were observed, which indicate primary and degraded excised oligomers. 
The gel images show that the primary excised oligomer lengths vary between 23-25nt.
Through the time course, gel images reflect more intense secondary products, indicating higher levels of degraded excised oligomers at later time points.
For normal human fibroblasts 1 (NHF1) and mouse lemur fibroblasts, this trend is similar. 
The human and lemur experiments were not conducted together and thus, the band intensities differ. 
The spike-in 50-mer internal control was used for quantification of the excision products between experiments.

Nucleotide excision repair in human, *S. cerevisiae* and *E. coli* was shown to yield primary excised oligomers predominantly in lengths of 27, 24 and 13 nucleotides, respectively [@doi:10.1074/jbc.M113.482257; @doi:10.1073/pnas.89.8.3664; @doi:10.1073/pnas.1801687115; @pmid:6380755; @doi:10.1073/pnas.1700230114]. 
Mouse lemur XR-seq resulted in oligomer length distribution identical to humans (Fig {@fig:fig1}B). 
For both CPD and (6-4)PP, nucleotide excision repair machinery incised 26nt oligomer as the predominant product.
A small variance in oligomer lengths between the in vivo excision assay and XR-seq might be due to the exonuclease activity on the excised oligomers in the vivo excision assay.
Because of the TFIIH coimmunoprecipitation in the excision repair sequencing technique, XR-seq primary products are not yet degraded, possibly because of TFIIH protection of the excised oligomers.


### Identical CPD and (6-4)PP nucleotide frequency between human and lemur

Nucleotide frequencies of the excised oligomers between human and mouse lemur are identical for both CPD and (6-4)PP (Fig {@fig:fig1}C). 
Pyrimidine enrichment was revealed at 19th, 20th and 21st nucleotides for CPD and (6-4)PP for both organisms, which is in agreement with the incision site at 6 to 8nt fixed distance to the 3' end of the DNA lesion. 
The nucleotide contents for CPD and (6-4)PP are different, as they were previously reported and discussed [@doi:10.1101/gad.261271.115].


![ Figure 1
<!-- Characteristics of excised oligomers upon repair of UV-induced damages. 
*(A)* In vivo excision assay for (6-4)PP and CPD are shown. 
"M": markers (sizes 17 and 24) and "-": no UV.  
*(B)* XR-seq read length distribution of excised oligomers for CPD (top) and (6-4)PP (bottom). Human and mouse lemur are shown in blue and red, respectively. 
*(C)* Nucleotide frequency in the predominant oligomer (26nt) for each primate and damage type. Representative data from replicate 1. -->
](images/fig-1.png){
    <!-- #fig:fig1 .white -->
    } 



### UV repair efficiency of mouse lemur in comparison with human {.page_break_before}

To identify the overall repair efficiency upon UV irradiation, we performed a colony survival assay. 
We seeded 300 cells and grew them for 16 hours. 
Cells were exposed to UV-C at 1 J/m<sup>2</sup>sec for variable time intervals to reach indicated doses ranging from 2.5 to 10 J/m<sup>2</sup>.
We counted cells for each dose and plotted survival curves for lemur and human fibroblasts (Fig {@fig:fig2}A).
Based on the survival curves of the cell lines used in this study, we infer that mouse lemur fibroblasts are more sensitive to UV relative to human cells. 
<!-- This suggests a differential repair rate between the two species, which is possibly reflective of the diurnal versus nocturnal habits of humans and mouse lemurs, respectively. -->

We then performed immunoslot blot assays to measure the DNA repair rates (Fig {@fig:fig2}B).
Cells were exposed to UV-C at 10J/m<sup>2</sup>. 
To separate UV-induced DNA lesions, we used specific antibodies for CPD and (6-4)PP. 
We quantified the remaining damage signal for each damage type for both species. 
We observe a difference between rates of repair of CPD and 6-4 PP, as has been previously described, which is due to several reasons including differences in damage recognition and differences in global repair versus transcription coupled repair discussed further below.
Both (6-4)PP and CPD repair appeared to be more efficient in human cells compared to mouse lemur. 
Human (6-4)PP repair was completed in 2 hours whereas in lemurs, it took 4 hours to complete (6-4)PP repair. 
20% of CPDs were still unrepaired for human cells 32 hours after UV irradiation. 
Unrepaired CPD ratio was more than 40% for mouse lemur cells. 
Survival and immonoslot blot assays are in harmony and they suggest mouse lemur cells have relatively low repair rates compared to NHF1 cells.

![ Figure 2
<!-- UV sensitivity and repair rates in human and lemur fibroblasts. 
*(A)* Clonogenic survival assay (left) and the quantified results (right).
*(B)* Immunoslot blot repair assays to examine the repair kinetics of UV-induced DNA lesions for both organisms and each damage type (left).
Remaining damage levels at each time point were quantified and plotted (right).
All experiments were repeated three times, and graphed data are presented as (mean+/-SD). 
Tests (t-test) performed (H~0~:μ~human~=μ~lemur~) for each dosage (A) and time points (B).
\* P ≤ 0.05; ** P ≤ 0.01; *** P ≤ 0.001; rest P > 0.05. -->
](images/fig-2.png){
    <!-- #fig:fig2 width=66% .white -->
    }

### Prominent transcription-coupled repair in mouse lemur

Nucleotide excision repair comprises two sub-pathways, each of which distinctly acts on damage recognition. 
GR is the genome-wide mechanism that is active in any region in the genome, although its efficiency depends on the type of damage as well as chromatin factors. 
On the other hand, TCR depends on the RNAPII stalled at the lesion. 
Stalled RNAPII recruits transcription-coupled repair factor CSB to enhance repair in the transcribed strand of the genes. 
DNA lesions in non-transcribed strands are subject to global repair. 
A major rate-limiting step of nucleotide excision repair is damage recognition. 
Therefore, minor helix-distorting lesions such as CPDs are recognized more efficiently when they stall elongating RNAPII, and thus genes have asymmetrical repair between two strands; the transcribed strand (TS) is repaired more efficiently compared to the nontranscribed strand (NTS) [@doi:10.1038/nrm2549;@doi:10.1038/342095a0]. 

Global repair for (6-4)PP is much faster than for CPD [@doi:10.1093/nar/gkz035; @doi:10.1073/pnas.1603388113]. 
TCR removes a minor fraction of the (6-4)PPs, and thus strand asymmetry between TS and NTS (TS/NTS ratio) is much weaker for (6-4)PP compared to TS-favored strand asymmetry of CPD repair [@doi:10.1073/pnas.1603388113].
CPDs, on the other hand, are harder to be recognized by global repair, and therefore it takes a longer time to remove CPDs from the genome. 
For this reason, at early time points (such as 1 hour after UV irradiation), we observe a strong TCR effect, which is indicated by the asymmetrical repair between TS and NTS. 
Here, we compared the genic strand asymmetry between human and mouse lemur (Fig {@fig:fig3}). 
We retrieved all annotated protein-coding genes from both genomes. 
We removed the genes that are closer to each other with less than 20kb in order to remove the signal that might have a "canceling-out" effect. 
We used 5277 and 3366 annotated genes for human and mouse lemur, respectively (see methods). 
We performed a meta-analysis where we aligned all transcription start and transcription end sites and calculated the RPKM (reads per kilobase per million mapped reads) values.
Interestingly, mouse lemur fibroblasts exhibited stronger TCR profiles compared to human cell lines at 1 hour after CPD formation (TS/NTS Mann-Whitney U test p=4.3*10<sup>-21</sup>). 
The higher TCR profile of lemur fibroblasts might be caused by the lower global repair efficiency relative to human fibroblasts (see discussion).

![ Figure 3
<!-- Transcription-coupled repair in mouse lemur and human fibroblasts. 
Average profiles of CPD XR-seq (top) and (6-4)PP XR-seq (bottom) over 5277 and 3366 annotated genes are plotted for human (left) and mouse lemur (right), respectively. 
Transcription start sites (TSS) and transcription end sites (TES) were retrieved from GTF (Gene Transfer Format) files for the two genomes.
Transcribed (red) and nontranscribed (blue) strand repair are shown in the downstream and upstream of TSS and TES, respectively.
10 kb upstream and downstream of TSS and TES were divided into 100 bp windows. 
XR-seq reads aligned to each bin were normalized to RPKM (reads per kilobase per million mapped reads). 
The XR-seq RPKM values were normalized by the shuffled RPKM values derived by mapped reads aligned to random genomic sites. 
Only replicate 1 is shown. TS/NTS median values for TSS downstream for CPD are 1.76 and 2.31 for human and mouse lemur, respectively. -->
](images/fig-3.png){
    <!-- #fig:fig3 .white -->
    }


### Conserved repair rates between homologous regions of two primates

In order to compare and contrast the genome-wide repair profiles between human and mouse lemur, we retrieved the homologous genomic regions between the organisms (Fig {@fig:fig4}A). 
We aligned the two genomes and kept the regions that have at least 80% identity.
Due to the low coverage of the XR-seq samples we removed the aligned segments that are shorter than 400 bases to eliminate the random effect due to scarce repair events. 
Lineage-specific duplications (or deletions) result in paralogs that can be co-orthologs to one locus in the other genome. 
This situation is known as the "one-to-multiple" orthology relationship. 
As we cannot be sure which one of the multiple paralogs in one genome is the "true" ortholog of a region in the other one, we filtered out one-to-multiple kinds of homologous relationships and kept one-to-one homologs only.

1.0% and 1.3% of the genomes were taken into account with one-to-one homologs for human and mouse lemur, respectively.
4-5% and 2-3% of the XR-seq reads mapped to those regions for CPD and (6-4)PP, respectively.
The reason why the mapping rate of CPD is higher than (6-4)PP could be due to its slower repair rate, as slowly repaired CPDs are more prone to the effects of chromatin. 
Because we applied stringent criteria to retrieve the homologous regions with at least 80% identity, they are likely to be more open compared to other regions in the genome.
Therefore, more CPD repair events aligned to those regions relative to (6-4)PP reads.

We investigated the correlation between the repair profiles of the two species. 
Because excised oligomers have a certain nucleotide bias due to the damage site, we normalized the true repair signal (XR-seq) by the simulated XR-seq reads.
Simulated reads have the nucleotide content of the true XR-seq reads (Fig {@fig:S1}), however, they are randomly retrieved from the homologous regions. 
With this approach, we eliminated the potential bias of the nucleotide content that creates a pseudo-correlation.
Although there is a clear difference in the relative TCR rates between two species (Fig {@fig:fig3}), homologous regions of two genomes exhibited a strong repair rate correlation; R values for (6-4)PP and CPD are 0.80 and 0.72 (p=0), respectively (Fig {@fig:fig4}B).
Not only repair but also transcription levels between the two species are correlated (Fig {@fig:fig4}C).
The similar repair and transcription profiles between two cell lines are independent of their chromosomal locations. 
Chromosomal homologous region associations are dispersed, as previously suggested [@doi:10.1186/s12915-017-0439-6].  


![ Figure 4
<!-- Comparative analysis methodology and correlated repair rates between human and mouse lemur. 
A) Approach to identify orthologous regions between human and mouse lemur (see methods for details). 
B) Scatter plots showing the normalized repair levels between two organisms. 
The relative repair was calculated by normalizing the repair signal by the simulated XR-seq reads to eliminate the sequence context bias. 
C) Mapped orthologous regions with repair and transcription profiles mapped at left and right outer rings, respectively. 
The inner circle represents the human (left) and mouse lemur (right) chromosomes. 
The inner lines connecting orthologous regions are colored based human chromosome color scale. 
Repair and transcription values (outer rings) for genomic regions that have no apparent ortholog (based on the criteria in A) are not shown. -->
](images/fig-4.png){
    <!-- #fig:fig4 .white -->
    }

### Repair profile correlation is associated with gene expression

To test whether gene expression is a factor associated with the correlation of repair between these two highly-diverged species of primates, we examined the correlation strength and gene transcription levels. 
We prepared RNA-seq data sets for mouse lemur fibroblasts and obtained RNA-seq datasets for human fibroblasts from a publicly available database (see methods). 
The strength of the correlation between the two repair profiles was found to be correlated with transcription levels (Fig {@fig:fig5}).
Highly expressed regions had better repair correlation between the two species (Figs @fig:S2;@fig:S3;@fig:S4;@fig:S5).
The data did not show a correlation between the identity levels and repair correlation between the two species, which suggests that gene expression levels rather than the degree of homology might be an important factor.

![ Figure 5
<!-- Repair rate consistency in correlation with gene expression. 
The transcription levels of the orthologous regions were divided into quartiles separately for human and lemur. 
Out of the quartiles, 10 bootstrapped subsamples were retrieved, and for those regions, repair correlation was analyzed. 
The correlation coefficients (R values) are represented on the y-axis; quartiles are on the x-axis. -->
](images/fig-5.png){
    <!-- #fig:fig5 .white -->
    }

### The same cell type from two primates have more similar repair profiles relative to two different cell types from the same organism

We cannot dismiss the possibility that the correlation of repair profiles between two primates could simply be due to the heterogenous mappability of the genomes. 
Such an association makes sense only when we measure the similarity of repair patterns relative to another dataset. 
For this reason, we used another human cell type, GM12878 - B-Lymphocytes to ask whether the repair profile of human fibroblasts is closer to lemur fibroblasts or human lymphocytes.
We performed XR-seq for CPD as it yields relatively high genome-wide heterogeneity compared to (6-4)PP.
Although repair profiles between human GM12878 and NHF1 are also correlated (Fig {@fig:fig6}A), this correlation is not as strong as the one between human and mouse lemur fibroblasts.
Additionally, human and mouse lemur fibroblasts clustered together in the PCA plot (Fig {@fig:fig6}B). 
Within fibroblasts, biological replicates grouped together. 
More interestingly, (6-4)PP and CPD samples formed distinct groups, each of which has human and mouse lemur samples in the subclusters.
The two replicates of CPD XR-seq samples of human lymphocytes (GM12878) clustered as an outgroup.
When we applied PCA differentially to genes and intergenic regions, we observed similar clustering (Fig @fig:S6).
This result suggests that not only transcriptional but also epigenetic profiles between human and mouse lemur fibroblasts are similar to yield correlated TCR (genes, Fig {@fig:S6}A) and global repair (intergenic regions Fig {@fig:S6}B), respectively. 
On the other hand, even though derived from the same organism, completely different human cell lines have a considerable variation with respect to repair patterns.
It is interesting to note that although TCR profiles between two primates are quite different from each other, they share high similarity in repair profiles. 

![ Figure 6
<!-- Cell type-based consistency of repair profiles between two primates. 
A) Scatter plot of normalized repair levels between two human cell lines, GM12878 and NHF1, show correlation (R=0.65, p=0). 
B) Principle component analysis of 10 XR-seq samples.  -->
](images/fig-6.png){
    <!-- #fig:fig6 width=100% .white -->
    }


## Discussion

XR-seq methodology has become prominent in genome-wide profiling of nucleotide excision repair at single-nucleotide resolution. 
This technique has been applied to a variety of model organisms, from humans to bacteria. 
Although genome-level understanding of human repair behavior is relatively well established in recent years, the associations between closely related primates with respect to UV damage response and genome-wide repair distribution were entirely lacking. 
With this pilot study, we established the differences and similarities between mouse lemur and human using a pair of similarly derived fibroblast cell lines.
First, we observed repair efficiency differences between the two genomes of the two major types of DNA damage induced by UV. 
Like in humans, (6-4)PP was much more quickly completed compared to CPD in mouse lemur. 
Both (6-4)PP and CPD repair efficiencies were slower in mouse lemur fibroblasts relative to human fibroblasts. 
This observation is in agreement with the survival assays, which show that mouse lemur cell lines were more sensitive to UV, which contains both damage types we studied. 
Although this comparison is limited to only two species, and further analysis would be required to make a more definitive conclusion, we postulate that the differences could result from differing repair enzyme activity between the species because of differences in fur which protects from UV exposure or due to the differences in diurnal and nocturnal lifestyles of the two species.

Interestingly, we also observed that TCR was relatively more efficient in mouse lemurs compared to humans, particularly for CPD repair. 
TCR of (6-4)PP was not prominent, as was previously shown, and our immunoslot blot assays indicate that (6-4)PP repair is fast and therefore, repaired mostly by GR rather than TCR. 
We observed similar TS and NTS repair signal for (6-4)PP in genic regions, also suggesting that GR is the primary subpathway to repair (6-4)PP.
A striking fact is an inverse correlation between TCR, which is measured by the TS/NTS repair signal, and overall repair efficiencies.
Mouse lemurs with more efficient TCR fall behind in total repair compared to the human. 
A possible explanation for this observation is that global repair is the primary determinant of the repair efficiency. 
XR-seq yields the entire nucleotide excision repair events with no distinction between TCR and GR. 
For this reason, GR and TCR efficiencies derived through XR-seq are relative to each other.
Like the relativity in any RNA-seq, which cannot measure the total transcription levels, XR-seq also gives the relative distribution of the repair events. 
Therefore, measured "TCR efficiency" actually corresponds to "GR deficiency" in the XR-seq profile.
In other words, when the damage is not efficiently recognized by the GR pathway, TCR takes over the recognition and removes DNA lesions in TS.
With this study, previously observed high TCR levels in *Arabidopsis thaliana* [@doi:10.1038/s41467-018-03922-5] and mouse [@doi:10.1073/pnas.1804493115] now also suggest relatively poor GR efficiency in those organisms.

Human evolution has been under investigation since the time of Charles Darwin. 
Comparative analyses between humans and other animals have revealed both shared and human-specific features. 
In the era of molecular biology, it has been possible to identify molecular features as they are shared between humans and their relatives. 
Primates are an extremely diverse group comprised of species that range in body size from 30-80 grams (mouse lemurs) to 150kg (African apes). 
Mouse lemurs stand out as the world's smallest living primate. 
Given detailed knowledge of primate phylogeny and divergence times [@doi:10.1093/sysbio/syy001], we know that the most recent common ancestor (MRCA) of humans and mouse lemurs is MRCA of all living primate species and that this MRCA likely arose near the Cretaceous-Paleogene (K-Pg) boundary 65 mya. 
Thus, we can reasonably conclude that features that are shared between humans and mouse lemurs are homologous and characteristic of the primate clade.

We have a limited knowledge of the epigenetic architecture of mouse lemurs as we far lack chromatin data. 
Prior to this study, it was unknown whether repair profiles would be similar in the comparison of two primates species. 
Our work revealed that human and mouse lemur fibroblasts are significantly similar with respect to their preference on which genomic region to repair first. 
This bias is likely to be caused by the chromatin structure, as previously shown in other organisms [@doi:10.1038/s41467-019-08290-2].
Damage distribution is known to be homogenous at large genomic scales [@doi:10.1073/pnas.1706522114]. 
Therefore, it is safe to assume that repair preference is the major factor for XR-seq heterogeneity.
The repair profiles of human and mouse lemur fibroblasts are much more similar to each other than the ones between two distinct human cell lines. 
Moreover, (6-4)PP and CPD repair profiles are more similar to themselves, suggesting similar repair dynamics between two organisms.
Further studies on mouse lemur epigenetics will possibly explain the most differentiated genomic repair levels in mouse lemurs. 
Better understanding mouse lemur genomics, as well as other species within the strepsirrhine clade, will provide additional insight into human evolution, repair and mutagenesis.






## Conclusions

We demonstrated that genome-wide UV repair profiles between human and and mouse lemur are strongly correlated.
Our results shed light onto deep homologies of same tissue types in primates by suggesting similar epigenetic architecture between them.
We believe our results on varying repair rates between the two species will guide follow-up studies investigating the evolutionary effects of day time activity and UV avoidance on differential nucleotide excision repair rates.

## Abbreviations

GR: Global repair

TCR: Transcription-coupled repair

RNAPII: RNA polymerase II

XR-seq: excision repair sequencing

CPD: cyclobutane pyrimidine dimer

(6-4)PP: 6-4 pyrimidine-pyrimidone photo products

NHF1: normal human fibroblasts 1

TS: transcribed strand

NTS: non-transcribed strand

RPKM: reads per kilobase per million mapped reads

GEO: Gene Expression Omnibus

## Methods
<!-- ## Materials and Methods -->

### Cell lines and reagents

#### Establishment of the immortalized mouse lemur fibroblast cell line

Primary fibroblasts were established using methods outlined in Larsen and Harris et al [@doi:10.1186/s12915-017-0439-6].
For the current study, we used an immortalized version of these primary mouse lemur fibroblasts.
This cell line was generated by transfecting passage six primary cells at approximately 60% confluency with 5 µg of human hTERT (hTERT) plasmid DNA per well of a 6-well plate using Lipofectamine® 2000 diluted in Opti-MEM® medium.
Twenty-four hours post-transfection, cells were placed under selection using G418 sulfate at a concentration of 300 µg/mL for two weeks.
Selected cells were then plated into a 96-well plate and remained under G418 sulfate maintenance at a concentration of 100 µg/mL to establish single cell colonies.
Approximately 2 weeks post-transfection, colonies were observed in 19 of the 96 wells and these cells underwent subsequent propagation and expansion.

#### Human cell lines

Telomerase-immortalized normal human fibroblasts (NHF1) were obtained from W.K. Kaufmann (University of North Carolina, Chapel Hill) as previously described [@doi:10.1128/MCB.22.24.8552-8561.2002;@doi:10.1101/gad.261271.115].
All cells were grown in Dulbecco’s modified Eagle’s medium supplemented with 10% fetal bovine serum at 37°C in a 5% CO2 humidified chamber.
Human lymphocyte GM12878 was purchased from Coriell.

### Clonogenic Survival Assay

Cells were seeded in 10 cm plates at 300 cells/plate, grown for 16 hours, and then UV irradiated as follows.
After removing and reserving culture medium, cells were placed under a GE germicidal lamp that emits primarily 254-nm UV light (UV-C) at 1 J/m^2^sec connected to a digital timer to give the indicated dose.
Following irradiation, the culture medium was replaced, and the cells were incubated for 7 days.
Plates were then washed with phosphate-buffered saline (PBS), the cells were fixed for 10 min in 75% methanol/25% acetic acid, and stained with 0.5% crystal violet stain (J.T. Baker) in 25% methanol for 30 min.
The plates were then washed extensively in tap water to remove the excess stain.
After drying, images of the stained cells were taken on a Molecular Imager Chemi-Doc XRS+ system (Bio-Rad) for the presentation of representative, qualitative results.
Colonies were counted and the surviving fraction was calculated by dividing the number of colonies on treated plates by the number on the untreated plates.
Each condition was performed in triplicate and the experiment was repeated three independent times.

### Immunoslot Blot Analysis
The repair of UV photoproducts from genomic DNA was measured as described previously [@doi:10.1158/0008-5472.CAN-10-0095].
Cells were UV irradiated, for a dose of 10J/m^2^.
Following irradiation, the culture medium was replaced, and the cells were incubated for the indicated periods of time.
Genomic DNA (250 ng), isolated with a QIAamp DNA Mini kit (Qiagen), was immobilized on a nitrocellulose membrane with a Bio-Dot SF Cell immunoslot blot apparatus (Bio-Rad) and heated for 90 min at 80 °C under vacuum.
Blots were blocked in 5% milk and probed with an anti-(6-4)PP antibody (Cosmo Bio 64M-2 cat#NM-DND-001) or anti-CPD antibody (Cosmo Bio TDM-2 cat#NM-DND-002) as indicated.
The secondary antibody was horseradish peroxidase-linked anti-mouse IgG from GE Healthcare (catalog no. NA931V) and the chemiluminescent signals were visualized with Clarity Western ECL Substrate (Bio-Rad) and using a Molecular Imager Chemi-Doc XRS+ system (Bio-Rad).
Following immunoblotting, the blots were re-blotted with anti-ssDNA antibody (Millipore MAB3034 clone 16-19) to ensure equal loading of DNA.
The experiment was repeated three times, and representative results are presented.

### Detection of Excised Oligonucleotide Products of Nucleotide Excision Repair
Nucleotide excision repair activity was visualized, as previously described [@doi:10.1074/jbc.M113.482257].
Cells grown to ~80% confluency in 15-cm plates were harvested the indicated time after irradiation with 20 J/m^2^ of UV.
The cells were pelleted by centrifugation, subjected to a modified Hirt procedure where cell pellets were resuspended in a 10× packed cell volume of lysis buffer (50 mM Tris-Cl, pH 8.0, 10 mM EDTA, 1.2% SDS and 100 μg/ml RNase A) and incubated for 15 min at room temperature.
Following the addition of a one-fourth volume of 5 M NaCl, the mixtures were gently mixed and incubated on ice for >16 h.
After centrifugation at maximum speed (20,000 × g) for 1 h, the supernatants were gently transferred to new tubes and treated with 20 μg of proteinase K for 15 min at 55 °C, extracted with phenol/chloroform, and then precipitated with ethanol.
The pellet was washed with 500 μl of 70% ethanol and resuspended in 10 μl of buffer EB (10 mM Tris-Cl (pH8.5)).
The excised oligonucleotide products of nucleotide excision repair were purified from the remaining material with either anti-(6-4)PP or anti-CPD antibodies as follows: For each reaction, 5 μl of protein G Dynabeads (Invitrogen, catalog no. 10003D) slurry and 5 μl of anti-rabbit Dynabeads (Invitrogen, catalog no. 11203D) slurry were washed three times with 50 μl of wash buffer I (20 mM Tris-Cl (pH 8.0), 2 mM EDTA, 150 mM NaCl, 1% Triton X-100, and 0.1% SDS) and then incubated with 1 μl of rabbit anti-mouse IgG and 1 μl of anti-(6-4)PP or anti-CPD antibody in 20 μl of IP buffer (20 mM Tris-Cl (pH 8.0), 2 mM EDTA, 150 mM NaCl, 1% Triton X-100, and 0.5% sodium deoxycholate) for 3 h at 4 °C.
After incubation, beads were separated from the liquid with a magnet and then mixed with 100 μl of IP buffer and 10 μl of DNA.
The mixtures were rotated at 4 °C overnight.
The beads were then washed sequentially with 200 μl each of wash buffer I, wash buffer II (20 mM Tris-Cl (pH 8.0), 2 mM EDTA, 500 mM NaCl, 1% Triton X-100, and 0.1% SDS), wash buffer III (10 mM Tris-Cl (pH 8.0), 1 mM EDTA, 150 mM LiCl, 1% Nonidet P-40, and 1% sodium deoxycholate), wash buffer IV (100 mM Tris-Cl (pH 8.0), 1 mM EDTA, 500 mM LiCl, 1% Nonidet P-40, and 1% sodium deoxycholate) and finally twice with TE (10 mM Tris-Cl (pH 8.0) and 1 mM EDTA).
The oligonucleotides containing UV photoproducts were eluted by incubation with 100 μl of elution buffer (50 mM NaHCO3, 1% SDS, and 20 μg/ml glycogen) at 65 °C for 15 min.
The eluted DNA was then isolated by phenol/chloroform extraction and followed by ethanol precipitation.
The excised oligonucleotides were resuspended in 10 μl of water, and half of the DNA was 3′-end-labeled for 1 h at 37 °C in a 10-μl reaction containing 6 units of terminal deoxynucleotidyl transferase (New England Biolabs), 0.25 mM CoCl2, and 1 μCi of [α-32P]-3′-deoxyadenosine 5′-triphosphate (cordycepin 5′-triphosphate, Perkin Elmer Life Sciences) in 1× terminal deoxynucleotidyl transferase buffer (New England Biolabs).
2.5 fmol of a 50-nucleotide oligomer was included in all reactions as an internal control, and oligonucleotides of known length were resolved on all gels as size markers.
Following phenol-chloroform extraction and ethanol precipitation, the DNA was separated on urea-containing polyacrylamide gels, detected with a phosphorimager and was quantified using ImageQuant software (version 5.2, GE Healthcare).
The experiment was repeated three times.

### XR-seq library preparation

Samples were processed for XR-seq, as previously described [@doi:10.1038/s41596-018-0093-7].
Cells grown to ~80% confluency in 15-cm plates were harvested either 5 minutes or 1 h after irradiation with 20 J/m^2^ of UV, depending on the damage, (6-4)PP or CPD respectively, to be analyzed.
Samples from four plates were pooled, lysed, and immunoprecipitated with anti-TFIIH antibodies p62 (Santa Cruz Biotechnology, sc292) and p89 (XPB, Santa Cruz Biotechnology, sc293), and then ligated to adapters and processed for sequencing.

### RNA-seq library preparation

Total RNA was isolated from one 10-cm plate of exponentially growing NHF1 or Lemur cells using Trizol (Invitrogen) and the RNeasy Mini Kit (Qiagen) following the manufacturer's instructions.
The library preparation and strand-specific paired-end sequencing (2 × 150 bp) on a HiSeq 4000 platform (Illumina) were performed by Novogene Co., Ltd.

### Human and lemur genome alignment

MUMmer version 3.23 [@pmid:14759262] was used to align the grey mouse lemur genome (Mmur_3.0) to the human genome (hg19) with `nucmer` subprogram. 
The alignments were filtered based on criteria taking length and identity into account.
Alignments were listed in “delta format” by default by Nucmer.
For every reference (human) - query (lemur) pair, we kept the alignments which form the longest mutually consistent set.
The filtering step was performed with `delta-filter` subprogram.
`show-coords` subprogram was used to display summary information such as position, percent identity and other features of each alignment, in Btab format with `-B` and `-rclo` arguments.

We used `generateBED.R` [@url:https://github.com/compGenomeLab/lemurRepair] to write homologous human-lemur regions in a bed file.
Due to low genome coverage in XR-seq data sets, short regions might introduce random repair values.
Since we were not interested in too short alignments, we filtered out alignments shorter than 400 bp while generating the bed file.
Additionally, alignments with at percent similarity lower than 80 were filtered out of the list.
The output was a BED file with each homologous region between two genomes.

This output file contained regions in either genome that align to multiple regions in the other genome. To remove such regions, we used, we used `bedtools intersect` [@doi:10.1093/bioinformatics/btq033] to intersect the file with itself.
The command line we used was: `bedtools intersect -wo -s -a humanOverlapsLemur_short_noFilter.bed -b humanOverlapsLemur_short_noFilter.bed > human_intersect.txt`.
We compared this intersected output file to the `bed` file generated previously using `findDupAln.R` [@url:https://github.com/compGenomeLab/lemurRepair] to exclude regions in both genomes that align to multiple regions in other genome.
This resulted in a `bed` file of one-to-one homologous regions without multiple alignments for one region.


### XR-seq analysis

We trimmed 3’ adapter sequences (TGGAATTCTCGGGTGCCAAGGAACTCCAGTNNNNNNACGATCTCGTATGCCGTCTTCTGCTTG) using Cutadapt [@doi:10.14806/ej.17.1.200].
Bowtie2 version 2.3.4.1 [@doi:10.1038/nmeth.1923] was used to align sequencing reads onto reference genomes with default parameters.
With a two-step conversion process, the `sam` format was first converted (with SAMtools [@doi:10.1093/bioinformatics/btp352]) to `bam` format, which was then converted to `bed` format with BEDTools [@doi:10.1093/bioinformatics/btq033].
The resulting `bed` files were sorted and duplicate regions were removed.
Command line we used was:
`sort -u -k1,1 -k2,2n -k3,3n ${SAMPLE}_cutadapt.bed >${SAMPLE}_cutadapt_sorted.bed`

### RNA-seq analysis

Sequencing reads were aligned to the reference genome using STAR Aligner 2.6.1a [@doi:10.1093/bioinformatics/bts635] with default parameters.
The aligned reads were converted to `bed` format with `bedtools bamtobed`.
The resulting `bed` files were sorted by coordinates and duplicate regions were removed.
Command-line we used was:
`sort -u -k1,1 -k2,2n -k3,3n ${SAMPLE}.bed >${SAMPLE}_sorted.bed`

### XR-seq simulation

We evaluated the correlation of repair mechanisms with both simulated and real XR-seq datasets.
Simulated datasets were generated for the overlapping regions between the grey mouse lemur genome and the human genome using ART simulator [@doi:10.1093/bioinformatics/btr708].
We used `bedtools getfasta` program with the `bed` file containing homologous regions between two organisms to generate a reference `fasta` file for ART simulator to produce synthetic reads.
By default, ART simulator produces a `fastq` file with flat nucleotide distribution.
For a better representation of XR-seq characteristics in the simulated dataset, we obtain nucleotide distribution frequencies of both species XR-seq reads.
We applied a scoring function where each nucleotide in the simulated read was scored based on the frequency of that nucleotide to be in that position in the actual XR-seq dataset and obtained a total score for each simulated read.
Accordingly, best scoring 10 million reads were selected for both species from the simulated dataset using `filter_syn.go` [@url:https://github.com/compGenomeLab/lemurRepair]

### Downstream Analysis

To calculate nucleotide frequencies of excised oligomers, we first converted aligned Xr-seq reads to `fasta` format using `bedtools getfasta`. Using `frequency.go`  [@url:https://github.com/compGenomeLab/lemurRepair] we calculated the nucleotide frequency of predominant oligomer (26nt) for each primate and damage type.

Command line used to get excised oligomer lengths:
`awk '{print $3-$2}' ${SAMPLE}_cutadapt_sorted.bed | sort -k1,1n | uniq -c | sed 's/\s\s*/ /g' | awk '{print $2"\t"$1}'`

Statistical analyses of immunoslot blot repair assays and clonogenic survival assay have been performed with Welch Two Sample t-test by using `t-test.R` [@url:https://github.com/compGenomeLab/lemurRepair].

For both genomes, all annotated protein-coding genes were retrieved and genes that are closer to each other with less than 20kb were filtered out.
We used 5277 and 3366 number of genes for human and mouse lemur, respectively.
10 kb upstream and downstream of TSS and TES were divided into 100 bp windows.
XR-seq reads falling onto each bin were normalized to RPKM with `tcr.py` [@url:https://github.com/compGenomeLab/lemurRepair]
Then XR-seq RPKM values were normalized by the RPKM values derived by the mapped reads onto random genomic sites which are prepared using `bedtools shuffle` for protein-coding genes that are not closer to each other with less than 20kb.
Statistical significance of TCR profiles was estimated using Mann-Whitney U-test [@doi:10.1214/aoms/1177730491].
Statistical analyses were performed using `mann-w.py` [@url:https://github.com/compGenomeLab/lemurRepair] with SciPy [@doi:10.1038/s41592-019-0686-2].

We used `regions_rpkm.go` [@url:https://github.com/compGenomeLab/lemurRepair] to calculate RPKM values of XR-seq, RNA-seq, and simulated XR-seq for each overlapping region.
Regions with no XR-seq or RNA-seq reads were filtered out.

Circos plot for comprehensive summarization of all findings has been generated with `circosPlot.R` [@url:https://github.com/compGenomeLab/lemurRepair] by using R package `circlize`.[@doi:10.1093/bioinformatics/btu393]

PCA plot has been generated from read counts with  `plotPCA.R` [@url:https://github.com/compGenomeLab/lemurRepair], while  applying variance stabilizing transformation as described in DESeq2 [@doi:10.1186/s13059-014-0550-8] vignette.

Plots were prepared using `plotly` and `ggplot2`[@doi:10.1080/15366367.2019.1565254]. 
The manuscript was written using Manubot [@doi:10.1371/journal.pcbi.1007128].

<!-- ## Data Access

All raw and processed sequencing data generated in this study have been submitted to the NCBI Gene Expression Omnibus (GEO; https://www.ncbi.nlm.nih.gov/geo/) under accession number GSE145883. -->


## Declarations

### Availability of data and materials

All raw and processed sequencing data generated in this study have been submitted to the NCBI Gene Expression Omnibus (GEO; https://www.ncbi.nlm.nih.gov/geo/) under accession number GSE145883. 
The code repository for the data analyses can be accessed at https://github.com/CompGenomeLab/lemurRepair.

### Funding

The design of the repair assays is supported by the National Institutes of Health [GM118102, ES027255 to A.S.]. 
Cell line establishment was supported by the National Science Foundation [DEB-1354610 to A.Y.D].
Conceptualization and computational analysis were supported by European Molecular Biology Organization Installation Grant (to O.A.)


### Competing interests
<!-- ## CONFLICT OF INTEREST -->

The authors declare that there is no competing interests regarding the publication of this article.

## Acknowledgements
<!-- ## ACKNOWLEDGEMENT -->

We would like to thank Dr. Beth A Sullivan for the construct to make the TERT-immortalized lines.
A.Y.D is supported by the John Simon Guggenheim Foundation, and the Alexander von Humboldt Foundation.
O.A. is supported by TUBITAK 2232 International Fellowship for Outstanding Researchers Program and Young Scientist Grant (BAGEP) awarded by Science Academy, Turkey. 

*Authors' contributions:* OA, AS, LL-B and ADY conceived of the project and designed the experiments. Mouse lemur tissue was acquired by ADY and PAL, cell lines were prepared by AB providing reagents for immortalization (hTERT construct). LL-B performed experimental assays, XR-seq and RNA-seq experiments. VOK, UA and ZK performed computational analyses. OA interpreted the data and wrote the first draft. All authors have read, revised and approved the final manuscript.

<!-- ## Disclosure Declaration -->
<!-- ## Competing interests -->

<!-- The authors declare that there is no conflict of interest regarding the publication of this article. -->

## References 

<!-- Explicitly insert bibliography here -->
<div id="refs"></div>


## Figure legends

![ Characteristics of excised oligomers upon repair of UV-induced damages. 
*(A)* In vivo excision assay for (6-4)PP and CPD are shown. 
*(B)* XR-seq read length distribution of excised oligomers for CPD (top) and (6-4)PP (bottom). Human and mouse lemur are shown in blue and red, respectively. 
*(C)* Nucleotide frequency in the predominant oligomer (26nt) for each primate and damage type. Representative data from replicate 1.
](images/fig-0.png){#fig:fig1 .white} 

![ UV sensitivity and repair rates in human and lemur fibroblasts. 
*(A)* Clonogenic survival assay (left) and the quantified results (right).
*(B)* Immunoslot blot repair assays to examine the repair kinetics of UV-induced DNA lesions for both organisms and each damage type (left).
Remaining damage levels at each time point were quantified and plotted (right).
All experiments were repeated three times, and graphed data are presented as (mean+/-SD). 
Tests (t-test) performed (H~0~:μ~human~=μ~lemur~) for each dosage (A) and time points (B).
\* P ≤ 0.05; ** P ≤ 0.01; *** P ≤ 0.001; rest P > 0.05.
](images/fig-0.png){#fig:fig2 .white}

![ Transcription-coupled repair in mouse lemur and human fibroblasts. 
Average profiles of CPD XR-seq (top) and (6-4)PP XR-seq (bottom) over 5277 and 3366 annotated genes are plotted for human (left) and mouse lemur (right), respectively. 
Transcription start sites (TSS) and transcription end sites (TES) were retrieved from GTF (Gene Transfer Format) files for the two genomes.
Transcribed (red) and nontranscribed (blue) strand repair are shown in the downstream and upstream of TSS and TES, respectively.
10 kb upstream and downstream of TSS and TES were divided into 100 bp windows. 
XR-seq reads aligned to each bin were normalized to RPKM (reads per kilobase per million mapped reads). 
The XR-seq RPKM values were normalized by the shuffled RPKM values derived by mapped reads aligned to random genomic sites. 
Only replicate 1 is shown. TS/NTS median values for TSS downstream for CPD are 1.76 and 2.31 for human and mouse lemur, respectively.
](images/fig-0.png){#fig:fig3 .white}

![ Comparative analysis methodology and correlated repair rates between human and mouse lemur. 
A) Approach to identify orthologous regions between human and mouse lemur (see methods for details). 
B) Scatter plots showing the normalized repair levels between two organisms. 
Relative repair was calculated by normalizing the repair signal by the simulated XR-seq reads to eliminate the sequence context bias. 
C) Mapped orthologous regions with repair and transcription profiles mapped at left and right outer rings, respectively. 
Inner circle represents the human (left) and mouse lemur (right) chromosomes. 
Inner lines connecting orthologous regions are colored based on the human chromosome color scale. 
Repair and transcription values (outer rings) for genomic regions that have no evident ortholog (based on the criteria in A) are not shown.
](images/fig-0.png){#fig:fig4 .white}

![ Repair rate consistency in correlation with gene expression. 
The transcription levels of the orthologous regions were divided into quartiles separately for human and lemur. 
Out of the quartiles, 10 bootstrapped subsamples were retrieved, and for those regions repair correlation was analyzed. 
Correlation coefficient (R values) are represented on the y-axis; quartiles are on the x-axis.
](images/fig-0.png){#fig:fig5 .white}

![ Cell type-based consistency of repair profiles between two primates. A) Scatter plot or normalized repair levels between two human cell lines, GM18787 and NHF1, show correlation (R=0.65, p=0). B) Principle component analysis of 10 XR-seq samples. 
](images/fig-0.png){#fig:fig6 .white}

## Supplementary Materials {.page_break_before}

![ Nucleotide content of simulated XR-seq pseudo-oligomers.
](images/simulation.png){#fig:S1 tag="S1"}

![ Human and mouse lemur (6-4)PP repair correlation in human transcription quartiles.
](images/Human_RNA_64.png){#fig:S2 tag="S2"}

![ Human and mouse lemur (6-4)PP repair correlation in mouse lemur transcription quartiles.
](images/Lemur_RNA_64.png){#fig:S3 tag="S3"}

![ Human and mouse lemur CPD repair correlation in human transcription quartiles.
](images/Human_RNA_CPD.png){#fig:S4 tag="S4"}

![ Human and mouse lemur CPD repair correlation in mouse lemur transcription quartiles.
](images/Lemur_RNA_CPD.png){#fig:S5 tag="S5"}

![ PCA plot for XR-seq reads mapped on the genic (A) and intergenic (B) regions.
](images/pca_all_v7.png){#fig:S6 tag="S6"}
