<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <meta name="author" content="Umit Akkose" />
  <meta name="author" content="Veysel Ogulcan Kaya" />
  <meta name="author" content="Laura Lindsey-Boltz" />
  <meta name="author" content="Zeynep Karagoz" />
  <meta name="author" content="Adam D. Brown" />
  <meta name="author" content="Peter A. Larsen" />
  <meta name="author" content="Anne D. Yoder" />
  <meta name="author" content="Aziz Sancar" />
  <meta name="author" content="Ogun Adebali" />
  <meta name="dcterms.date" content="2021-05-01" />
  <meta name="keywords" content="DNA repair, Nucleotide excision repair, UV damage, Gray Mouse Lemur, Fibroblasts" />
  <title>Comparative analyses of two primate species diverged by more than 60 million years show different rates but similar distribution of genome-wide UV repair events</title>
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    span.underline{text-decoration: underline;}
    div.column{display: inline-block; vertical-align: top; width: 50%;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
  </style>
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
  <!--
  Manubot generated metadata rendered from header-includes-template.html.
  Suggest improvements at https://github.com/manubot/manubot/blob/main/manubot/process/header-includes-template.html
  -->
  <meta name="dc.format" content="text/html" />
  <meta name="dc.title" content="Comparative analyses of two primate species diverged by more than 60 million years show different rates but similar distribution of genome-wide UV repair events" />
  <meta name="citation_title" content="Comparative analyses of two primate species diverged by more than 60 million years show different rates but similar distribution of genome-wide UV repair events" />
  <meta property="og:title" content="Comparative analyses of two primate species diverged by more than 60 million years show different rates but similar distribution of genome-wide UV repair events" />
  <meta property="twitter:title" content="Comparative analyses of two primate species diverged by more than 60 million years show different rates but similar distribution of genome-wide UV repair events" />
  <meta name="dc.date" content="2021-05-01" />
  <meta name="citation_publication_date" content="2021-05-01" />
  <meta name="dc.language" content="en-US" />
  <meta name="citation_language" content="en-US" />
  <meta name="dc.relation.ispartof" content="Manubot" />
  <meta name="dc.publisher" content="Manubot" />
  <meta name="citation_journal_title" content="Manubot" />
  <meta name="citation_technical_report_institution" content="Manubot" />
  <meta name="citation_author" content="Umit Akkose" />
  <meta name="citation_author_institution" content="Molecular Biology, Genetics &amp; Bioengineering Program, Faculty of Engineering and Natural Sciences, Sabanci University, Istanbul, 34956, Turkey" />
  <meta name="citation_author_orcid" content="0000-0002-9731-3702" />
  <meta name="citation_author" content="Veysel Ogulcan Kaya" />
  <meta name="citation_author_institution" content="Molecular Biology, Genetics &amp; Bioengineering Program, Faculty of Engineering and Natural Sciences, Sabanci University, Istanbul, 34956, Turkey" />
  <meta name="citation_author_orcid" content="0000-0002-3447-0735" />
  <meta name="citation_author" content="Laura Lindsey-Boltz" />
  <meta name="citation_author_institution" content="Department of Biochemistry and Biophysics, School of Medicine, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina, 27599, USA" />
  <meta name="citation_author_orcid" content="0000-0001-6493-8194" />
  <meta name="citation_author" content="Zeynep Karagoz" />
  <meta name="citation_author_institution" content="Molecular Biology, Genetics &amp; Bioengineering Program, Faculty of Engineering and Natural Sciences, Sabanci University, Istanbul, 34956, Turkey" />
  <meta name="citation_author_orcid" content="0000-0002-7627-9218" />
  <meta name="citation_author" content="Adam D. Brown" />
  <meta name="citation_author_institution" content="Department of Pharmacology and Cancer Biology, Duke University, Durham, North Carolina, 27708, USA" />
  <meta name="citation_author_orcid" content="0000-0002-9122-4960" />
  <meta name="citation_author" content="Peter A. Larsen" />
  <meta name="citation_author_institution" content="Department of Biology, Duke University, Durham, North Carolina, 27708, USA" />
  <meta name="citation_author_orcid" content="0000-0002-3634-3625" />
  <meta name="citation_author" content="Anne D. Yoder" />
  <meta name="citation_author_institution" content="Department of Biology, Duke University, Durham, North Carolina, 27708, USA" />
  <meta name="citation_author_orcid" content="0000-0002-1781-9552" />
  <meta name="citation_author" content="Aziz Sancar" />
  <meta name="citation_author_institution" content="Department of Biochemistry and Biophysics, School of Medicine, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina, 27599, USA" />
  <meta name="citation_author_orcid" content="0000-0001-6469-4900" />
  <meta name="citation_author" content="Ogun Adebali" />
  <meta name="citation_author_institution" content="Molecular Biology, Genetics &amp; Bioengineering Program, Faculty of Engineering and Natural Sciences, Sabanci University, Istanbul, 34956, Turkey" />
  <meta name="citation_author_orcid" content="0000-0001-9213-4070" />
  <meta name="twitter:creator" content="@compGenomics" />
  <link rel="canonical" href="https://CompGenomeLab.github.io/lemur-manu/" />
  <meta property="og:url" content="https://CompGenomeLab.github.io/lemur-manu/" />
  <meta property="twitter:url" content="https://CompGenomeLab.github.io/lemur-manu/" />
  <meta name="citation_fulltext_html_url" content="https://CompGenomeLab.github.io/lemur-manu/" />
  <meta name="citation_pdf_url" content="https://CompGenomeLab.github.io/lemur-manu/manuscript.pdf" />
  <link rel="alternate" type="application/pdf" href="https://CompGenomeLab.github.io/lemur-manu/manuscript.pdf" />
  <link rel="alternate" type="text/html" href="https://CompGenomeLab.github.io/lemur-manu/v/44f5bc48b6869d288b107da645614c3991d2bcb5/" />
  <meta name="manubot_html_url_versioned" content="https://CompGenomeLab.github.io/lemur-manu/v/44f5bc48b6869d288b107da645614c3991d2bcb5/" />
  <meta name="manubot_pdf_url_versioned" content="https://CompGenomeLab.github.io/lemur-manu/v/44f5bc48b6869d288b107da645614c3991d2bcb5/manuscript.pdf" />
  <meta property="og:type" content="article" />
  <meta property="twitter:card" content="summary_large_image" />
  <link rel="icon" type="image/png" sizes="192x192" href="https://manubot.org/favicon-192x192.png" />
  <link rel="mask-icon" href="https://manubot.org/safari-pinned-tab.svg" color="#ad1457" />
  <meta name="theme-color" content="#ad1457" />
  <!-- end Manubot generated metadata -->
</head>
<body>
<header id="title-block-header">
<h1 class="title">Comparative analyses of two primate species diverged by more than 60 million years show different rates but similar distribution of genome-wide UV repair events</h1>
</header>
<!-- [![ORCID icon](images/orcid.svg){height="11px" width="11px"}](https://orcid.org/0000-0002-9731-3702) -->
<p>Umit Akkose<sup>1,†</sup>,
<!-- [![ORCID icon](images/orcid.svg){height="11px" width="11px"}](https://orcid.org/0000-0002-3447-0735) -->
Veysel Ogulcan Kaya<sup>1,†</sup>,
<!-- [![ORCID icon](images/orcid.svg){height="11px" width="11px"}](https://orcid.org/0000-0001-6493-8194) -->
Laura Lindsey-Boltz<sup>2,†</sup>,
<!-- [![ORCID icon](images/orcid.svg){height="11px" width="11px"}](https://orcid.org/0000-0002-7627-9218) -->
Zeynep Karagoz<sup>1</sup>,
<!-- [![ORCID icon](images/orcid.svg){height="11px" width="11px"}](https://orcid.org/0000-0002-9122-4960) -->
Adam D. Brown<sup>3</sup>,
<!-- [![ORCID icon](images/orcid.svg){height="11px" width="11px"}](https://orcid.org/0000-0002-3634-3625) -->
Peter A. Larsen<sup>4</sup>,
<!-- [![ORCID icon](images/orcid.svg){height="11px" width="11px"}](https://orcid.org/0000-0002-1781-9552) -->
Anne D. Yoder<sup>4</sup>,
<!-- [![ORCID icon](images/orcid.svg){height="11px" width="11px"}](https://orcid.org/0000-0001-6469-4900) -->
Aziz Sancar<sup>2</sup>,
<!-- [![ORCID icon](images/orcid.svg){height="11px" width="11px"}](https://orcid.org/0000-0001-9213-4070) -->
Ogun Adebali<sup>1</sup>,*</p>
<p><small></p>
<ol type="1">
<li>Molecular Biology, Genetics &amp; Bioengineering Program, Faculty of Engineering and Natural Sciences, Sabanci University, Istanbul, 34956, Turkey</li>
<li>Department of Biochemistry and Biophysics, School of Medicine, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina, 27599, USA</li>
<li>Department of Pharmacology and Cancer Biology, Duke University, Durham, North Carolina, 27708, USA</li>
<li>Department of Biology, Duke University, Durham, North Carolina, 27708, USA</li>
</ol>
<!-- ^#^ Current Address: Department of Veterinary and Biomedical Sciences, University of Minnesota, St. Paul, Minnesota, 55112, USA <br> -->
<!-- <br> -->
<p><sup>†</sup> Joint Authors</p>
<p>* To whom correspondence should be addressed. Tel: +90 216 568 7043; Fax: +90 216 483 9550; E-mail: oadebali@sabanciuniv.edu</p>
<p>Present Address: [Peter A. Larsen] Department of Veterinary and Biomedical Sciences, University of Minnesota, St. Paul, Minnesota, 55112, USA</p>
<p></small></p>
<p><small>
<!-- **Running title**: Nucleotide excision repair profiles in primates -->
</small></p>
<p><small>
<!-- **Keywords:** Nucleotide excision repair, UV damage, XR-seq, Mouse Lemur, (6-4)PP, CPD, Primate -->
</small></p>
<!-- ## Abstract {.page_break_before} -->
<h2 id="abstract">Abstract</h2>
<!-- 150 words Nat Comm -->
<!-- Nucleotide excision repair is the primary DNA repair mechanism that removes bulky DNA adducts such as UV-induced pyrimidine dimers. 
Although human repair profiles were extensively studied, how repair maps vary between primates is yet to be investigated.
Here, we characterized the genome-wide UV-induced damage repair in gray mouse lemur, *Microcebus murinus*, in comparison to human.
Mouse lemurs are strictly nocturnal, are the world's smallest living primates, and last shared a common ancestor with humans at least 60 million years ago. 
We derived fibroblast cell lines from mouse lemur, exposed them to UV irradiation and captured excised oligomers through the XR-seq protocol. 
Overall, UV sensitivity, repair efficiency, and transcription-coupled repair levels differed between the two primates.
Despite this, genome-wide repair profiles of the homologous regions are highly correlated. 
We suggest mouse lemurs and humans, and possibly primates in general, share a homologous repair mechanism as well as genomic variance distribution, albeit with their variable repair efficiency.
Our results also emphasize the deep homologies of individual tissue types across the eukaryotic phylogeny. -->
<!-- 200 words NAR -->
<!-- Nucleotide excision repair is the primary DNA repair mechanism that removes bulky DNA adducts such as UV-induced pyrimidine dimers. 
Correspondingly, genome-wide mapping of nucleotide excision repair with eXcision Repair sequencing (XR-seq), provides comprehensive profiling of DNA damage repair. 
Although human repair profiles were extensively studied, how repair maps vary between primates is yet to be investigated.
Here, we characterized the genome-wide UV-induced damage repair in gray mouse lemur, *Microcebus murinus*, in comparison to human.
Mouse lemurs are strictly nocturnal, are the world's smallest living primates, and last shared a common ancestor with humans at least 60 million years ago. 
We derived fibroblast cell lines from mouse lemur, exposed them to UV irradiation and captured excised oligomers through the XR-seq protocol. 
Mouse lemur repair profiles were analyzed in comparison to the human fibroblasts.
Overall, UV sensitivity, repair efficiency, and transcription-coupled repair levels differed between the two primates.
Despite this, genome-wide repair profiles of the homologous regions are highly correlated. 
We suggest mouse lemurs and humans, and possibly primates in general, share a homologous repair mechanism as well as genomic variance distribution, albeit with their variable repair efficiency.
Our results also emphasize the deep homologies of individual tissue types across the eukaryotic phylogeny. -->
<!-- Key points
* Nucleotide excision repair mechanisms in mouse lemur and human are homologous.
* UV repair rates between human and lemur differ.
* Genome-wide UV repair profiles are similar between the two primates.
 -->
<!-- Nucleotide excision repair is the primary DNA repair mechanism that removes bulky DNA adducts such as UV-induced pyrimidine dimers. 
Correspondingly, genome-wide mapping of nucleotide excision repair with eXcision Repair sequencing (XR-seq), provides comprehensive profiling of DNA damage repair. 
A number of XR-seq experiments at a variety of conditions for different damage types revealed heterogenous repair in the human genome.
Although human repair profiles were extensively studied, how repair maps vary between primates is yet to be investigated.
Here, we characterized the genome-wide UV-induced damage repair in gray mouse lemur,*Microcebus murinus*, in comparison to human.
Mouse lemurs are strictly nocturnal, are the world's smallest living primates, and last shared a common ancestor with humans at least 60 million years ago. 
We derived fibroblast cell lines from mouse lemur, exposed them to UV irradiation. 
The following repair events were captured genome-wide through the XR-seq protocol. 
Mouse lemur repair profiles were analyzed in comparison to the equivalent human fibroblast datasets.
We found that overall UV sensitivity, repair efficiency, and transcription-coupled repair levels differ between the two primates.
Despite this, comparative analysis of human and mouse lemur fibroblasts revealed that genome-wide repair profiles of the homologous regions are highly correlated. 
This correlation is stronger for the highly expressed genes.
With the inclusion of an additional XR-seq sample derived from another human cell line in the analysis, we found that fibroblasts of the two primates repair UV-induced DNA lesions in a more similar pattern than two distinct human cell lines do.
Our results suggest that mouse lemurs and humans, and possibly primates in general, share a homologous repair mechanism as well as genomic variance distribution, albeit with their variable repair efficiency.
This result also emphasizes the deep homologies of individual tissue types across the eukaryotic phylogeny. -->
<!-- 250 words Genome Research-->
<h3 id="background">Background</h3>
<p>Nucleotide excision repair is the primary DNA repair mechanism that removes bulky DNA adducts such as UV-induced pyrimidine dimers.
Correspondingly, genome-wide mapping of nucleotide excision repair with eXcision Repair sequencing (XR-seq), provides comprehensive profiling of DNA damage repair.
A number of XR-seq experiments at a variety of conditions for different damage types revealed heterogenous repair in the human genome.
Although human repair profiles were extensively studied, how repair maps vary between primates is yet to be investigated.
Here, we characterized the genome-wide UV-induced damage repair in gray mouse lemur,<em>Microcebus murinus</em>, in comparison to human.</p>
<h3 id="results">Results</h3>
<p>We derived fibroblast cell lines from mouse lemur, exposed them to UV irradiation, and analyzed the repair events genome-wide using the XR-seq protocol.
Mouse lemur repair profiles were analyzed in comparison to the equivalent human fibroblast datasets.
We found that overall UV sensitivity, repair efficiency, and transcription-coupled repair levels differ between the two primates.
Despite this, comparative analysis of human and mouse lemur fibroblasts revealed that genome-wide repair profiles of the homologous regions are highly correlated, and this correlation is stronger for highly expressed genes.
With the inclusion of an additional XR-seq sample derived from another human cell line in the analysis, we found that fibroblasts of the two primates repair UV-induced DNA lesions in a more similar pattern than two distinct human cell lines do.</p>
<h3 id="conclusion">Conclusion</h3>
<p>Our results suggest that mouse lemurs and humans, and possibly primates in general, share a homologous repair mechanism as well as genomic variance distribution, albeit with their variable repair efficiency.
This result also emphasizes the deep homologies of individual tissue types across the eukaryotic phylogeny.</p>
<h2 id="keywords">Keywords</h2>
<p>Nucleotide excision repair, UV damage, XR-seq, Mouse Lemur, (6-4)PP, CPD, Primate</p>
<!-- ## Introduction  -->
<h2 id="background-1">Background</h2>
<p>Nucleotide excision repair is an essential mechanism to remove bulky DNA adducts, including UV-induced DNA lesions <span class="citation" data-cites="bYJTOmGw">[<a href="#ref-bYJTOmGw" role="doc-biblioref">1</a>]</span>.
As in other repair systems, excision repair starts with damage recognition.
Two subpathways based on damage recognition lead to two repair mechanisms: global repair (GR) and transcription-coupled repair (TCR).
GR is active throughout the genome, whereas TCR is only active on the transcribed strands as it is initiated by damage recognition through stalled RNA polymerase II (RNAPII).
To date, many techniques have been developed to detect DNA damage and repair <span class="citation" data-cites="cZtF6zIe">[<a href="#ref-cZtF6zIe" role="doc-biblioref">2</a>]</span>.
The approach of some methodologies has been NGS-based, which allows answering genome-wide questions.
To reveal genome-wide excision repair dynamics, heterogeneity and associations, eXcsion Repair sequencing (XR-seq) was developed.
The XR-seq technique directly measures the repair events by capturing excised DNA oligomers containing the lesion.
It was found that TCR is more efficient, particularly for slowly-repaired DNA lesions.
For instance, among UV photoproducts 6-4 pyrimidine-pyrimidone ([6-4]PP) and cyclobutane pyrimidine dimer (CPD), CPD is more prone to TCR as GR less efficiently repairs it.
Although DNA lesions might preferentially form at certain local sites in the genome, the overall heterogeneity of repair is mainly due to uneven repair efficiency throughout the genome <span class="citation" data-cites="10FU8PPzJ zxIvoFqn">[<a href="#ref-10FU8PPzJ" role="doc-biblioref">3</a>,<a href="#ref-zxIvoFqn" role="doc-biblioref">4</a>]</span>.
Genome-wide heterogenous repair distribution is mostly caused by transcription and chromatin structure <span class="citation" data-cites="OhKg9ZHE ZAV7FWhL 4WTs70ml hJRoVR8L">[<a href="#ref-OhKg9ZHE" role="doc-biblioref">5</a>,<a href="#ref-ZAV7FWhL" role="doc-biblioref">6</a>,<a href="#ref-4WTs70ml" role="doc-biblioref">7</a>,<a href="#ref-hJRoVR8L" role="doc-biblioref">8</a>]</span>.</p>
<p>To date, genome-wide repair maps were generated for model organisms, including <em>Escherichia coli</em> <span class="citation" data-cites="18edpunap 1190rUeCp">[<a href="#ref-18edpunap" role="doc-biblioref">9</a>,<a href="#ref-1190rUeCp" role="doc-biblioref">10</a>]</span>, <em>Saccharomyces cerevisiae</em> <span class="citation" data-cites="OSVE68co">[<a href="#ref-OSVE68co" role="doc-biblioref">11</a>]</span>, <em>Drosophila melanogaster</em> <span class="citation" data-cites="iyn6rN7q">[<a href="#ref-iyn6rN7q" role="doc-biblioref">12</a>]</span>, <em>Arabidopsis thaliana</em> <span class="citation" data-cites="12APhIkm8">[<a href="#ref-12APhIkm8" role="doc-biblioref">13</a>]</span>, <em>Mus musculus</em> <span class="citation" data-cites="AXdS7vVN">[<a href="#ref-AXdS7vVN" role="doc-biblioref">14</a>]</span>, and <em>Homo sapiens</em> <span class="citation" data-cites="OhKg9ZHE 10FU8PPzJ">[<a href="#ref-10FU8PPzJ" role="doc-biblioref">3</a>,<a href="#ref-OhKg9ZHE" role="doc-biblioref">5</a>]</span>.
TCR presence was verified for each of these species.
For eukaryotic genomes, the consistent finding was the efficient repair in open chromatin regions.
Heterochromatin regions were found to be repaired at later time points.
Human repair profiles were extensively studied with respect to damage formation and chromatin states.
Whether regions in the human genome that are efficiently repaired are organism-specific is yet to be investigated.</p>
<p>To study whether repair patterns are unique to the organism of interest, we aimed to compare human and a deeply diverged non-human primate.
Gray mouse lemur (<em>Microcebus murinus</em>) stands out as a promising model organism candidate because of its small body size, short gestation time (2 months) and fast sexual maturation (6-8 months) <span class="citation" data-cites="CoujlGZF fKeXK6h7">[<a href="#ref-CoujlGZF" role="doc-biblioref">15</a>,<a href="#ref-fKeXK6h7" role="doc-biblioref">16</a>]</span>.
A near chromosome level reference genome for the gray mouse lemur was recently sequenced and assembled <span class="citation" data-cites="1FNsl9JHL">[<a href="#ref-1FNsl9JHL" role="doc-biblioref">17</a>]</span>.
With no surprise, it was shown that mouse lemur and human orthologs share ~91% identity.
Although a robust genome assembly is available, we lack an in-depth understanding of this species’ genomic features such as epigenetic maps, transcriptomes and methylomes.</p>
<p>Given that mouse lemurs and humans last shared a common ancestor at the base of the primate clade <span class="citation" data-cites="1Gbi7LeDk">[<a href="#ref-1Gbi7LeDk" role="doc-biblioref">18</a>]</span>, the same cell types from human and mouse lemur should behave similarly in response to DNA damage as a reflection of their deep homology.
With this motivation, we carried out a comparative study between these two primates to understand similarities and differences between their repair profiles.
We derived primary fibroblasts from mouse lemur and immortalized them.
We performed survival assays in response to UV stress, immunoslot blot assays and in vivo excision assays for both cell lines.
From mouse lemur fibroblasts, we obtained transcriptomes, exposed cells to UV and performed XR-seq.
XR-seq captured the excised oligomers as repair products for two main UV-induced damage types: (6-4)PP and CPD.
We compared lemur and human fibroblast XR-seq datasets for their genomic repair distribution.</p>
<h2 id="results-1">Results</h2>
<h3 id="excised-oligomer-characteristics">Excised oligomer characteristics</h3>
<p>The in vivo excision assay was used to analyze the excised oligomers containing UV-induced lesions (Fig <a href="#fig:fig1">1</a>A).
Excised oligomers were captured for the two distinct damage types with specific anti-damage antibodies.
The oligomer length distribution varies from 16 to 30nt.
Two intense bands were observed, which indicate primary and degraded excised oligomers.
The gel images show that the primary excised oligomer lengths vary between 23-25nt.
Through the time course, gel images reflect more intense secondary products, indicating higher levels of degraded excised oligomers at later time points.
For normal human fibroblasts 1 (NHF1) and mouse lemur fibroblasts, this trend is similar.
The human and lemur experiments were not conducted together and thus, the band intensities differ.
The spike-in 50-mer internal control was used for quantification of the excision products between experiments.</p>
<p>Nucleotide excision repair in human, <em>S. cerevisiae</em> and <em>E. coli</em> was shown to yield primary excised oligomers predominantly in lengths of 27, 24 and 13 nucleotides, respectively <span class="citation" data-cites="1GUFhcYbP RCFCwjid OSVE68co DObZlup3 18edpunap">[<a href="#ref-18edpunap" role="doc-biblioref">9</a>,<a href="#ref-OSVE68co" role="doc-biblioref">11</a>,<a href="#ref-1GUFhcYbP" role="doc-biblioref">19</a>,<a href="#ref-RCFCwjid" role="doc-biblioref">20</a>,<a href="#ref-DObZlup3" role="doc-biblioref">21</a>]</span>.
Mouse lemur XR-seq resulted in oligomer length distribution identical to humans (Fig <a href="#fig:fig1">1</a>B).
For both CPD and (6-4)PP, nucleotide excision repair machinery incised 26nt oligomer as the predominant product.
A small variance in oligomer lengths between the in vivo excision assay and XR-seq might be due to the exonuclease activity on the excised oligomers in the vivo excision assay.
Because of the TFIIH coimmunoprecipitation in the excision repair sequencing technique, XR-seq primary products are not yet degraded, possibly because of TFIIH protection of the excised oligomers.</p>
<h3 id="identical-cpd-and-6-4pp-nucleotide-frequency-between-human-and-lemur">Identical CPD and (6-4)PP nucleotide frequency between human and lemur</h3>
<p>Nucleotide frequencies of the excised oligomers between human and mouse lemur are identical for both CPD and (6-4)PP (Fig <a href="#fig:fig1">1</a>C).
Pyrimidine enrichment was revealed at 19th, 20th and 21st nucleotides for CPD and (6-4)PP for both organisms, which is in agreement with the incision site at 6 to 8nt fixed distance to the 3’ end of the DNA lesion.
The nucleotide contents for CPD and (6-4)PP are different, as they were previously reported and discussed <span class="citation" data-cites="OhKg9ZHE">[<a href="#ref-OhKg9ZHE" role="doc-biblioref">5</a>]</span>.</p>
<figure>
<img src="images/fig-1.png" alt="" /><figcaption>Figure 1
<!-- Characteristics of excised oligomers upon repair of UV-induced damages. 
*(A)* In vivo excision assay for (6-4)PP and CPD are shown. 
"M": markers (sizes 17 and 24) and "-": no UV.  
*(B)* XR-seq read length distribution of excised oligomers for CPD (top) and (6-4)PP (bottom). Human and mouse lemur are shown in blue and red, respectively. 
*(C)* Nucleotide frequency in the predominant oligomer (26nt) for each primate and damage type. Representative data from replicate 1. --></figcaption>
</figure>
<h3 class="page_break_before" id="uv-repair-efficiency-of-mouse-lemur-in-comparison-with-human">UV repair efficiency of mouse lemur in comparison with human</h3>
<p>To identify the overall repair efficiency upon UV irradiation, we performed a colony survival assay.
We seeded 300 cells and grew them for 16 hours.
Cells were exposed to UV-C at 1 J/m<sup>2</sup>sec for variable time intervals to reach indicated doses ranging from 2.5 to 10 J/m<sup>2</sup>.
We counted cells for each dose and plotted survival curves for lemur and human fibroblasts (Fig <a href="#fig:fig2">2</a>A).
Based on the survival curves of the cell lines used in this study, we infer that mouse lemur fibroblasts are more sensitive to UV relative to human cells.
<!-- This suggests a differential repair rate between the two species, which is possibly reflective of the diurnal versus nocturnal habits of humans and mouse lemurs, respectively. --></p>
<p>We then performed immunoslot blot assays to measure the DNA repair rates (Fig <a href="#fig:fig2">2</a>B).
Cells were exposed to UV-C at 10J/m<sup>2</sup>.
To separate UV-induced DNA lesions, we used specific antibodies for CPD and (6-4)PP.
We quantified the remaining damage signal for each damage type for both species.
We observe a difference between rates of repair of CPD and 6-4 PP, as has been previously described, which is due to several reasons including differences in damage recognition and differences in global repair versus transcription coupled repair discussed further below.
Both (6-4)PP and CPD repair appeared to be more efficient in human cells compared to mouse lemur.
Human (6-4)PP repair was completed in 2 hours whereas in lemurs, it took 4 hours to complete (6-4)PP repair.
20% of CPDs were still unrepaired for human cells 32 hours after UV irradiation.
Unrepaired CPD ratio was more than 40% for mouse lemur cells.
Survival and immonoslot blot assays are in harmony and they suggest mouse lemur cells have relatively low repair rates compared to NHF1 cells.</p>
<figure>
<img src="images/fig-2.png" alt="" /><figcaption>Figure 2
<!-- UV sensitivity and repair rates in human and lemur fibroblasts. 
*(A)* Clonogenic survival assay (left) and the quantified results (right).
*(B)* Immunoslot blot repair assays to examine the repair kinetics of UV-induced DNA lesions for both organisms and each damage type (left).
Remaining damage levels at each time point were quantified and plotted (right).
All experiments were repeated three times, and graphed data are presented as (mean+/-SD). 
Tests (t-test) performed (H~0~:μ~human~=μ~lemur~) for each dosage (A) and time points (B).
\* P ≤ 0.05; ** P ≤ 0.01; *** P ≤ 0.001; rest P > 0.05. --></figcaption>
</figure>
<h3 id="prominent-transcription-coupled-repair-in-mouse-lemur">Prominent transcription-coupled repair in mouse lemur</h3>
<p>Nucleotide excision repair comprises two sub-pathways, each of which distinctly acts on damage recognition.
GR is the genome-wide mechanism that is active in any region in the genome, although its efficiency depends on the type of damage as well as chromatin factors.
On the other hand, TCR depends on the RNAPII stalled at the lesion.
Stalled RNAPII recruits transcription-coupled repair factor CSB to enhance repair in the transcribed strand of the genes.
DNA lesions in non-transcribed strands are subject to global repair.
A major rate-limiting step of nucleotide excision repair is damage recognition.
Therefore, minor helix-distorting lesions such as CPDs are recognized more efficiently when they stall elongating RNAPII, and thus genes have asymmetrical repair between two strands; the transcribed strand (TS) is repaired more efficiently compared to the nontranscribed strand (NTS) <span class="citation" data-cites="29W7k2UV NkhwUBAb">[<a href="#ref-29W7k2UV" role="doc-biblioref">22</a>,<a href="#ref-NkhwUBAb" role="doc-biblioref">23</a>]</span>.</p>
<p>Global repair for (6-4)PP is much faster than for CPD <span class="citation" data-cites="LyGwcDX7 4WTs70ml">[<a href="#ref-4WTs70ml" role="doc-biblioref">7</a>,<a href="#ref-LyGwcDX7" role="doc-biblioref">24</a>]</span>.
TCR removes a minor fraction of the (6-4)PPs, and thus strand asymmetry between TS and NTS (TS/NTS ratio) is much weaker for (6-4)PP compared to TS-favored strand asymmetry of CPD repair <span class="citation" data-cites="4WTs70ml">[<a href="#ref-4WTs70ml" role="doc-biblioref">7</a>]</span>.
CPDs, on the other hand, are harder to be recognized by global repair, and therefore it takes a longer time to remove CPDs from the genome.
For this reason, at early time points (such as 1 hour after UV irradiation), we observe a strong TCR effect, which is indicated by the asymmetrical repair between TS and NTS.
Here, we compared the genic strand asymmetry between human and mouse lemur (Fig <a href="#fig:fig3">3</a>).
We retrieved all annotated protein-coding genes from both genomes.
We removed the genes that are closer to each other with less than 20kb in order to remove the signal that might have a “canceling-out” effect.
We used 5277 and 3366 annotated genes for human and mouse lemur, respectively (see methods).
We performed a meta-analysis where we aligned all transcription start and transcription end sites and calculated the RPKM (reads per kilobase per million mapped reads) values.
Interestingly, mouse lemur fibroblasts exhibited stronger TCR profiles compared to human cell lines at 1 hour after CPD formation (TS/NTS Mann-Whitney U test p=4.3*10<sup>-21</sup>).
The higher TCR profile of lemur fibroblasts might be caused by the lower global repair efficiency relative to human fibroblasts (see discussion).</p>
<figure>
<img src="images/fig-3.png" alt="" /><figcaption>Figure 3
<!-- Transcription-coupled repair in mouse lemur and human fibroblasts. 
Average profiles of CPD XR-seq (top) and (6-4)PP XR-seq (bottom) over 5277 and 3366 annotated genes are plotted for human (left) and mouse lemur (right), respectively. 
Transcription start sites (TSS) and transcription end sites (TES) were retrieved from GTF (Gene Transfer Format) files for the two genomes.
Transcribed (red) and nontranscribed (blue) strand repair are shown in the downstream and upstream of TSS and TES, respectively.
10 kb upstream and downstream of TSS and TES were divided into 100 bp windows. 
XR-seq reads aligned to each bin were normalized to RPKM (reads per kilobase per million mapped reads). 
The XR-seq RPKM values were normalized by the shuffled RPKM values derived by mapped reads aligned to random genomic sites. 
Only replicate 1 is shown. TS/NTS median values for TSS downstream for CPD are 1.76 and 2.31 for human and mouse lemur, respectively. --></figcaption>
</figure>
<h3 id="conserved-repair-rates-between-homologous-regions-of-two-primates">Conserved repair rates between homologous regions of two primates</h3>
<p>In order to compare and contrast the genome-wide repair profiles between human and mouse lemur, we retrieved the homologous genomic regions between the organisms (Fig <a href="#fig:fig4">4</a>A).
We aligned the two genomes and kept the regions that have at least 80% identity.
Due to the low coverage of the XR-seq samples we removed the aligned segments that are shorter than 400 bases to eliminate the random effect due to scarce repair events.
Lineage-specific duplications (or deletions) result in paralogs that can be co-orthologs to one locus in the other genome.
This situation is known as the “one-to-multiple” orthology relationship.
As we cannot be sure which one of the multiple paralogs in one genome is the “true” ortholog of a region in the other one, we filtered out one-to-multiple kinds of homologous relationships and kept one-to-one homologs only.</p>
<p>1.0% and 1.3% of the genomes were taken into account with one-to-one homologs for human and mouse lemur, respectively.
4-5% and 2-3% of the XR-seq reads mapped to those regions for CPD and (6-4)PP, respectively.
The reason why the mapping rate of CPD is higher than (6-4)PP could be due to its slower repair rate, as slowly repaired CPDs are more prone to the effects of chromatin.
Because we applied stringent criteria to retrieve the homologous regions with at least 80% identity, they are likely to be more open compared to other regions in the genome.
Therefore, more CPD repair events aligned to those regions relative to (6-4)PP reads.</p>
<p>We investigated the correlation between the repair profiles of the two species.
Because excised oligomers have a certain nucleotide bias due to the damage site, we normalized the true repair signal (XR-seq) by the simulated XR-seq reads.
Simulated reads have the nucleotide content of the true XR-seq reads (Fig <a href="#fig:S1">S1</a>), however, they are randomly retrieved from the homologous regions.
With this approach, we eliminated the potential bias of the nucleotide content that creates a pseudo-correlation.
Although there is a clear difference in the relative TCR rates between two species (Fig <a href="#fig:fig3">3</a>), homologous regions of two genomes exhibited a strong repair rate correlation; R values for (6-4)PP and CPD are 0.80 and 0.72 (p=0), respectively (Fig <a href="#fig:fig4">4</a>B).
Not only repair but also transcription levels between the two species are correlated (Fig <a href="#fig:fig4">4</a>C).
The similar repair and transcription profiles between two cell lines are independent of their chromosomal locations.
Chromosomal homologous region associations are dispersed, as previously suggested <span class="citation" data-cites="1FNsl9JHL">[<a href="#ref-1FNsl9JHL" role="doc-biblioref">17</a>]</span>.</p>
<figure>
<img src="images/fig-4.png" alt="" /><figcaption>Figure 4
<!-- Comparative analysis methodology and correlated repair rates between human and mouse lemur. 
A) Approach to identify orthologous regions between human and mouse lemur (see methods for details). 
B) Scatter plots showing the normalized repair levels between two organisms. 
The relative repair was calculated by normalizing the repair signal by the simulated XR-seq reads to eliminate the sequence context bias. 
C) Mapped orthologous regions with repair and transcription profiles mapped at left and right outer rings, respectively. 
The inner circle represents the human (left) and mouse lemur (right) chromosomes. 
The inner lines connecting orthologous regions are colored based human chromosome color scale. 
Repair and transcription values (outer rings) for genomic regions that have no apparent ortholog (based on the criteria in A) are not shown. --></figcaption>
</figure>
<h3 id="repair-profile-correlation-is-associated-with-gene-expression">Repair profile correlation is associated with gene expression</h3>
<p>To test whether gene expression is a factor associated with the correlation of repair between these two highly-diverged species of primates, we examined the correlation strength and gene transcription levels.
We prepared RNA-seq data sets for mouse lemur fibroblasts and obtained RNA-seq datasets for human fibroblasts from a publicly available database (see methods).
The strength of the correlation between the two repair profiles was found to be correlated with transcription levels (Fig <a href="#fig:fig5">5</a>).
Highly expressed regions had better repair correlation between the two species (Figs <a href="#fig:S2">S2</a>;<a href="#fig:S3">S3</a>;<a href="#fig:S4">S4</a>;<a href="#fig:S5">S5</a>).
The data did not show a correlation between the identity levels and repair correlation between the two species, which suggests that gene expression levels rather than the degree of homology might be an important factor.</p>
<figure>
<img src="images/fig-5.png" alt="" /><figcaption>Figure 5
<!-- Repair rate consistency in correlation with gene expression. 
The transcription levels of the orthologous regions were divided into quartiles separately for human and lemur. 
Out of the quartiles, 10 bootstrapped subsamples were retrieved, and for those regions, repair correlation was analyzed. 
The correlation coefficients (R values) are represented on the y-axis; quartiles are on the x-axis. --></figcaption>
</figure>
<h3 id="the-same-cell-type-from-two-primates-have-more-similar-repair-profiles-relative-to-two-different-cell-types-from-the-same-organism">The same cell type from two primates have more similar repair profiles relative to two different cell types from the same organism</h3>
<p>We cannot dismiss the possibility that the correlation of repair profiles between two primates could simply be due to the heterogenous mappability of the genomes.
Such an association makes sense only when we measure the similarity of repair patterns relative to another dataset.
For this reason, we used another human cell type, GM12878 - B-Lymphocytes to ask whether the repair profile of human fibroblasts is closer to lemur fibroblasts or human lymphocytes.
We performed XR-seq for CPD as it yields relatively high genome-wide heterogeneity compared to (6-4)PP.
Although repair profiles between human GM12878 and NHF1 are also correlated (Fig <a href="#fig:fig6">6</a>A), this correlation is not as strong as the one between human and mouse lemur fibroblasts.
Additionally, human and mouse lemur fibroblasts clustered together in the PCA plot (Fig <a href="#fig:fig6">6</a>B).
Within fibroblasts, biological replicates grouped together.
More interestingly, (6-4)PP and CPD samples formed distinct groups, each of which has human and mouse lemur samples in the subclusters.
The two replicates of CPD XR-seq samples of human lymphocytes (GM12878) clustered as an outgroup.
When we applied PCA differentially to genes and intergenic regions, we observed similar clustering (Fig <a href="#fig:S6">S6</a>).
This result suggests that not only transcriptional but also epigenetic profiles between human and mouse lemur fibroblasts are similar to yield correlated TCR (genes, Fig <a href="#fig:S6">S6</a>A) and global repair (intergenic regions Fig <a href="#fig:S6">S6</a>B), respectively.
On the other hand, even though derived from the same organism, completely different human cell lines have a considerable variation with respect to repair patterns.
It is interesting to note that although TCR profiles between two primates are quite different from each other, they share high similarity in repair profiles.</p>
<figure>
<img src="images/fig-6.png" alt="" /><figcaption>Figure 6
<!-- Cell type-based consistency of repair profiles between two primates. 
A) Scatter plot of normalized repair levels between two human cell lines, GM12878 and NHF1, show correlation (R=0.65, p=0). 
B) Principle component analysis of 10 XR-seq samples.  --></figcaption>
</figure>
<h2 id="discussion">Discussion</h2>
<p>XR-seq methodology has become prominent in genome-wide profiling of nucleotide excision repair at single-nucleotide resolution.
This technique has been applied to a variety of model organisms, from humans to bacteria.
Although genome-level understanding of human repair behavior is relatively well established in recent years, the associations between closely related primates with respect to UV damage response and genome-wide repair distribution were entirely lacking.
With this pilot study, we established the differences and similarities between mouse lemur and human using a pair of similarly derived fibroblast cell lines.
First, we observed repair efficiency differences between the two genomes of the two major types of DNA damage induced by UV.
Like in humans, (6-4)PP was much more quickly completed compared to CPD in mouse lemur.
Both (6-4)PP and CPD repair efficiencies were slower in mouse lemur fibroblasts relative to human fibroblasts.
This observation is in agreement with the survival assays, which show that mouse lemur cell lines were more sensitive to UV, which contains both damage types we studied.
Although this comparison is limited to only two species, and further analysis would be required to make a more definitive conclusion, we postulate that the differences could result from differing repair enzyme activity between the species because of differences in fur which protects from UV exposure or due to the differences in diurnal and nocturnal lifestyles of the two species.</p>
<p>Interestingly, we also observed that TCR was relatively more efficient in mouse lemurs compared to humans, particularly for CPD repair.
TCR of (6-4)PP was not prominent, as was previously shown, and our immunoslot blot assays indicate that (6-4)PP repair is fast and therefore, repaired mostly by GR rather than TCR.
We observed similar TS and NTS repair signal for (6-4)PP in genic regions, also suggesting that GR is the primary subpathway to repair (6-4)PP.
A striking fact is an inverse correlation between TCR, which is measured by the TS/NTS repair signal, and overall repair efficiencies.
Mouse lemurs with more efficient TCR fall behind in total repair compared to the human.
A possible explanation for this observation is that global repair is the primary determinant of the repair efficiency.
XR-seq yields the entire nucleotide excision repair events with no distinction between TCR and GR.
For this reason, GR and TCR efficiencies derived through XR-seq are relative to each other.
Like the relativity in any RNA-seq, which cannot measure the total transcription levels, XR-seq also gives the relative distribution of the repair events.
Therefore, measured “TCR efficiency” actually corresponds to “GR deficiency” in the XR-seq profile.
In other words, when the damage is not efficiently recognized by the GR pathway, TCR takes over the recognition and removes DNA lesions in TS.
With this study, previously observed high TCR levels in <em>Arabidopsis thaliana</em> <span class="citation" data-cites="12APhIkm8">[<a href="#ref-12APhIkm8" role="doc-biblioref">13</a>]</span> and mouse <span class="citation" data-cites="AXdS7vVN">[<a href="#ref-AXdS7vVN" role="doc-biblioref">14</a>]</span> now also suggest relatively poor GR efficiency in those organisms.</p>
<p>Human evolution has been under investigation since the time of Charles Darwin.
Comparative analyses between humans and other animals have revealed both shared and human-specific features.
In the era of molecular biology, it has been possible to identify molecular features as they are shared between humans and their relatives.
Primates are an extremely diverse group comprised of species that range in body size from 30-80 grams (mouse lemurs) to 150kg (African apes).
Mouse lemurs stand out as the world’s smallest living primate.
Given detailed knowledge of primate phylogeny and divergence times <span class="citation" data-cites="QH4z4kb6">[<a href="#ref-QH4z4kb6" role="doc-biblioref">25</a>]</span>, we know that the most recent common ancestor (MRCA) of humans and mouse lemurs is MRCA of all living primate species and that this MRCA likely arose near the Cretaceous-Paleogene (K-Pg) boundary 65 mya.
Thus, we can reasonably conclude that features that are shared between humans and mouse lemurs are homologous and characteristic of the primate clade.</p>
<p>We have a limited knowledge of the epigenetic architecture of mouse lemurs as we far lack chromatin data.
Prior to this study, it was unknown whether repair profiles would be similar in the comparison of two primates species.
Our work revealed that human and mouse lemur fibroblasts are significantly similar with respect to their preference on which genomic region to repair first.
This bias is likely to be caused by the chromatin structure, as previously shown in other organisms <span class="citation" data-cites="hJRoVR8L">[<a href="#ref-hJRoVR8L" role="doc-biblioref">8</a>]</span>.
Damage distribution is known to be homogenous at large genomic scales <span class="citation" data-cites="10FU8PPzJ">[<a href="#ref-10FU8PPzJ" role="doc-biblioref">3</a>]</span>.
Therefore, it is safe to assume that repair preference is the major factor for XR-seq heterogeneity.
The repair profiles of human and mouse lemur fibroblasts are much more similar to each other than the ones between two distinct human cell lines.
Moreover, (6-4)PP and CPD repair profiles are more similar to themselves, suggesting similar repair dynamics between two organisms.
Further studies on mouse lemur epigenetics will possibly explain the most differentiated genomic repair levels in mouse lemurs.
Better understanding mouse lemur genomics, as well as other species within the strepsirrhine clade, will provide additional insight into human evolution, repair and mutagenesis.</p>
<h2 id="conclusions">Conclusions</h2>
<p>We demonstrated that genome-wide UV repair profiles between human and and mouse lemur are strongly correlated.
Our results shed light onto deep homologies of same tissue types in primates by suggesting similar epigenetic architecture between them.
We believe our results on varying repair rates between the two species will guide follow-up studies investigating the evolutionary effects of day time activity and UV avoidance on differential nucleotide excision repair rates.</p>
<h2 id="abbreviations">Abbreviations</h2>
<p>GR: Global repair</p>
<p>TCR: Transcription-coupled repair</p>
<p>RNAPII: RNA polymerase II</p>
<p>XR-seq: excision repair sequencing</p>
<p>CPD: cyclobutane pyrimidine dimer</p>
<p>(6-4)PP: 6-4 pyrimidine-pyrimidone photo products</p>
<p>NHF1: normal human fibroblasts 1</p>
<p>TS: transcribed strand</p>
<p>NTS: non-transcribed strand</p>
<p>RPKM: reads per kilobase per million mapped reads</p>
<p>GEO: Gene Expression Omnibus</p>
<h2 id="methods">Methods</h2>
<!-- ## Materials and Methods -->
<h3 id="cell-lines-and-reagents">Cell lines and reagents</h3>
<h4 id="establishment-of-the-immortalized-mouse-lemur-fibroblast-cell-line">Establishment of the immortalized mouse lemur fibroblast cell line</h4>
<p>Primary fibroblasts were established using methods outlined in Larsen and Harris et al <span class="citation" data-cites="1FNsl9JHL">[<a href="#ref-1FNsl9JHL" role="doc-biblioref">17</a>]</span>.
For the current study, we used an immortalized version of these primary mouse lemur fibroblasts.
This cell line was generated by transfecting passage six primary cells at approximately 60% confluency with 5 µg of human hTERT (hTERT) plasmid DNA per well of a 6-well plate using Lipofectamine® 2000 diluted in Opti-MEM® medium.
Twenty-four hours post-transfection, cells were placed under selection using G418 sulfate at a concentration of 300 µg/mL for two weeks.
Selected cells were then plated into a 96-well plate and remained under G418 sulfate maintenance at a concentration of 100 µg/mL to establish single cell colonies.
Approximately 2 weeks post-transfection, colonies were observed in 19 of the 96 wells and these cells underwent subsequent propagation and expansion.</p>
<h4 id="human-cell-lines">Human cell lines</h4>
<p>Telomerase-immortalized normal human fibroblasts (NHF1) were obtained from W.K. Kaufmann (University of North Carolina, Chapel Hill) as previously described <span class="citation" data-cites="6H2Giv1x OhKg9ZHE">[<a href="#ref-OhKg9ZHE" role="doc-biblioref">5</a>,<a href="#ref-6H2Giv1x" role="doc-biblioref">26</a>]</span>.
All cells were grown in Dulbecco’s modified Eagle’s medium supplemented with 10% fetal bovine serum at 37°C in a 5% CO2 humidified chamber.
Human lymphocyte GM12878 was purchased from Coriell.</p>
<h3 id="clonogenic-survival-assay">Clonogenic Survival Assay</h3>
<p>Cells were seeded in 10 cm plates at 300 cells/plate, grown for 16 hours, and then UV irradiated as follows.
After removing and reserving culture medium, cells were placed under a GE germicidal lamp that emits primarily 254-nm UV light (UV-C) at 1 J/m<sup>2</sup>sec connected to a digital timer to give the indicated dose.
Following irradiation, the culture medium was replaced, and the cells were incubated for 7 days.
Plates were then washed with phosphate-buffered saline (PBS), the cells were fixed for 10 min in 75% methanol/25% acetic acid, and stained with 0.5% crystal violet stain (J.T. Baker) in 25% methanol for 30 min.
The plates were then washed extensively in tap water to remove the excess stain.
After drying, images of the stained cells were taken on a Molecular Imager Chemi-Doc XRS+ system (Bio-Rad) for the presentation of representative, qualitative results.
Colonies were counted and the surviving fraction was calculated by dividing the number of colonies on treated plates by the number on the untreated plates.
Each condition was performed in triplicate and the experiment was repeated three independent times.</p>
<h3 id="immunoslot-blot-analysis">Immunoslot Blot Analysis</h3>
<p>The repair of UV photoproducts from genomic DNA was measured as described previously <span class="citation" data-cites="1Cn9oXJVt">[<a href="#ref-1Cn9oXJVt" role="doc-biblioref">27</a>]</span>.
Cells were UV irradiated, for a dose of 10J/m<sup>2</sup>.
Following irradiation, the culture medium was replaced, and the cells were incubated for the indicated periods of time.
Genomic DNA (250 ng), isolated with a QIAamp DNA Mini kit (Qiagen), was immobilized on a nitrocellulose membrane with a Bio-Dot SF Cell immunoslot blot apparatus (Bio-Rad) and heated for 90 min at 80 °C under vacuum.
Blots were blocked in 5% milk and probed with an anti-(6-4)PP antibody (Cosmo Bio 64M-2 cat#NM-DND-001) or anti-CPD antibody (Cosmo Bio TDM-2 cat#NM-DND-002) as indicated.
The secondary antibody was horseradish peroxidase-linked anti-mouse IgG from GE Healthcare (catalog no. NA931V) and the chemiluminescent signals were visualized with Clarity Western ECL Substrate (Bio-Rad) and using a Molecular Imager Chemi-Doc XRS+ system (Bio-Rad).
Following immunoblotting, the blots were re-blotted with anti-ssDNA antibody (Millipore MAB3034 clone 16-19) to ensure equal loading of DNA.
The experiment was repeated three times, and representative results are presented.</p>
<h3 id="detection-of-excised-oligonucleotide-products-of-nucleotide-excision-repair">Detection of Excised Oligonucleotide Products of Nucleotide Excision Repair</h3>
<p>Nucleotide excision repair activity was visualized, as previously described <span class="citation" data-cites="1GUFhcYbP">[<a href="#ref-1GUFhcYbP" role="doc-biblioref">19</a>]</span>.
Cells grown to ~80% confluency in 15-cm plates were harvested the indicated time after irradiation with 20 J/m<sup>2</sup> of UV.
The cells were pelleted by centrifugation, subjected to a modified Hirt procedure where cell pellets were resuspended in a 10× packed cell volume of lysis buffer (50 mM Tris-Cl, pH 8.0, 10 mM EDTA, 1.2% SDS and 100 μg/ml RNase A) and incubated for 15 min at room temperature.
Following the addition of a one-fourth volume of 5 M NaCl, the mixtures were gently mixed and incubated on ice for &gt;16 h.
After centrifugation at maximum speed (20,000 × g) for 1 h, the supernatants were gently transferred to new tubes and treated with 20 μg of proteinase K for 15 min at 55 °C, extracted with phenol/chloroform, and then precipitated with ethanol.
The pellet was washed with 500 μl of 70% ethanol and resuspended in 10 μl of buffer EB (10 mM Tris-Cl (pH8.5)).
The excised oligonucleotide products of nucleotide excision repair were purified from the remaining material with either anti-(6-4)PP or anti-CPD antibodies as follows: For each reaction, 5 μl of protein G Dynabeads (Invitrogen, catalog no. 10003D) slurry and 5 μl of anti-rabbit Dynabeads (Invitrogen, catalog no. 11203D) slurry were washed three times with 50 μl of wash buffer I (20 mM Tris-Cl (pH 8.0), 2 mM EDTA, 150 mM NaCl, 1% Triton X-100, and 0.1% SDS) and then incubated with 1 μl of rabbit anti-mouse IgG and 1 μl of anti-(6-4)PP or anti-CPD antibody in 20 μl of IP buffer (20 mM Tris-Cl (pH 8.0), 2 mM EDTA, 150 mM NaCl, 1% Triton X-100, and 0.5% sodium deoxycholate) for 3 h at 4 °C.
After incubation, beads were separated from the liquid with a magnet and then mixed with 100 μl of IP buffer and 10 μl of DNA.
The mixtures were rotated at 4 °C overnight.
The beads were then washed sequentially with 200 μl each of wash buffer I, wash buffer II (20 mM Tris-Cl (pH 8.0), 2 mM EDTA, 500 mM NaCl, 1% Triton X-100, and 0.1% SDS), wash buffer III (10 mM Tris-Cl (pH 8.0), 1 mM EDTA, 150 mM LiCl, 1% Nonidet P-40, and 1% sodium deoxycholate), wash buffer IV (100 mM Tris-Cl (pH 8.0), 1 mM EDTA, 500 mM LiCl, 1% Nonidet P-40, and 1% sodium deoxycholate) and finally twice with TE (10 mM Tris-Cl (pH 8.0) and 1 mM EDTA).
The oligonucleotides containing UV photoproducts were eluted by incubation with 100 μl of elution buffer (50 mM NaHCO3, 1% SDS, and 20 μg/ml glycogen) at 65 °C for 15 min.
The eluted DNA was then isolated by phenol/chloroform extraction and followed by ethanol precipitation.
The excised oligonucleotides were resuspended in 10 μl of water, and half of the DNA was 3′-end-labeled for 1 h at 37 °C in a 10-μl reaction containing 6 units of terminal deoxynucleotidyl transferase (New England Biolabs), 0.25 mM CoCl2, and 1 μCi of [α-32P]-3′-deoxyadenosine 5′-triphosphate (cordycepin 5′-triphosphate, Perkin Elmer Life Sciences) in 1× terminal deoxynucleotidyl transferase buffer (New England Biolabs).
2.5 fmol of a 50-nucleotide oligomer was included in all reactions as an internal control, and oligonucleotides of known length were resolved on all gels as size markers.
Following phenol-chloroform extraction and ethanol precipitation, the DNA was separated on urea-containing polyacrylamide gels, detected with a phosphorimager and was quantified using ImageQuant software (version 5.2, GE Healthcare).
The experiment was repeated three times.</p>
<h3 id="xr-seq-library-preparation">XR-seq library preparation</h3>
<p>Samples were processed for XR-seq, as previously described <span class="citation" data-cites="FND0ibVT">[<a href="#ref-FND0ibVT" role="doc-biblioref">28</a>]</span>.
Cells grown to ~80% confluency in 15-cm plates were harvested either 5 minutes or 1 h after irradiation with 20 J/m<sup>2</sup> of UV, depending on the damage, (6-4)PP or CPD respectively, to be analyzed.
Samples from four plates were pooled, lysed, and immunoprecipitated with anti-TFIIH antibodies p62 (Santa Cruz Biotechnology, sc292) and p89 (XPB, Santa Cruz Biotechnology, sc293), and then ligated to adapters and processed for sequencing.</p>
<h3 id="rna-seq-library-preparation">RNA-seq library preparation</h3>
<p>Total RNA was isolated from one 10-cm plate of exponentially growing NHF1 or Lemur cells using Trizol (Invitrogen) and the RNeasy Mini Kit (Qiagen) following the manufacturer’s instructions.
The library preparation and strand-specific paired-end sequencing (2 × 150 bp) on a HiSeq 4000 platform (Illumina) were performed by Novogene Co., Ltd.</p>
<h3 id="human-and-lemur-genome-alignment">Human and lemur genome alignment</h3>
<p>MUMmer version 3.23 <span class="citation" data-cites="19WxheWFA">[<a href="#ref-19WxheWFA" role="doc-biblioref">29</a>]</span> was used to align the grey mouse lemur genome (Mmur_3.0) to the human genome (hg19) with <code>nucmer</code> subprogram.
The alignments were filtered based on criteria taking length and identity into account.
Alignments were listed in “delta format” by default by Nucmer.
For every reference (human) - query (lemur) pair, we kept the alignments which form the longest mutually consistent set.
The filtering step was performed with <code>delta-filter</code> subprogram.
<code>show-coords</code> subprogram was used to display summary information such as position, percent identity and other features of each alignment, in Btab format with <code>-B</code> and <code>-rclo</code> arguments.</p>
<p>We used <code>generateBED.R</code> <span class="citation" data-cites="ZvXLvNIS">[<a href="#ref-ZvXLvNIS" role="doc-biblioref">30</a>]</span> to write homologous human-lemur regions in a bed file.
Due to low genome coverage in XR-seq data sets, short regions might introduce random repair values.
Since we were not interested in too short alignments, we filtered out alignments shorter than 400 bp while generating the bed file.
Additionally, alignments with at percent similarity lower than 80 were filtered out of the list.
The output was a BED file with each homologous region between two genomes.</p>
<p>This output file contained regions in either genome that align to multiple regions in the other genome. To remove such regions, we used, we used <code>bedtools intersect</code> <span class="citation" data-cites="1HWiAHnIw">[<a href="#ref-1HWiAHnIw" role="doc-biblioref">31</a>]</span> to intersect the file with itself.
The command line we used was: <code>bedtools intersect -wo -s -a humanOverlapsLemur_short_noFilter.bed -b humanOverlapsLemur_short_noFilter.bed &gt; human_intersect.txt</code>.
We compared this intersected output file to the <code>bed</code> file generated previously using <code>findDupAln.R</code> <span class="citation" data-cites="ZvXLvNIS">[<a href="#ref-ZvXLvNIS" role="doc-biblioref">30</a>]</span> to exclude regions in both genomes that align to multiple regions in other genome.
This resulted in a <code>bed</code> file of one-to-one homologous regions without multiple alignments for one region.</p>
<h3 id="xr-seq-analysis">XR-seq analysis</h3>
<p>We trimmed 3’ adapter sequences (TGGAATTCTCGGGTGCCAAGGAACTCCAGTNNNNNNACGATCTCGTATGCCGTCTTCTGCTTG) using Cutadapt <span class="citation" data-cites="1Fp9ui2nK">[<a href="#ref-1Fp9ui2nK" role="doc-biblioref">32</a>]</span>.
Bowtie2 version 2.3.4.1 <span class="citation" data-cites="PiS0h6Mu">[<a href="#ref-PiS0h6Mu" role="doc-biblioref">33</a>]</span> was used to align sequencing reads onto reference genomes with default parameters.
With a two-step conversion process, the <code>sam</code> format was first converted (with SAMtools <span class="citation" data-cites="IPc8cy7s">[<a href="#ref-IPc8cy7s" role="doc-biblioref">34</a>]</span>) to <code>bam</code> format, which was then converted to <code>bed</code> format with BEDTools <span class="citation" data-cites="1HWiAHnIw">[<a href="#ref-1HWiAHnIw" role="doc-biblioref">31</a>]</span>.
The resulting <code>bed</code> files were sorted and duplicate regions were removed.
Command line we used was:
<code>sort -u -k1,1 -k2,2n -k3,3n ${SAMPLE}_cutadapt.bed &gt;${SAMPLE}_cutadapt_sorted.bed</code></p>
<h3 id="rna-seq-analysis">RNA-seq analysis</h3>
<p>Sequencing reads were aligned to the reference genome using STAR Aligner 2.6.1a <span class="citation" data-cites="tTu8Ds9Z">[<a href="#ref-tTu8Ds9Z" role="doc-biblioref">35</a>]</span> with default parameters.
The aligned reads were converted to <code>bed</code> format with <code>bedtools bamtobed</code>.
The resulting <code>bed</code> files were sorted by coordinates and duplicate regions were removed.
Command-line we used was:
<code>sort -u -k1,1 -k2,2n -k3,3n ${SAMPLE}.bed &gt;${SAMPLE}_sorted.bed</code></p>
<h3 id="xr-seq-simulation">XR-seq simulation</h3>
<p>We evaluated the correlation of repair mechanisms with both simulated and real XR-seq datasets.
Simulated datasets were generated for the overlapping regions between the grey mouse lemur genome and the human genome using ART simulator <span class="citation" data-cites="znONJtTo">[<a href="#ref-znONJtTo" role="doc-biblioref">36</a>]</span>.
We used <code>bedtools getfasta</code> program with the <code>bed</code> file containing homologous regions between two organisms to generate a reference <code>fasta</code> file for ART simulator to produce synthetic reads.
By default, ART simulator produces a <code>fastq</code> file with flat nucleotide distribution.
For a better representation of XR-seq characteristics in the simulated dataset, we obtain nucleotide distribution frequencies of both species XR-seq reads.
We applied a scoring function where each nucleotide in the simulated read was scored based on the frequency of that nucleotide to be in that position in the actual XR-seq dataset and obtained a total score for each simulated read.
Accordingly, best scoring 10 million reads were selected for both species from the simulated dataset using <code>filter_syn.go</code> <span class="citation" data-cites="ZvXLvNIS">[<a href="#ref-ZvXLvNIS" role="doc-biblioref">30</a>]</span></p>
<h3 id="downstream-analysis">Downstream Analysis</h3>
<p>To calculate nucleotide frequencies of excised oligomers, we first converted aligned Xr-seq reads to <code>fasta</code> format using <code>bedtools getfasta</code>. Using <code>frequency.go</code> <span class="citation" data-cites="ZvXLvNIS">[<a href="#ref-ZvXLvNIS" role="doc-biblioref">30</a>]</span> we calculated the nucleotide frequency of predominant oligomer (26nt) for each primate and damage type.</p>
<p>Command line used to get excised oligomer lengths:
<code>awk '{print $3-$2}' ${SAMPLE}_cutadapt_sorted.bed | sort -k1,1n | uniq -c | sed 's/\s\s*/ /g' | awk '{print $2"\t"$1}'</code></p>
<p>Statistical analyses of immunoslot blot repair assays and clonogenic survival assay have been performed with Welch Two Sample t-test by using <code>t-test.R</code> <span class="citation" data-cites="ZvXLvNIS">[<a href="#ref-ZvXLvNIS" role="doc-biblioref">30</a>]</span>.</p>
<p>For both genomes, all annotated protein-coding genes were retrieved and genes that are closer to each other with less than 20kb were filtered out.
We used 5277 and 3366 number of genes for human and mouse lemur, respectively.
10 kb upstream and downstream of TSS and TES were divided into 100 bp windows.
XR-seq reads falling onto each bin were normalized to RPKM with <code>tcr.py</code> <span class="citation" data-cites="ZvXLvNIS">[<a href="#ref-ZvXLvNIS" role="doc-biblioref">30</a>]</span>
Then XR-seq RPKM values were normalized by the RPKM values derived by the mapped reads onto random genomic sites which are prepared using <code>bedtools shuffle</code> for protein-coding genes that are not closer to each other with less than 20kb.
Statistical significance of TCR profiles was estimated using Mann-Whitney U-test <span class="citation" data-cites="MyzxBJ57">[<a href="#ref-MyzxBJ57" role="doc-biblioref">37</a>]</span>.
Statistical analyses were performed using <code>mann-w.py</code> <span class="citation" data-cites="ZvXLvNIS">[<a href="#ref-ZvXLvNIS" role="doc-biblioref">30</a>]</span> with SciPy <span class="citation" data-cites="8Miti2Gz">[<a href="#ref-8Miti2Gz" role="doc-biblioref">38</a>]</span>.</p>
<p>We used <code>regions_rpkm.go</code> <span class="citation" data-cites="ZvXLvNIS">[<a href="#ref-ZvXLvNIS" role="doc-biblioref">30</a>]</span> to calculate RPKM values of XR-seq, RNA-seq, and simulated XR-seq for each overlapping region.
Regions with no XR-seq or RNA-seq reads were filtered out.</p>
<p>Circos plot for comprehensive summarization of all findings has been generated with <code>circosPlot.R</code> <span class="citation" data-cites="ZvXLvNIS">[<a href="#ref-ZvXLvNIS" role="doc-biblioref">30</a>]</span> by using R package <code>circlize</code>.<span class="citation" data-cites="uO1r94Ec">[<a href="#ref-uO1r94Ec" role="doc-biblioref">39</a>]</span></p>
<p>PCA plot has been generated from read counts with <code>plotPCA.R</code> <span class="citation" data-cites="ZvXLvNIS">[<a href="#ref-ZvXLvNIS" role="doc-biblioref">30</a>]</span>, while applying variance stabilizing transformation as described in DESeq2 <span class="citation" data-cites="w9AOzBMw">[<a href="#ref-w9AOzBMw" role="doc-biblioref">40</a>]</span> vignette.</p>
<p>Plots were prepared using <code>plotly</code> and <code>ggplot2</code><span class="citation" data-cites="7F5oVreR">[<a href="#ref-7F5oVreR" role="doc-biblioref">41</a>]</span>.
The manuscript was written using Manubot <span class="citation" data-cites="YuJbg3zO">[<a href="#ref-YuJbg3zO" role="doc-biblioref">42</a>]</span>.</p>
<!-- ## Data Access

All raw and processed sequencing data generated in this study have been submitted to the NCBI Gene Expression Omnibus (GEO; https://www.ncbi.nlm.nih.gov/geo/) under accession number GSE145883. -->
<h2 id="declarations">Declarations</h2>
<h3 id="availability-of-data-and-materials">Availability of data and materials</h3>
<p>All raw and processed sequencing data generated in this study have been submitted to the NCBI Gene Expression Omnibus (GEO; https://www.ncbi.nlm.nih.gov/geo/) under accession number GSE145883.
The code repository for the data analyses can be accessed at https://github.com/CompGenomeLab/lemurRepair.</p>
<h3 id="funding">Funding</h3>
<p>The design of the repair assays is supported by the National Institutes of Health [GM118102, ES027255 to A.S.].
Cell line establishment was supported by the National Science Foundation [DEB-1354610 to A.Y.D].
Conceptualization and computational analysis were supported by European Molecular Biology Organization Installation Grant (to O.A.)</p>
<h3 id="competing-interests">Competing interests</h3>
<!-- ## CONFLICT OF INTEREST -->
<p>The authors declare that there is no competing interests regarding the publication of this article.</p>
<h2 id="acknowledgements">Acknowledgements</h2>
<!-- ## ACKNOWLEDGEMENT -->
<p>We would like to thank Dr. Beth A Sullivan for the construct to make the TERT-immortalized lines.
A.Y.D is supported by the John Simon Guggenheim Foundation, and the Alexander von Humboldt Foundation.
O.A. is supported by TUBITAK 2232 International Fellowship for Outstanding Researchers Program and Young Scientist Grant (BAGEP) awarded by Science Academy, Turkey.</p>
<p><em>Authors’ contributions:</em> OA, AS, LL-B and ADY conceived of the project and designed the experiments. Mouse lemur tissue was acquired by ADY and PAL, cell lines were prepared by AB providing reagents for immortalization (hTERT construct). LL-B performed experimental assays, XR-seq and RNA-seq experiments. VOK, UA and ZK performed computational analyses. OA interpreted the data and wrote the first draft. All authors have read, revised and approved the final manuscript.</p>
<!-- ## Disclosure Declaration -->
<!-- ## Competing interests -->
<!-- The authors declare that there is no conflict of interest regarding the publication of this article. -->
<h2 id="references">References</h2>
<!-- Explicitly insert bibliography here -->
<div id="refs" class="references hanging-indent" role="doc-bibliography">
<div id="ref-bYJTOmGw">
<p>1. <strong>Molecular mechanisms and genomic maps of DNA excision repair in Escherichia coli and humans</strong> <br />
Jinchuan Hu, Christopher P. Selby, Sheera Adar, Ogun Adebali, Aziz Sancar<br />
<em>Journal of Biological Chemistry</em> (2017-09) <a href="https://doi.org/gftwfh">https://doi.org/gftwfh</a> <br />
DOI: <a href="https://doi.org/10.1074/jbc.r117.807453">10.1074/jbc.r117.807453</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28798238">28798238</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5612094">PMC5612094</a></p>
</div>
<div id="ref-cZtF6zIe">
<p>2. <strong>Methodologies for detecting environmentally induced DNA damage and repair</strong> <br />
Wentao Li, Aziz Sancar<br />
<em>Environmental and Molecular Mutagenesis</em> (2020-02-29) <a href="https://doi.org/ggmr42">https://doi.org/ggmr42</a> <br />
DOI: <a href="https://doi.org/10.1002/em.22365">10.1002/em.22365</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32083352">32083352</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7442611">PMC7442611</a></p>
</div>
<div id="ref-10FU8PPzJ">
<p>3. <strong>Dynamic maps of UV damage formation and repair for the human genome</strong> <br />
Jinchuan Hu, Ogun Adebali, Sheera Adar, Aziz Sancar<br />
<em>Proceedings of the National Academy of Sciences</em> (2017-06-12) <a href="https://doi.org/ggdfws">https://doi.org/ggdfws</a> <br />
DOI: <a href="https://doi.org/10.1073/pnas.1706522114">10.1073/pnas.1706522114</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28607063">28607063</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5495279">PMC5495279</a></p>
</div>
<div id="ref-zxIvoFqn">
<p>4. <strong>ETS transcription factors induce a unique UV damage signature that drives recurrent mutagenesis in melanoma</strong> <br />
Peng Mao, Alexander J. Brown, Shingo Esaki, Svetlana Lockwood, Gregory M. K. Poon, Michael J. Smerdon, Steven A. Roberts, John J. Wyrick<br />
<em>Nature Communications</em> (2018-07-06) <a href="https://doi.org/gdxcvk">https://doi.org/gdxcvk</a> <br />
DOI: <a href="https://doi.org/10.1038/s41467-018-05064-0">10.1038/s41467-018-05064-0</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29980679">29980679</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6035183">PMC6035183</a></p>
</div>
<div id="ref-OhKg9ZHE">
<p>5. <strong>Genome-wide analysis of human global and transcription-coupled excision repair of UV damage at single-nucleotide resolution</strong> <br />
Jinchuan Hu, Sheera Adar, Christopher P. Selby, Jason D. Lieb, Aziz Sancar<br />
<em>Genes &amp; Development</em> (2015-05-01) <a href="https://doi.org/f69xk2">https://doi.org/f69xk2</a> <br />
DOI: <a href="https://doi.org/10.1101/gad.261271.115">10.1101/gad.261271.115</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25934506">25934506</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4421983">PMC4421983</a></p>
</div>
<div id="ref-ZAV7FWhL">
<p>6. <strong>Cisplatin DNA damage and repair maps of the human genome at single-nucleotide resolution</strong> <br />
Jinchuan Hu, Jason D. Lieb, Aziz Sancar, Sheera Adar<br />
<em>Proceedings of the National Academy of Sciences</em> (2016-10-11) <a href="https://doi.org/f86sdj">https://doi.org/f86sdj</a> <br />
DOI: <a href="https://doi.org/10.1073/pnas.1614430113">10.1073/pnas.1614430113</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27688757">27688757</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5068337">PMC5068337</a></p>
</div>
<div id="ref-4WTs70ml">
<p>7. <strong>Genome-wide kinetics of DNA excision repair in relation to chromatin state and mutagenesis</strong> <br />
Sheera Adar, Jinchuan Hu, Jason D. Lieb, Aziz Sancar<br />
<em>Proceedings of the National Academy of Sciences</em> (2016-04-12) <a href="https://doi.org/f8g4ps">https://doi.org/f8g4ps</a> <br />
DOI: <a href="https://doi.org/10.1073/pnas.1603388113">10.1073/pnas.1603388113</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27036006">27036006</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4839430">PMC4839430</a></p>
</div>
<div id="ref-hJRoVR8L">
<p>8. <strong>Differential damage and repair of DNA-adducts induced by anti-cancer drug cisplatin across mouse organs</strong> <br />
Askar Yimit, Ogun Adebali, Aziz Sancar, Yuchao Jiang<br />
<em>Nature Communications</em> (2019-01-18) <a href="https://doi.org/ggh7vb">https://doi.org/ggh7vb</a> <br />
DOI: <a href="https://doi.org/10.1038/s41467-019-08290-2">10.1038/s41467-019-08290-2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30659176">30659176</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6338751">PMC6338751</a></p>
</div>
<div id="ref-18edpunap">
<p>9. <strong>Genome-wide transcription-coupled repair in <em>Escherichia coli</em> is mediated by the Mfd translocase</strong> <br />
Ogun Adebali, Yi-Ying Chiou, Jinchuan Hu, Aziz Sancar, Christopher P. Selby<br />
<em>Proceedings of the National Academy of Sciences</em> (2017-03-14) <a href="https://doi.org/f9v7nt">https://doi.org/f9v7nt</a> <br />
DOI: <a href="https://doi.org/10.1073/pnas.1700230114">10.1073/pnas.1700230114</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28167766">28167766</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5358382">PMC5358382</a></p>
</div>
<div id="ref-1190rUeCp">
<p>10. <strong>Mfd translocase is necessary and sufficient for transcription-coupled repair in Escherichia coli</strong> <br />
Ogun Adebali, Aziz Sancar, Christopher P. Selby<br />
<em>Journal of Biological Chemistry</em> (2017-11) <a href="https://doi.org/ggdfwv">https://doi.org/ggdfwv</a> <br />
DOI: <a href="https://doi.org/10.1074/jbc.c117.818807">10.1074/jbc.c117.818807</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28986449">28986449</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5682952">PMC5682952</a></p>
</div>
<div id="ref-OSVE68co">
<p>11. <strong>Single-nucleotide resolution dynamic repair maps of UV damage in <em>Saccharomyces cerevisiae</em> genome</strong> <br />
Wentao Li, Ogun Adebali, Yanyan Yang, Christopher P. Selby, Aziz Sancar<br />
<em>Proceedings of the National Academy of Sciences</em> (2018-04-10) <a href="https://doi.org/gdc9gh">https://doi.org/gdc9gh</a> <br />
DOI: <a href="https://doi.org/10.1073/pnas.1801687115">10.1073/pnas.1801687115</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29581276">29581276</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5899493">PMC5899493</a></p>
</div>
<div id="ref-iyn6rN7q">
<p>12. <strong>Drosophila, which lacks canonical transcription-coupled repair proteins, performs transcription-coupled repair</strong> <br />
Nazli Deger, Yanyan Yang, Laura A. Lindsey-Boltz, Aziz Sancar, Christopher P. Selby<br />
<em>Journal of Biological Chemistry</em> (2019-11) <a href="https://doi.org/ggdfwt">https://doi.org/ggdfwt</a> <br />
DOI: <a href="https://doi.org/10.1074/jbc.ac119.011448">10.1074/jbc.ac119.011448</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31624146">31624146</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6885609">PMC6885609</a></p>
</div>
<div id="ref-12APhIkm8">
<p>13. <strong>Genome-wide excision repair in Arabidopsis is coupled to transcription and reflects circadian gene expression patterns</strong> <br />
Onur Oztas, Christopher P. Selby, Aziz Sancar, Ogun Adebali<br />
<em>Nature Communications</em> (2018-04-17) <a href="https://doi.org/gdd6q4">https://doi.org/gdd6q4</a> <br />
DOI: <a href="https://doi.org/10.1038/s41467-018-03922-5">10.1038/s41467-018-03922-5</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29666379">29666379</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5904149">PMC5904149</a></p>
</div>
<div id="ref-AXdS7vVN">
<p>14. <strong>Cisplatin-DNA adduct repair of transcribed genes is controlled by two circadian programs in mouse tissues</strong> <br />
Yanyan Yang, Ogun Adebali, Gang Wu, Christopher P. Selby, Yi-Ying Chiou, Naim Rashid, Jinchuan Hu, John B. Hogenesch, Aziz Sancar<br />
<em>Proceedings of the National Academy of Sciences</em> (2018-05-22) <a href="https://doi.org/gdk8bg">https://doi.org/gdk8bg</a> <br />
DOI: <a href="https://doi.org/10.1073/pnas.1804493115">10.1073/pnas.1804493115</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29735688">29735688</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6003508">PMC6003508</a></p>
</div>
<div id="ref-CoujlGZF">
<p>15. <strong>Unpredictable environments, opportunistic responses: Reproduction and population turnover in two wild mouse lemur species ( <em>Microcebus rufus</em> and <em>M. griseorufus</em> ) from eastern and western Madagascar</strong> <br />
Marina B. Blanco, Emilienne Rasoazanabary, Laurie R. Godfrey<br />
<em>American Journal of Primatology</em> (2015-09) <a href="https://doi.org/f7n2hd">https://doi.org/f7n2hd</a> <br />
DOI: <a href="https://doi.org/10.1002/ajp.22423">10.1002/ajp.22423</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26031477">26031477</a></p>
</div>
<div id="ref-fKeXK6h7">
<p>16. <strong>The Mouse Lemur, a Genetic Model Organism for Primate Biology, Behavior, and Health</strong> <br />
Camille Ezran, Caitlin J Karanewsky, Jozeph L Pendleton, Alex Sholtz, Maya R Krasnow, Jason Willick, Andriamahery Razafindrakoto, Sarah Zohdy, Megan A Albertelli, Mark A Krasnow<br />
<em>Genetics</em> (2017-06-01) <a href="https://doi.org/gbh929">https://doi.org/gbh929</a> <br />
DOI: <a href="https://doi.org/10.1534/genetics.116.199448">10.1534/genetics.116.199448</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28592502">28592502</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5499178">PMC5499178</a></p>
</div>
<div id="ref-1FNsl9JHL">
<p>17. <strong>Hybrid de novo genome assembly and centromere characterization of the gray mouse lemur (Microcebus murinus)</strong> <br />
Peter A. Larsen, R. Alan Harris, Yue Liu, Shwetha C. Murali, C. Ryan Campbell, Adam D. Brown, Beth A. Sullivan, Jennifer Shelton, Susan J. Brown, Muthuswamy Raveendran, … Kim C. Worley<br />
<em>BMC Biology</em> (2017-11-16) <a href="https://doi.org/ggdf3h">https://doi.org/ggdf3h</a> <br />
DOI: <a href="https://doi.org/10.1186/s12915-017-0439-6">10.1186/s12915-017-0439-6</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29145861">29145861</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5689209">PMC5689209</a></p>
</div>
<div id="ref-1Gbi7LeDk">
<p>18. <strong>Primate comparative genomics: lemur biology and evolution</strong> <br />
Julie E. Horvath, Huntington F. Willard<br />
<em>Trends in Genetics</em> (2007-04) <a href="https://doi.org/bwv8j4">https://doi.org/bwv8j4</a> <br />
DOI: <a href="https://doi.org/10.1016/j.tig.2007.02.007">10.1016/j.tig.2007.02.007</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17331617">17331617</a></p>
</div>
<div id="ref-1GUFhcYbP">
<p>19. <strong>Nucleotide Excision Repair in Human Cells</strong> <br />
Jinchuan Hu, Jun-Hyuk Choi, Shobhan Gaddameedhi, Michael G. Kemp, Joyce T. Reardon, Aziz Sancar<br />
<em>Journal of Biological Chemistry</em> (2013-07) <a href="https://doi.org/f46nq2">https://doi.org/f46nq2</a> <br />
DOI: <a href="https://doi.org/10.1074/jbc.m113.482257">10.1074/jbc.m113.482257</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23749995">23749995</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3774362">PMC3774362</a></p>
</div>
<div id="ref-RCFCwjid">
<p>20. <strong>Human nucleotide excision nuclease removes thymine dimers from DNA by incising the 22nd phosphodiester bond 5’ and the 6th phosphodiester bond 3’ to the photodimer.</strong> <br />
J. C. Huang, D. L. Svoboda, J. T. Reardon, A. Sancar<br />
<em>Proceedings of the National Academy of Sciences</em> (1992-04-15) <a href="https://doi.org/c8cz9k">https://doi.org/c8cz9k</a> <br />
DOI: <a href="https://doi.org/10.1073/pnas.89.8.3664">10.1073/pnas.89.8.3664</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/1314396">1314396</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC48929">PMC48929</a></p>
</div>
<div id="ref-DObZlup3">
<p>21. <strong>A novel repair enzyme: UVRABC excision nuclease of Escherichia coli cuts a DNA strand on both sides of the damaged region.</strong> <br />
A Sancar, WD Rupp<br />
<em>Cell</em> (1983-05) <a href="https://www.ncbi.nlm.nih.gov/pubmed/6380755">https://www.ncbi.nlm.nih.gov/pubmed/6380755</a> <br />
DOI: <a href="https://doi.org/10.1016/0092-8674(83)90354-9">10.1016/0092-8674(83)90354-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/6380755">6380755</a></p>
</div>
<div id="ref-29W7k2UV">
<p>22. <strong>Transcription-coupled DNA repair: two decades of progress and surprises</strong> <br />
Philip C. Hanawalt, Graciela Spivak<br />
<em>Nature Reviews Molecular Cell Biology</em> (2008-12) <a href="https://doi.org/brt6w7">https://doi.org/brt6w7</a> <br />
DOI: <a href="https://doi.org/10.1038/nrm2549">10.1038/nrm2549</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19023283">19023283</a></p>
</div>
<div id="ref-NkhwUBAb">
<p>23. <strong>Induction of the Escherichia coli lactose operon selectively increases repair of its transcribed DNA strand</strong> <br />
Isabel Mellon, Philip C. Hanawalt<br />
<em>Nature</em> (1989-11) <a href="https://doi.org/c9gdjm">https://doi.org/c9gdjm</a> <br />
DOI: <a href="https://doi.org/10.1038/342095a0">10.1038/342095a0</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/2554145">2554145</a></p>
</div>
<div id="ref-LyGwcDX7">
<p>24. <strong>Fluorescently-labelled CPD and 6-4PP photolyases: new tools for live-cell DNA damage quantification and laser-assisted repair</strong> <br />
Barbara Steurer, Yasemin Turkyilmaz, Marvin van Toorn, Wessel van Leeuwen, Paula Escudero-Ferruz, Jurgen A Marteijn<br />
<em>Nucleic Acids Research</em> (2019-04-23) <a href="https://doi.org/ggkszp">https://doi.org/ggkszp</a> <br />
DOI: <a href="https://doi.org/10.1093/nar/gkz035">10.1093/nar/gkz035</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30698791">30698791</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6468286">PMC6468286</a></p>
</div>
<div id="ref-QH4z4kb6">
<p>25. <strong>Using Phylogenomic Data to Explore the Effects of Relaxed Clocks and Calibration Strategies on Divergence Time Estimation: Primates as a Test Case</strong> <br />
Mario Dos Reis, Gregg F Gunnell, Jose Barba-Montoya, Alex Wilkins, Ziheng Yang, Anne D Yoder<br />
<em>Systematic Biology</em> (2018-07) <a href="https://doi.org/gct4ft">https://doi.org/gct4ft</a> <br />
DOI: <a href="https://doi.org/10.1093/sysbio/syy001">10.1093/sysbio/syy001</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29342307">29342307</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6005039">PMC6005039</a></p>
</div>
<div id="ref-6H2Giv1x">
<p>26. <strong>An ATR- and Chk1-Dependent S Checkpoint Inhibits Replicon Initiation following UVC-Induced DNA Damage</strong> <br />
Timothy P. Heffernan, Dennis A. Simpson, Alexandra R. Frank, Alexandra N. Heinloth, Richard S. Paules, Marila Cordeiro-Stone, William K. Kaufmann<br />
<em>Molecular and Cellular Biology</em> (2002-12-15) <a href="https://doi.org/d5q4pz">https://doi.org/d5q4pz</a> <br />
DOI: <a href="https://doi.org/10.1128/mcb.22.24.8552-8561.2002">10.1128/mcb.22.24.8552-8561.2002</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12446774">12446774</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC139882">PMC139882</a></p>
</div>
<div id="ref-1Cn9oXJVt">
<p>27. <strong>Similar Nucleotide Excision Repair Capacity in Melanocytes and Melanoma Cells</strong> <br />
Shobhan Gaddameedhi, Michael G. Kemp, Joyce T. Reardon, Janiel M. Shields, Stephanie L. Smith-Roe, William K. Kaufmann, Aziz Sancar<br />
<em>Cancer Research</em> (2010-06-15) <a href="https://doi.org/cv5w99">https://doi.org/cv5w99</a> <br />
DOI: <a href="https://doi.org/10.1158/0008-5472.can-10-0095">10.1158/0008-5472.can-10-0095</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20501836">20501836</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2891231">PMC2891231</a></p>
</div>
<div id="ref-FND0ibVT">
<p>28. <strong>Genome-wide mapping of nucleotide excision repair with XR-seq</strong> <br />
Jinchuan Hu, Wentao Li, Ogun Adebali, Yanyan Yang, Onur Oztas, Christopher P. Selby, Aziz Sancar<br />
<em>Nature Protocols</em> (2018-12-14) <a href="https://doi.org/ggkvth">https://doi.org/ggkvth</a> <br />
DOI: <a href="https://doi.org/10.1038/s41596-018-0093-7">10.1038/s41596-018-0093-7</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30552409">30552409</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6429938">PMC6429938</a></p>
</div>
<div id="ref-19WxheWFA">
<p>29. <strong>Versatile and open software for comparing large genomes.</strong> <br />
Stefan Kurtz, Adam Phillippy, Arthur L Delcher, Michael Smoot, Martin Shumway, Corina Antonescu, Steven L Salzberg<br />
<em>Genome biology</em> (2004-01-30) <a href="https://www.ncbi.nlm.nih.gov/pubmed/14759262">https://www.ncbi.nlm.nih.gov/pubmed/14759262</a> <br />
DOI: <a href="https://doi.org/10.1186/gb-2004-5-2-r12">10.1186/gb-2004-5-2-r12</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14759262">14759262</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC395750">PMC395750</a></p>
</div>
<div id="ref-ZvXLvNIS">
<p>30. <strong>compGenomeLab/lemurRepair GitHub repository</strong> <br />
Umit Akkose, Veysel Ogulcan Kaya, Zeynep Karagoz, Ogun Adebali<br />
<em>GitHub</em> (2019) <a href="https://github.com/compGenomeLab/lemurRepair">https://github.com/compGenomeLab/lemurRepair</a></p>
</div>
<div id="ref-1HWiAHnIw">
<p>31. <strong>BEDTools: a flexible suite of utilities for comparing genomic features</strong> <br />
Aaron R. Quinlan, Ira M. Hall<br />
<em>Bioinformatics</em> (2010-03-15) <a href="https://doi.org/cmrms3">https://doi.org/cmrms3</a> <br />
DOI: <a href="https://doi.org/10.1093/bioinformatics/btq033">10.1093/bioinformatics/btq033</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20110278">20110278</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2832824">PMC2832824</a></p>
</div>
<div id="ref-1Fp9ui2nK">
<p>32. <strong>Cutadapt removes adapter sequences from high-throughput sequencing reads</strong> <br />
Marcel Martin<br />
<em>EMBnet.journal</em> (2011-05-02) <a href="https://doi.org/gdh7xt">https://doi.org/gdh7xt</a> <br />
DOI: <a href="https://doi.org/10.14806/ej.17.1.200">10.14806/ej.17.1.200</a></p>
</div>
<div id="ref-PiS0h6Mu">
<p>33. <strong>Fast gapped-read alignment with Bowtie 2</strong> <br />
Ben Langmead, Steven L Salzberg<br />
<em>Nature Methods</em> (2012-03-04) <a href="https://doi.org/gd2xzn">https://doi.org/gd2xzn</a> <br />
DOI: <a href="https://doi.org/10.1038/nmeth.1923">10.1038/nmeth.1923</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22388286">22388286</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3322381">PMC3322381</a></p>
</div>
<div id="ref-IPc8cy7s">
<p>34. <strong>The Sequence Alignment/Map format and SAMtools</strong> <br />
H. Li, B. Handsaker, A. Wysoker, T. Fennell, J. Ruan, N. Homer, G. Marth, G. Abecasis, R. Durbin, 1000 Genome Project Data Processing Subgroup<br />
<em>Bioinformatics</em> (2009-06-08) <a href="https://doi.org/ff6426">https://doi.org/ff6426</a> <br />
DOI: <a href="https://doi.org/10.1093/bioinformatics/btp352">10.1093/bioinformatics/btp352</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19505943">19505943</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2723002">PMC2723002</a></p>
</div>
<div id="ref-tTu8Ds9Z">
<p>35. <strong>STAR: ultrafast universal RNA-seq aligner</strong> <br />
Alexander Dobin, Carrie A. Davis, Felix Schlesinger, Jorg Drenkow, Chris Zaleski, Sonali Jha, Philippe Batut, Mark Chaisson, Thomas R. Gingeras<br />
<em>Bioinformatics</em> (2013-01) <a href="https://doi.org/f4h523">https://doi.org/f4h523</a> <br />
DOI: <a href="https://doi.org/10.1093/bioinformatics/bts635">10.1093/bioinformatics/bts635</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23104886">23104886</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3530905">PMC3530905</a></p>
</div>
<div id="ref-znONJtTo">
<p>36. <strong>ART: a next-generation sequencing read simulator</strong> <br />
Weichun Huang, Leping Li, Jason R. Myers, Gabor T. Marth<br />
<em>Bioinformatics</em> (2012-02-15) <a href="https://doi.org/fzf84c">https://doi.org/fzf84c</a> <br />
DOI: <a href="https://doi.org/10.1093/bioinformatics/btr708">10.1093/bioinformatics/btr708</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22199392">22199392</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3278762">PMC3278762</a></p>
</div>
<div id="ref-MyzxBJ57">
<p>37. <strong>On a Test of Whether one of Two Random Variables is Stochastically Larger than the Other</strong> <br />
H. B. Mann, D. R. Whitney<br />
<em>The Annals of Mathematical Statistics</em> (1947-03) <a href="https://doi.org/fgp7wb">https://doi.org/fgp7wb</a> <br />
DOI: <a href="https://doi.org/10.1214/aoms/1177730491">10.1214/aoms/1177730491</a></p>
</div>
<div id="ref-8Miti2Gz">
<p>38. <strong>SciPy 1.0: fundamental algorithms for scientific computing in Python</strong> <br />
Pauli Virtanen, Ralf Gommers, Travis E. Oliphant, Matt Haberland, Tyler Reddy, David Cournapeau, Evgeni Burovski, Pearu Peterson, Warren Weckesser, Jonathan Bright, … SciPy 1.0 Contributors<br />
<em>Nature Methods</em> (2020-02-03) <a href="https://doi.org/ggj45f">https://doi.org/ggj45f</a> <br />
DOI: <a href="https://doi.org/10.1038/s41592-019-0686-2">10.1038/s41592-019-0686-2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32015543">32015543</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7056644">PMC7056644</a></p>
</div>
<div id="ref-uO1r94Ec">
<p>39. <strong>circlize implements and enhances circular visualization in R</strong> <br />
Zuguang Gu, Lei Gu, Roland Eils, Matthias Schlesner, Benedikt Brors<br />
<em>Bioinformatics</em> (2014-10) <a href="https://doi.org/f6ktws">https://doi.org/f6ktws</a> <br />
DOI: <a href="https://doi.org/10.1093/bioinformatics/btu393">10.1093/bioinformatics/btu393</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24930139">24930139</a></p>
</div>
<div id="ref-w9AOzBMw">
<p>40. <strong>Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2</strong> <br />
Michael I Love, Wolfgang Huber, Simon Anders<br />
<em>Genome Biology</em> (2014-12-05) <a href="https://doi.org/gd3zvn">https://doi.org/gd3zvn</a> <br />
DOI: <a href="https://doi.org/10.1186/s13059-014-0550-8">10.1186/s13059-014-0550-8</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25516281">25516281</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4302049">PMC4302049</a></p>
</div>
<div id="ref-7F5oVreR">
<p>41. <strong>ggplot2: Elegant Graphics for Data Analysis (2nd ed.)</strong> <br />
Randle Aaron M. Villanueva, Zhuo Job Chen<br />
<em>Measurement: Interdisciplinary Research and Perspectives</em> (2019-08-27) <a href="https://doi.org/ggqrmf">https://doi.org/ggqrmf</a> <br />
DOI: <a href="https://doi.org/10.1080/15366367.2019.1565254">10.1080/15366367.2019.1565254</a></p>
</div>
<div id="ref-YuJbg3zO">
<p>42. <strong>Open collaborative writing with Manubot</strong> <br />
Daniel S. Himmelstein, Vincent Rubinetti, David R. Slochower, Dongbo Hu, Venkat S. Malladi, Casey S. Greene, Anthony Gitter<br />
<em>PLOS Computational Biology</em> (2019-06-24) <a href="https://doi.org/c7np">https://doi.org/c7np</a> <br />
DOI: <a href="https://doi.org/10.1371/journal.pcbi.1007128">10.1371/journal.pcbi.1007128</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31233491">31233491</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6611653">PMC6611653</a></p>
</div>
</div>
<h2 id="figure-legends">Figure legends</h2>
<div id="fig:fig1" class="fignos">
<figure>
<img src="images/fig-0.png" class="white" alt="" /><figcaption><span>Figure 1:</span> Characteristics of excised oligomers upon repair of UV-induced damages.
<em>(A)</em> In vivo excision assay for (6-4)PP and CPD are shown.
<em>(B)</em> XR-seq read length distribution of excised oligomers for CPD (top) and (6-4)PP (bottom). Human and mouse lemur are shown in blue and red, respectively.
<em>(C)</em> Nucleotide frequency in the predominant oligomer (26nt) for each primate and damage type. Representative data from replicate 1.</figcaption>
</figure>
</div>
<div id="fig:fig2" class="fignos">
<figure>
<img src="images/fig-0.png" class="white" alt="" /><figcaption><span>Figure 2:</span> UV sensitivity and repair rates in human and lemur fibroblasts.
<em>(A)</em> Clonogenic survival assay (left) and the quantified results (right).
<em>(B)</em> Immunoslot blot repair assays to examine the repair kinetics of UV-induced DNA lesions for both organisms and each damage type (left).
Remaining damage levels at each time point were quantified and plotted (right).
All experiments were repeated three times, and graphed data are presented as (mean+/-SD).
Tests (t-test) performed (H<sub>0</sub>:μ<sub>human</sub>=μ<sub>lemur</sub>) for each dosage (A) and time points (B).
* P ≤ 0.05; ** P ≤ 0.01; *** P ≤ 0.001; rest P &gt; 0.05.</figcaption>
</figure>
</div>
<div id="fig:fig3" class="fignos">
<figure>
<img src="images/fig-0.png" class="white" alt="" /><figcaption><span>Figure 3:</span> Transcription-coupled repair in mouse lemur and human fibroblasts.
Average profiles of CPD XR-seq (top) and (6-4)PP XR-seq (bottom) over 5277 and 3366 annotated genes are plotted for human (left) and mouse lemur (right), respectively.
Transcription start sites (TSS) and transcription end sites (TES) were retrieved from GTF (Gene Transfer Format) files for the two genomes.
Transcribed (red) and nontranscribed (blue) strand repair are shown in the downstream and upstream of TSS and TES, respectively.
10 kb upstream and downstream of TSS and TES were divided into 100 bp windows.
XR-seq reads aligned to each bin were normalized to RPKM (reads per kilobase per million mapped reads).
The XR-seq RPKM values were normalized by the shuffled RPKM values derived by mapped reads aligned to random genomic sites.
Only replicate 1 is shown. TS/NTS median values for TSS downstream for CPD are 1.76 and 2.31 for human and mouse lemur, respectively.</figcaption>
</figure>
</div>
<div id="fig:fig4" class="fignos">
<figure>
<img src="images/fig-0.png" class="white" alt="" /><figcaption><span>Figure 4:</span> Comparative analysis methodology and correlated repair rates between human and mouse lemur.
A) Approach to identify orthologous regions between human and mouse lemur (see methods for details).
B) Scatter plots showing the normalized repair levels between two organisms.
Relative repair was calculated by normalizing the repair signal by the simulated XR-seq reads to eliminate the sequence context bias.
C) Mapped orthologous regions with repair and transcription profiles mapped at left and right outer rings, respectively.
Inner circle represents the human (left) and mouse lemur (right) chromosomes.
Inner lines connecting orthologous regions are colored based on the human chromosome color scale.
Repair and transcription values (outer rings) for genomic regions that have no evident ortholog (based on the criteria in A) are not shown.</figcaption>
</figure>
</div>
<div id="fig:fig5" class="fignos">
<figure>
<img src="images/fig-0.png" class="white" alt="" /><figcaption><span>Figure 5:</span> Repair rate consistency in correlation with gene expression.
The transcription levels of the orthologous regions were divided into quartiles separately for human and lemur.
Out of the quartiles, 10 bootstrapped subsamples were retrieved, and for those regions repair correlation was analyzed.
Correlation coefficient (R values) are represented on the y-axis; quartiles are on the x-axis.</figcaption>
</figure>
</div>
<div id="fig:fig6" class="fignos">
<figure>
<img src="images/fig-0.png" class="white" alt="" /><figcaption><span>Figure 6:</span> Cell type-based consistency of repair profiles between two primates. A) Scatter plot or normalized repair levels between two human cell lines, GM18787 and NHF1, show correlation (R=0.65, p=0). B) Principle component analysis of 10 XR-seq samples.</figcaption>
</figure>
</div>
<h2 class="page_break_before" id="supplementary-materials">Supplementary Materials</h2>
<div id="fig:S1" class="fignos">
<figure>
<img src="images/simulation.png" data-tag="S1" alt="" /><figcaption><span>Figure S1:</span> Nucleotide content of simulated XR-seq pseudo-oligomers.</figcaption>
</figure>
</div>
<div id="fig:S2" class="fignos">
<figure>
<img src="images/Human_RNA_64.png" data-tag="S2" alt="" /><figcaption><span>Figure S2:</span> Human and mouse lemur (6-4)PP repair correlation in human transcription quartiles.</figcaption>
</figure>
</div>
<div id="fig:S3" class="fignos">
<figure>
<img src="images/Lemur_RNA_64.png" data-tag="S3" alt="" /><figcaption><span>Figure S3:</span> Human and mouse lemur (6-4)PP repair correlation in mouse lemur transcription quartiles.</figcaption>
</figure>
</div>
<div id="fig:S4" class="fignos">
<figure>
<img src="images/Human_RNA_CPD.png" data-tag="S4" alt="" /><figcaption><span>Figure S4:</span> Human and mouse lemur CPD repair correlation in human transcription quartiles.</figcaption>
</figure>
</div>
<div id="fig:S5" class="fignos">
<figure>
<img src="images/Lemur_RNA_CPD.png" data-tag="S5" alt="" /><figcaption><span>Figure S5:</span> Human and mouse lemur CPD repair correlation in mouse lemur transcription quartiles.</figcaption>
</figure>
</div>
<div id="fig:S6" class="fignos">
<figure>
<img src="images/pca_all_v7.png" data-tag="S6" alt="" /><figcaption><span>Figure S6:</span> PCA plot for XR-seq reads mapped on the genic (A) and intergenic (B) regions.</figcaption>
</figure>
</div>
<!-- default theme -->

<style>
    /* import google fonts */
    @import url("https://fonts.googleapis.com/css?family=Open+Sans:400,600,700");
    @import url("https://fonts.googleapis.com/css?family=Source+Code+Pro");

    /* -------------------------------------------------- */
    /* global */
    /* -------------------------------------------------- */

    /* all elements */
    * {
        /* force sans-serif font unless specified otherwise */
        /* font-family: "Open Sans", "Helvetica", sans-serif; */
        font-family: "Arial", "Helvetica", sans-serif;

        /* prevent text inflation on some mobile browsers */
        -webkit-text-size-adjust: none !important;
        -moz-text-size-adjust: none !important;
        -o-text-size-adjust: none !important;
        text-size-adjust: none !important;
    }

    @media only screen {
        /* "page" element */
        body {
            position: relative;
            box-sizing: border-box;
            font-size: 12pt;
            line-height: 1.15;
            max-width: 8.5in;
            margin: 20px auto;
            padding: 40px;
            border-radius: 5px;
            border: solid 1px #bdbdbd;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
            background: #ffffff;
        }
    }

    /* when on screen < 8.5in wide */
    @media only screen and (max-width: 8.5in) {
        /* "page" element */
        body {
            padding: 20px;
            margin: 0;
            border-radius: 0;
            border: none;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05) inset;
            background: none;
        }
    }

    /* -------------------------------------------------- */
    /* headings */
    /* -------------------------------------------------- */

    /* all headings */
    h1,
    h2,
    h3,
    h4,
    h5,
    h6 {
        margin: 20px 0;
        padding: 0;
        font-weight: bold;
    }

    /* biggest heading */
    h1 {
        margin: 40px 0;
        text-align: center;
        text-align: left;
    }

    /* second biggest heading */
    h2 {
        margin-top: 30px;
        padding-bottom: 5px;
        border-bottom: solid 1px #bdbdbd;
    }

    /* -------------------------------------------------- */
    /* manuscript header */
    /* -------------------------------------------------- */

    /* manuscript title */
    header > h1 {
        margin: 0;
    }

    /* manuscript title caption text (ie "automatically generated on") */
    header + p {
        /* text-align: center; */
        text-align: left;
        margin-top: 10px;
    }

    /* -------------------------------------------------- */
    /* text elements */
    /* -------------------------------------------------- */

    /* links */
    a {
        color: #2196f3;
        overflow-wrap: break-word;
    }

    /* normal links (not empty, not button link, not syntax highlighting link) */
    a:not(:empty):not(.button):not(.sourceLine) {
        padding-left: 1px;
        padding-right: 1px;
    }

    /* superscripts and subscripts */
    sub,
    sup {
        /* prevent from affecting line height */
        line-height: 0;
    }

    /* unordered and ordered lists*/
    ul,
    ol {
        padding-left: 20px;
    }

    /* class for styling text semibold */
    .semibold {
        font-weight: 600;
    }

    /* class for styling elements horizontally left aligned */
    .left {
        display: block;
        text-align: left;
        margin-left: auto;
        margin-right: 0;
        justify-content: left;
    }

    /* class for styling elements horizontally centered */
    .center {
        display: block;
        text-align: center;
        margin-left: auto;
        margin-right: auto;
        justify-content: center;
    }

    /* class for styling elements horizontally right aligned */
    .right {
        display: block;
        text-align: right;
        margin-left: 0;
        margin-right: auto;
        justify-content: right;
    }

    /* -------------------------------------------------- */
    /* section elements */
    /* -------------------------------------------------- */

    /* horizontal divider line */
    hr {
        border: none;
        height: 1px;
        background: #bdbdbd;
    }

    /* paragraphs, horizontal dividers, figures, tables, code */
    p,
    hr,
    figure,
    table,
    pre {
        /* treat all as "paragraphs", with consistent vertical margins */
        margin-top: 20px;
        margin-bottom: 20px;
    }

    /* -------------------------------------------------- */
    /* figures */
    /* -------------------------------------------------- */

    /* figure */
    figure {
        max-width: 100%;
        margin-left: auto;
        margin-right: auto;
    }

    /* figure caption */
    figcaption {
        padding: 0;
        padding-top: 10px;
    }

    /* figure image element */
    figure img {
        max-width: 100%;
        display: block;
        margin-left: auto;
        margin-right: auto;
    }

    /* figure auto-number */
    img + figcaption > span:first-of-type {
        font-weight: bold;
        margin-right: 5px;
    }

    /* -------------------------------------------------- */
    /* tables */
    /* -------------------------------------------------- */

    /* table */
    table {
        border-collapse: collapse;
        border-spacing: 0;
        width: 100%;
        margin-left: auto;
        margin-right: auto;
    }

    /* table cells */
    th,
    td {
        border: solid 1px #bdbdbd;
        padding: 10px;
        /* squash table if too wide for page by forcing line breaks */
        overflow-wrap: break-word;
    }

    /* header row and even rows */
    th,
    tr:nth-child(2n) {
        background-color: #fafafa;
    }

    /* odd rows */
    tr:nth-child(2n + 1) {
        background-color: #ffffff;
    }

    /* table caption */
    caption {
        text-align: left;
        padding: 0;
        padding-bottom: 10px;
    }

    /* table auto-number */
    table > caption > span:first-of-type,
    div.table_wrapper > table > caption > span:first-of-type {
        font-weight: bold;
        margin-right: 5px;
    }

    /* -------------------------------------------------- */
    /* code */
    /* -------------------------------------------------- */

    /* multi-line code block */
    pre {
        padding: 10px;
        background-color: #eeeeee;
        color: #000000;
        border-radius: 5px;
        break-inside: avoid;
        text-align: left;
    }

    /* inline code, ie code within normal text */
    :not(pre) > code {
        padding: 0 4px;
        background-color: #eeeeee;
        color: #000000;
        border-radius: 5px;
    }

    /* code text */
    /* apply all children, to reach syntax highlighting sub-elements */
    code,
    code * {
        /* force monospace font */
        font-family: "Source Code Pro", "Courier New", monospace;
    }

    /* -------------------------------------------------- */
    /* quotes */
    /* -------------------------------------------------- */

    /* quoted text */
    blockquote {
        margin: 0;
        padding: 0;
        border-left: 4px solid #bdbdbd;
        padding-left: 16px;
        break-inside: avoid;
    }

    /* -------------------------------------------------- */
    /* banners */
    /* -------------------------------------------------- */

    /* info banners */
    .banner {
        box-sizing: border-box;
        display: block;
        position: relative;
        width: 100%;
        margin-top: 20px;
        margin-bottom: 20px;
        padding: 20px;
        text-align: center;
    }

    /* paragraph in banner */
    .banner > p {
        margin: 0;
    }

    /* -------------------------------------------------- */
    /* highlight colors */
    /* -------------------------------------------------- */

    .white {
        background: #ffffff;
    }
    .lightgrey {
        background: #eeeeee;
    }
    .grey {
        background: #757575;
    }
    .darkgrey {
        background: #424242;
    }
    .black {
        background: #000000;
    }
    .lightred {
        background: #ffcdd2;
    }
    .lightyellow {
        background: #ffecb3;
    }
    .lightgreen {
        background: #dcedc8;
    }
    .lightblue {
        background: #e3f2fd;
    }
    .lightpurple {
        background: #f3e5f5;
    }
    .red {
        background: #f44336;
    }
    .orange {
        background: #ff9800;
    }
    .yellow {
        background: #ffeb3b;
    }
    .green {
        background: #4caf50;
    }
    .blue {
        background: #2196f3;
    }
    .purple {
        background: #9c27b0;
    }
    .white,
    .lightgrey,
    .lightred,
    .lightyellow,
    .lightgreen,
    .lightblue,
    .lightpurple,
    .orange,
    .yellow,
    .white a,
    .lightgrey a,
    .lightred a,
    .lightyellow a,
    .lightgreen a,
    .lightblue a,
    .lightpurple a,
    .orange a,
    .yellow a {
        color: #000000;
    }
    .grey,
    .darkgrey,
    .black,
    .red,
    .green,
    .blue,
    .purple,
    .grey a,
    .darkgrey a,
    .black a,
    .red a,
    .green a,
    .blue a,
    .purple a {
        color: #ffffff;
    }

    /* -------------------------------------------------- */
    /* buttons */
    /* -------------------------------------------------- */

    /* class for styling links like buttons */
    .button {
        display: inline-flex;
        justify-content: center;
        align-items: center;
        margin: 5px;
        padding: 10px 20px;
        font-size: 0.75em;
        font-weight: 600;
        text-transform: uppercase;
        text-decoration: none;
        letter-spacing: 1px;
        background: none;
        color: #2196f3;
        border: solid 1px #bdbdbd;
        border-radius: 5px;
    }

    /* buttons when hovered */
    .button:hover:not([disabled]),
    .icon_button:hover:not([disabled]) {
        cursor: pointer;
        background: #f5f5f5;
    }

    /* buttons when disabled */
    .button[disabled],
    .icon_button[disabled] {
        opacity: 0.35;
        pointer-events: none;
    }

    /* class for styling buttons containg only single icon */
    .icon_button {
        display: inline-flex;
        justify-content: center;
        align-items: center;
        text-decoration: none;
        margin: 0;
        padding: 0;
        background: none;
        border-radius: 5px;
        border: none;
        width: 20px;
        height: 20px;
        min-width: 20px;
        min-height: 20px;
    }

    /* icon button inner svg image */
    .icon_button > svg {
        height: 16px;
    }

    /* -------------------------------------------------- */
    /* icons */
    /* -------------------------------------------------- */

    /* class for styling icons inline with text */
    .inline_icon {
        height: 1em;
        position: relative;
        top: 0.125em;
    }

    /* -------------------------------------------------- */
    /* print control */
    /* -------------------------------------------------- */

    @media print {
        @page {
            /* suggested printing margin */
            margin: 0.5in;
        }

        /* document and "page" elements */
        html, body {
            margin: 0;
            padding: 0;
            width: 100%;
            height: 100%;
        }

        /* "page" element */
        body {
            font-size: 12pt !important;
            line-height: 2;
        }

        /* all headings */
        h1,
        h2,
        h3,
        h4,
        h5,
        h6 {
            margin: 15px 0;
        }

        /* heading 1 */
        h1 {
            font-size: 1.25em;
        }

        /* heading 2 */
        h2 {
            font-size: 1.15em;
            margin-top: 0;
        }

        /* heading 3 */
        h3 {
            font-size: 1.10em;
        }

        /* figures and tables */
        figure, table {
            font-size: 0.85em;
        }

        /* table cells */
        th,
        td {
            padding: 5px;
        }

        /* shrink font awesome icons */
        i.fas,
        i.fab,
        i.far,
        i.fal {
            transform: scale(0.85);
        }

        /* decrease banner margins */
        .banner {
            margin-top: 15px;
            margin-bottom: 15px;
            padding: 15px;
        }

        /* class for centering an element vertically on its own page */
        .page_center {
            margin: auto;
            width: 100%;
            height: 100%;
            display: flex;
            align-items: center;
            vertical-align: middle;
            break-before: page;
            break-after: page;
        }

        /* always insert a page break before the element */
        .page_break_before {
            break-before: page;
        }

        /* always insert a page break after the element */
        .page_break_after {
            break-after: page;
        }

        /* avoid page break before the element */
        .page_break_before_avoid {
            break-before: avoid;
        }

        /* avoid page break after the element */
        .page_break_after_avoid {
            break-after: avoid;
        }

        /* avoid page break inside the element */
        .page_break_inside_avoid {
            break-inside: avoid;
        }
    }

    /* -------------------------------------------------- */
    /* override pandoc css quirks */
    /* -------------------------------------------------- */

    .sourceCode {
        /* prevent unsightly overflow in wide code blocks */
        overflow: auto !important;
    }

    div.sourceCode {
        /* prevent background fill on top-most code block  container */
        background: none !important;
    }

    .sourceCode * {
        /* force consistent line spacing */
        line-height: 1.5 !important;
    }

    div.sourceCode {
        /* style code block margins same as <pre> element */
        margin-top: 20px;
        margin-bottom: 20px;
    }

    /* -------------------------------------------------- */
    /* mathjax */
    /* -------------------------------------------------- */

    /* mathjax containers */
    .math.display > span:not(.MathJax_Preview) {
        /* turn inline element (no dimensions) into block (allows fixed width and thus scrolling) */
        display: flex !important;
        overflow-x: auto !important;
        overflow-y: hidden !important;
        justify-content: center;
        align-items: center;
        margin: 0 !important;
    }

    /* right click menu */
    .MathJax_Menu {
        border-radius: 5px !important;
        border: solid 1px #bdbdbd !important;
        box-shadow: none !important;
    }

    /* equation auto-number */
    span[id^="eq:"] > span.math.display + span {
        font-weight: 600;
    }

    /* equation */
    span[id^="eq:"] > span.math.display > span {
        /* nudge to make room for equation auto-number */
        margin-right: 40px !important;
    }

    /* -------------------------------------------------- */
    /* table scroll plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* table wrapper */
        .table_wrapper {
            /* show scrollbar on tables if necessary to prevent overflow */
            overflow: auto;
            width: 100%;
            margin: 20px 0;
        }

        /* table within table wrapper */
        .table_wrapper table,
        .table_wrapper table * {
            /* don't break table words */
            word-break: normal !important;
        }

        .table_wrapper > table {
            /* move margins from table to table_wrapper to allow margin collapsing */
            margin: 0;
        }
    }

    /* -------------------------------------------------- */
    /* anchors plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* anchor button */
        .anchor {
            opacity: 0;
            margin-left: 5px;
        }

        /* anchor buttons within <h2>'s */
        h2 .anchor {
            margin-left: 10px;
        }

        /* anchor buttons when hovered/focused and anything containing an anchor button when hovered */
        *:hover > .anchor,
        .anchor:hover,
        .anchor:focus {
            opacity: 1;
        }

        /* anchor button when hovered */
        .anchor:hover {
            cursor: pointer;
        }
    }

    /* always show anchor button on devices with no mouse/hover ability */
    @media (hover: none) {
        .anchor {
            opacity: 1;
        }
    }

    /* always hide anchor button on print */
    @media only print {
        .anchor {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* accordion plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* accordion arrow button */
        .accordion_arrow {
            margin-right: 10px;
        }

        /* arrow icon when <h2> data-collapsed attribute true */
        h2[data-collapsed="true"] > .accordion_arrow > svg {
            transform: rotate(-90deg);
        }

        /* all elements (except <h2>'s) when data-collapsed attribute true */
        *:not(h2)[data-collapsed="true"] {
            display: none;
        }

        /* accordion arrow button when hovered and <h2>'s when hovered */
        .accordion_arrow:hover,
        h2[data-collapsed="true"]:hover,
        h2[data-collapsed="false"]:hover {
            cursor: pointer;
        }
    }

    /* always hide accordion arrow button on print */
    @media only print {
        .accordion_arrow {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* tooltips plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* tooltip container */
        #tooltip {
            position: absolute;
            width: 50%;
            min-width: 240px;
            max-width: 75%;
            z-index: 1;
        }

        /* tooltip content */
        #tooltip_content {
            margin-bottom: 5px;
            padding: 20px;
            border-radius: 5px;
            border: solid 1px #bdbdbd;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
            background: #ffffff;
            overflow-wrap: break-word;
        }

        /* tooltip copy of paragraphs and figures */
        #tooltip_content > p,
        #tooltip_content > figure {
            margin: 0;
            max-height: 320px;
            overflow-y: auto;
        }

        /* tooltip copy of <img> */
        #tooltip_content > figure > img {
            max-height: 260px;
        }

        /* navigation bar */
        #tooltip_nav_bar {
            margin-top: 10px;
            text-align: center;
        }

        /* navigation bar previous/next buton */
        #tooltip_nav_bar > .icon_button {
            position: relative;
            top: 3px;
        }

        /* navigation bar previous button */
        #tooltip_nav_bar > .icon_button:first-of-type {
            margin-right: 5px;
        }

        /* navigation bar next button */
        #tooltip_nav_bar > .icon_button:last-of-type {
            margin-left: 5px;
        }
    }

    /* always hide tooltip on print */
    @media only print {
        #tooltip {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* jump to first plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* jump button */
        .jump_arrow {
            position: relative;
            top: 0.125em;
            margin-right: 5px;
        }
    }

    /* always hide jump button on print */
    @media only print {
        .jump_arrow {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* link highlight plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* anything with data-highlighted attribute true */
        [data-highlighted="true"] {
            background: #ffeb3b;
        }

        /* anything with data-selected attribute true */
        [data-selected="true"] {
            background: #ff8a65 !important;
        }

        /* animation definition for glow */
        @keyframes highlight_glow {
            0% {
                background: none;
            }
            10% {
                background: #bbdefb;
            }
            100% {
                background: none;
            }
        }

        /* anything with data-glow attribute true */
        [data-glow="true"] {
            animation: highlight_glow 2s;
        }
    }

    /* -------------------------------------------------- */
    /* table of contents plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* toc panel */
        #toc_panel {
            box-sizing: border-box;
            position: fixed;
            top: 0;
            left: 0;
            background: #ffffff;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
            z-index: 2;
        }

        /* toc panel when closed */
        #toc_panel[data-open="false"] {
            min-width: 60px;
            width: 60px;
            height: 60px;
            border-right: solid 1px #bdbdbd;
            border-bottom: solid 1px #bdbdbd;
        }

        /* toc panel when open */
        #toc_panel[data-open="true"] {
            min-width: 260px;
            max-width: 480px;
            /* keep panel edge consistent distance away from "page" edge */
            width: calc(((100vw - 8.5in) / 2) - 30px - 40px);
            bottom: 0;
            border-right: solid 1px #bdbdbd;
        }

        /* toc panel header */
        #toc_header {
            box-sizing: border-box;
            display: flex;
            flex-direction: row;
            align-items: center;
            height: 60px;
            margin: 0;
            padding: 20px;
        }

        /* toc panel header when hovered */
        #toc_header:hover {
            cursor: pointer;
        }

        /* toc panel header when panel open */
        #toc_panel[data-open="true"] > #toc_header {
            border-bottom: solid 1px #bdbdbd;
        }

        /* toc open/close header button */
        #toc_button {
            margin-right: 20px;
        }

        /* hide toc list and header text when closed */
        #toc_panel[data-open="false"] > #toc_header > *:not(#toc_button),
        #toc_panel[data-open="false"] > #toc_list {
            display: none;
        }

        /* toc list of entries */
        #toc_list {
            box-sizing: border-box;
            width: 100%;
            padding: 20px;
            position: absolute;
            top: calc(60px + 1px);
            bottom: 0;
            overflow: auto;
        }

        /* toc entry, link to section in document */
        .toc_link {
            display: block;
            padding: 5px;
            position: relative;
            font-weight: 600;
            text-decoration: none;
        }

        /* toc entry when hovered or when "viewed" */
        .toc_link:hover,
        .toc_link[data-viewing="true"] {
            background: #f5f5f5;
        }

        /* toc entry, level 1 indentation */
        .toc_link[data-level="1"] {
            margin-left: 0;
        }

        /* toc entry, level 2 indentation */
        .toc_link[data-level="2"] {
            margin-left: 20px;
        }

        /* toc entry, level 3 indentation */
        .toc_link[data-level="3"] {
            margin-left: 40px;
        }

        /* toc entry, level 4 indentation */
        .toc_link[data-level="4"] {
            margin-left: 60px;
        }

        /* toc entry bullets */
        #toc_panel[data-bullets="true"] .toc_link[data-level]:before {
            position: absolute;
            left: -15px;
            top: -1px;
            font-size: 1.5em;
        }

        /* toc entry, level 2 bullet */
        #toc_panel[data-bullets="true"] .toc_link[data-level="2"]:before {
            content: "\2022";
        }

        /* toc entry, level 3 bullet */
        #toc_panel[data-bullets="true"] .toc_link[data-level="3"]:before {
            content: "\25AB";
        }

        /* toc entry, level 4 bullet */
        #toc_panel[data-bullets="true"] .toc_link[data-level="4"]:before {
            content: "-";
        }
    }

    /* when on screen < 8.5in wide */
    @media only screen and (max-width: 8.5in) {
        /* push <body> ("page") element down to make room for toc icon */
        .toc_body_nudge {
            padding-top: 60px;
        }

        /* toc icon when panel closed and not hovered */
        #toc_panel[data-open="false"]:not(:hover) {
            background: rgba(255, 255, 255, 0.75);
        }
    }

    /* always hide toc panel on print */
    @media only print {
        #toc_panel {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* lightbox plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* regular <img> in document when hovered */
        .lightbox_document_img:hover {
            cursor: pointer;
        }

        .body_no_scroll {
            overflow: hidden !important;
        }

        /* screen overlay */
        #lightbox_overlay {
            display: flex;
            flex-direction: column;
            position: fixed;
            left: 0;
            top: 0;
            right: 0;
            bottom: 0;
            background: rgba(0, 0, 0, 0.75);
            z-index: 3;
        }

        /* middle area containing lightbox image */
        #lightbox_image_container {
            flex-grow: 1;
            display: flex;
            justify-content: center;
            align-items: center;
            overflow: hidden;
            position: relative;
            padding: 20px;
        }

        /* bottom area containing caption */
        #lightbox_bottom_container {
            display: flex;
            justify-content: center;
            align-items: center;
            height: 100px;
            min-height: 100px;
            max-height: 100px;
            background: rgba(0, 0, 0, 0.5);
        }

        /* image number info text box */
        #lightbox_number_info {
            position: absolute;
            color: #ffffff;
            font-weight: 600;
            left: 2px;
            top: 0;
            z-index: 4;
        }

        /* zoom info text box */
        #lightbox_zoom_info {
            position: absolute;
            color: #ffffff;
            font-weight: 600;
            right: 2px;
            top: 0;
            z-index: 4;
        }

        /* copy of image caption */
        #lightbox_caption {
            box-sizing: border-box;
            display: inline-block;
            width: 100%;
            max-height: 100%;
            padding: 10px 0;
            text-align: center;
            overflow-y: auto;
            color: #ffffff;
        }

        /* navigation previous/next button */
        .lightbox_button {
            width: 100px;
            height: 100%;
            min-width: 100px;
            min-height: 100%;
            color: #ffffff;
        }

        /* navigation previous/next button when hovered */
        .lightbox_button:hover {
            background: none !important;
        }

        /* navigation button icon */
        .lightbox_button > svg {
            height: 25px;
        }

        /* figure auto-number */
        #lightbox_caption > span:first-of-type {
            font-weight: bold;
            margin-right: 5px;
        }

        /* lightbox image when hovered */
        #lightbox_img:hover {
            cursor: grab;
        }

        /* lightbox image when grabbed */
        #lightbox_img:active {
            cursor: grabbing;
        }
    }

    /* when on screen < 480px wide */
    @media only screen and (max-width: 480px) {
        /* make navigation buttons skinnier on small screens to make more room for caption text */
        .lightbox_button {
            width: 50px;
            min-width: 50px;
        }
    }

    /* always hide lightbox on print */
    @media only print {
        #lightbox_overlay {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* hypothesis (annotations) plugin */
    /* -------------------------------------------------- */

    /* side panel */
    .annotator-frame {
        width: 280px !important;
        z-index: 0 !important;
    }

    /* match highlight color to rest of theme */
    .annotator-highlights-always-on .annotator-hl {
        background-color: #ffeb3b !important;
    }

    /* match focused color to rest of theme */
    .annotator-hl.annotator-hl-focused {
        background-color: #ff8a65 !important;
    }

    /* match bucket bar color to rest of theme */
    .annotator-bucket-bar {
        background: #f5f5f5 !important;
    }

    /* always hide toolbar and tooltip on print */
    @media only print {
        .annotator-frame {
            display: none !important;
        }

        hypothesis-adder {
            display: none !important;
        }
    }
</style>
<!-- 
    Plugin Core

    Functions needed for and shared across all first-party plugins.
-->

<script>
  // get element that is target of hash (from link element or url)
  function getHashTarget(link) {
    const hash = link ? link.hash : window.location.hash;
    const id = hash.slice(1);
    let target = document.querySelector(`[id="${id}"]`);
    if (!target) return;

    // if figure or table, modify target to get expected element
    if (id.indexOf("fig:") === 0) target = target.querySelector("figure");
    if (id.indexOf("tbl:") === 0) target = target.querySelector("table");

    return target;
  }

  // get position/dimensions of element or viewport
  function getRectInView(element) {
    let rect = {};
    rect.left = 0;
    rect.top = 0;
    rect.right = document.documentElement.clientWidth;
    rect.bottom = document.documentElement.clientHeight;
    let style = {};

    if (element instanceof HTMLElement) {
      rect = element.getBoundingClientRect();
      style = window.getComputedStyle(element);
    }

    const margin = {};
    margin.left = parseFloat(style.marginLeftWidth) || 0;
    margin.top = parseFloat(style.marginTopWidth) || 0;
    margin.right = parseFloat(style.marginRightWidth) || 0;
    margin.bottom = parseFloat(style.marginBottomWidth) || 0;

    const border = {};
    border.left = parseFloat(style.borderLeftWidth) || 0;
    border.top = parseFloat(style.borderTopWidth) || 0;
    border.right = parseFloat(style.borderRightWidth) || 0;
    border.bottom = parseFloat(style.borderBottomWidth) || 0;

    const newRect = {};
    newRect.left = rect.left + margin.left + border.left;
    newRect.top = rect.top + margin.top + border.top;
    newRect.right = rect.right + margin.right + border.right;
    newRect.bottom = rect.bottom + margin.bottom + border.bottom;
    newRect.width = newRect.right - newRect.left;
    newRect.height = newRect.bottom - newRect.top;

    return newRect;
  }

  // get position of element relative to page
  function getRectInPage(element) {
    const rect = getRectInView(element);
    const body = getRectInView(document.body);

    const newRect = {};
    newRect.left = rect.left - body.left;
    newRect.top = rect.top - body.top;
    newRect.right = rect.right - body.left;
    newRect.bottom = rect.bottom - body.top;
    newRect.width = rect.width;
    newRect.height = rect.height;

    return newRect;
  }

  // get closest element before specified element that matches query
  function firstBefore(element, query) {
    while (element && element !== document.body && !element.matches(query))
      element = element.previousElementSibling || element.parentNode;

    return element;
  }

  // check if element is part of collapsed heading
  function isCollapsed(element) {
    while (element && element !== document.body) {
      if (element.dataset.collapsed === "true") return true;
      element = element.parentNode;
    }
    return false;
  }

  // expand any collapsed parent containers of element if necessary
  function expandElement(element) {
    if (isCollapsed(element)) {
      // accordion plugin
      const heading = firstBefore(element, "h2");
      if (heading) heading.click();
      // details/summary HTML element
      const summary = firstBefore(element, "summary");
      if (summary) summary.click();
    }
  }

  // scroll to and focus element
  function goToElement(element, offset) {
    // expand accordion section if collapsed
    expandElement(element);
    const y =
      getRectInView(element).top -
      getRectInView(document.documentElement).top -
      (offset || 0);

    // trigger any function listening for "onscroll" event
    window.dispatchEvent(new Event("scroll"));
    window.scrollTo(0, y);
    document.activeElement.blur();
    element.focus();
  }

  // get list of elements after a start element up to element matching query
  function nextUntil(element, query, exclude) {
    const elements = [];
    while (((element = element.nextElementSibling), element)) {
      if (element.matches(query)) break;
      if (!element.matches(exclude)) elements.push(element);
    }
    return elements;
  }
</script>
<!--
  Accordion Plugin

  Allows sections of content under h2 headings to be collapsible.
-->

<script type="module">
  // whether to always start expanded ('false'), always start collapsed
  // ('true'), or start collapsed when screen small ('auto')
  const startCollapsed = "auto";

  // start script
  function start() {
    // run through each <h2> heading
    const headings = document.querySelectorAll("h2");
    for (const heading of headings) {
      addArrow(heading);

      // start expanded/collapsed based on option
      if (
        startCollapsed === "true" ||
        (startCollapsed === "auto" && isSmallScreen())
      )
        collapseHeading(heading);
      else expandElement(heading);
    }

    // attach hash change listener to window
    window.addEventListener("hashchange", onHashChange);
  }

  // when hash (eg manuscript.html#introduction) changes
  function onHashChange() {
    const target = getHashTarget();
    if (target) goToElement(target);
  }

  // add arrow to heading
  function addArrow(heading) {
    // add arrow button
    const arrow = document.createElement("button");
    arrow.innerHTML = document.querySelector(".icon_angle_down").innerHTML;
    arrow.classList.add("icon_button", "accordion_arrow");
    heading.insertBefore(arrow, heading.firstChild);

    // attach click listener to heading and button
    heading.addEventListener("click", onHeadingClick);
    arrow.addEventListener("click", onArrowClick);
  }

  // determine if on mobile-like device with small screen
  function isSmallScreen() {
    return Math.min(window.innerWidth, window.innerHeight) < 480;
  }

  // when <h2> heading is clicked
  function onHeadingClick(event) {
    // only collapse if <h2> itself is target of click (eg, user did
    // not click on anchor within <h2>)
    if (event.target === this) toggleCollapse(this);
  }

  // when arrow button is clicked
  function onArrowClick() {
    toggleCollapse(this.parentNode);
  }

  // collapse section if expanded, expand if collapsed
  function toggleCollapse(heading) {
    if (heading.dataset.collapsed === "false") collapseHeading(heading);
    else expandElement(heading);
  }

  // elements to exclude from collapse, such as table of contents panel,
  // hypothesis panel, etc
  const exclude = "#toc_panel, div.annotator-frame, #lightbox_overlay";

  // collapse section
  function collapseHeading(heading) {
    heading.setAttribute("data-collapsed", "true");
    const children = getChildren(heading);
    for (const child of children) child.setAttribute("data-collapsed", "true");
  }

  // expand section
  function expandElement(heading) {
    heading.setAttribute("data-collapsed", "false");
    const children = getChildren(heading);
    for (const child of children) child.setAttribute("data-collapsed", "false");
  }

  // get list of elements between this <h2> and next <h2> or <h1>
  // ("children" of the <h2> section)
  function getChildren(heading) {
    return nextUntil(heading, "h2, h1", exclude);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- angle down icon -->

<template class="icon_angle_down">
  <!-- modified from: https://fontawesome.com/icons/angle-down -->
  <svg width="16" height="16" viewBox="0 0 448 512">
    <path
      fill="currentColor"
      d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* accordion arrow button */
    .accordion_arrow {
      margin-right: 10px;
    }

    /* arrow icon when <h2> data-collapsed attribute true */
    h2[data-collapsed="true"] > .accordion_arrow > svg {
      transform: rotate(-90deg);
    }

    /* all elements (except <h2>'s) when data-collapsed attribute true */
    *:not(h2)[data-collapsed="true"] {
      display: none;
    }

    /* accordion arrow button when hovered and <h2>'s when hovered */
    .accordion_arrow:hover,
    h2[data-collapsed="true"]:hover,
    h2[data-collapsed="false"]:hover {
      cursor: pointer;
    }
  }

  /* always hide accordion arrow button on print */
  @media only print {
    .accordion_arrow {
      display: none;
    }
  }
</style>
<!--
  Anchors Plugin

  Adds an anchor next to each of a certain type of element that provides a
  human-readable url to that specific item/position in the document (e.g.
  "manuscript.html#abstract"). It also makes it such that scrolling out of view
  of a target removes its identifier from the url.
-->

<script type="module">
  // which types of elements to add anchors next to, in "document.querySelector"
  // format
  const typesQuery = 'h1, h2, h3, [id^="fig:"], [id^="tbl:"], [id^="eq:"]';

  // start script
  function start() {
    // add anchor to each element of specified types
    const elements = document.querySelectorAll(typesQuery);
    for (const element of elements) addAnchor(element);

    // attach scroll listener to window
    window.addEventListener("scroll", onScroll);
  }

  // when window is scrolled
  function onScroll() {
    // if url has hash and user has scrolled out of view of hash
    // target, remove hash from url
    const tolerance = 100;
    const target = getHashTarget();
    if (target) {
      if (
        target.getBoundingClientRect().top > window.innerHeight + tolerance ||
        target.getBoundingClientRect().bottom < 0 - tolerance
      )
        history.pushState(null, null, " ");
    }
  }

  // add anchor to element
  function addAnchor(element) {
    let addTo; // element to add anchor button to

    // if figure or table, modify withId and addTo to get expected
    // elements
    if (element.id.indexOf("fig:") === 0) {
      addTo = element.querySelector("figcaption");
    } else if (element.id.indexOf("tbl:") === 0) {
      addTo = element.querySelector("caption");
    } else if (element.id.indexOf("eq:") === 0) {
      addTo = element.querySelector(".eqnos-number");
    }

    addTo = addTo || element;
    const id = element.id || null;

    // do not add anchor if element doesn't have assigned id.
    // id is generated by pandoc and is assumed to be unique and
    // human-readable
    if (!id) return;

    // create anchor button
    const anchor = document.createElement("a");
    anchor.innerHTML = document.querySelector(".icon_link").innerHTML;
    anchor.title = "Link to this part of the document";
    anchor.classList.add("icon_button", "anchor");
    anchor.dataset.ignore = "true";
    anchor.href = "#" + id;
    addTo.appendChild(anchor);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- link icon -->

<template class="icon_link">
  <!-- modified from: https://fontawesome.com/icons/link -->
  <svg width="16" height="16" viewBox="0 0 512 512">
    <path
      fill="currentColor"
      d="M326.612 185.391c59.747 59.809 58.927 155.698.36 214.59-.11.12-.24.25-.36.37l-67.2 67.2c-59.27 59.27-155.699 59.262-214.96 0-59.27-59.26-59.27-155.7 0-214.96l37.106-37.106c9.84-9.84 26.786-3.3 27.294 10.606.648 17.722 3.826 35.527 9.69 52.721 1.986 5.822.567 12.262-3.783 16.612l-13.087 13.087c-28.026 28.026-28.905 73.66-1.155 101.96 28.024 28.579 74.086 28.749 102.325.51l67.2-67.19c28.191-28.191 28.073-73.757 0-101.83-3.701-3.694-7.429-6.564-10.341-8.569a16.037 16.037 0 0 1-6.947-12.606c-.396-10.567 3.348-21.456 11.698-29.806l21.054-21.055c5.521-5.521 14.182-6.199 20.584-1.731a152.482 152.482 0 0 1 20.522 17.197zM467.547 44.449c-59.261-59.262-155.69-59.27-214.96 0l-67.2 67.2c-.12.12-.25.25-.36.37-58.566 58.892-59.387 154.781.36 214.59a152.454 152.454 0 0 0 20.521 17.196c6.402 4.468 15.064 3.789 20.584-1.731l21.054-21.055c8.35-8.35 12.094-19.239 11.698-29.806a16.037 16.037 0 0 0-6.947-12.606c-2.912-2.005-6.64-4.875-10.341-8.569-28.073-28.073-28.191-73.639 0-101.83l67.2-67.19c28.239-28.239 74.3-28.069 102.325.51 27.75 28.3 26.872 73.934-1.155 101.96l-13.087 13.087c-4.35 4.35-5.769 10.79-3.783 16.612 5.864 17.194 9.042 34.999 9.69 52.721.509 13.906 17.454 20.446 27.294 10.606l37.106-37.106c59.271-59.259 59.271-155.699.001-214.959z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* anchor button */
    .anchor {
      opacity: 0;
      margin-left: 5px;
    }

    /* anchor buttons within <h2>'s */
    h2 .anchor {
      margin-left: 10px;
    }

    /* anchor buttons when hovered/focused and anything containing an anchor button when hovered */
    *:hover > .anchor,
    .anchor:hover,
    .anchor:focus {
      opacity: 1;
    }

    /* anchor button when hovered */
    .anchor:hover {
      cursor: pointer;
    }
  }

  /* always show anchor button on devices with no mouse/hover ability */
  @media (hover: none) {
    .anchor {
      opacity: 1;
    }
  }

  /* always hide anchor button on print */
  @media only print {
    .anchor {
      display: none;
    }
  }
</style>
<!-- 
    Attributes Plugin

    Allows arbitrary HTML attributes to be attached to (almost) any element.
    Place an HTML comment inside or next to the desired element with the content:
    $attribute="value"
-->

<script type="module">
  // start script
  function start() {
    // get list of comments in document
    const comments = findComments();

    for (const comment of comments)
      if (comment.parentElement)
        addAttributes(comment.parentElement, comment.nodeValue.trim());
  }

  // add html attributes to specified element based on string of
  // html attributes and values
  function addAttributes(element, text) {
    // regex's for finding attribute/value pairs in the format of
    // attribute="value" or attribute='value
    const regex2 = /\$([a-zA-Z\-]+)?=\"(.+?)\"/;
    const regex1 = /\$([a-zA-Z\-]+)?=\'(.+?)\'/;

    // loop through attribute/value pairs
    let match;
    while ((match = text.match(regex2) || text.match(regex1))) {
      // get attribute and value from regex capture groups
      let attribute = match[1];
      let value = match[2];

      // remove from string
      text = text.substring(match.index + match[0].length);

      if (!attribute || !value) break;

      // set attribute of parent element
      try {
        element.setAttribute(attribute, value);
      } catch (error) {
        console.log(error);
      }

      // special case for colspan
      if (attribute === "colspan") removeTableCells(element, value);
    }
  }

  // get list of comment elements in document
  function findComments() {
    const comments = [];

    // iterate over comment nodes in document
    function acceptNode(node) {
      return NodeFilter.FILTER_ACCEPT;
    }
    const iterator = document.createNodeIterator(
      document.body,
      NodeFilter.SHOW_COMMENT,
      acceptNode
    );
    let node;
    while ((node = iterator.nextNode())) comments.push(node);

    return comments;
  }

  // remove certain number of cells after specified cell
  function removeTableCells(cell, number) {
    number = parseInt(number);
    if (!number) return;

    // remove elements
    for (; number > 1; number--) {
      if (cell.nextElementSibling) cell.nextElementSibling.remove();
    }
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>
<!--
  Jump to First Plugin

  Adds a button next to each reference entry, figure, and table that jumps the
  page to the first occurrence of a link to that item in the manuscript.
-->

<script type="module">
  // whether to add buttons next to reference entries
  const references = "true";
  // whether to add buttons next to figures
  const figures = "true";
  // whether to add buttons next to tables
  const tables = "true";

  // start script
  function start() {
    if (references !== "false") makeButtons("ref-", "p", "reference");
    if (figures !== "false") makeButtons("fig:", "figcaption", "figure");
    if (tables !== "false") makeButtons("tbl:", "caption", "table");
  }

  // when jump button clicked
  function onButtonClick() {
    const first = getFirstOccurrence(this.dataset.id);
    if (!first) return;

    // update url hash so navigating "back" in history will return user to button
    window.location.hash = this.dataset.id;
    // scroll to link
    const timeout = function () {
      goToElement(first, window.innerHeight * 0.5);
    };
    window.setTimeout(timeout, 0);
  }

  // get first occurrence of link to item in document
  function getFirstOccurrence(id) {
    let query = "a";
    query += '[href="#' + id + '"]';
    // exclude buttons, anchor links, toc links, etc
    query += ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    return document.querySelector(query);
  }

  // add button next to each reference entry, figure, or table
  function makeButtons(query, containerQuery, subject) {
    const elements = document.querySelectorAll(`[id^="${query}"`);
    for (const element of elements) {
      const id = element.id;
      const buttonContainer = element.querySelector(containerQuery);
      const first = getFirstOccurrence(id);

      // if can't find link to reference or place to put button, ignore
      if (!first || !buttonContainer) continue;

      // make jump button
      let button = document.createElement("button");
      button.classList.add("icon_button", "jump_arrow");
      button.title = `Jump to the first occurrence of this ${subject} in the document`;
      const icon = document.querySelector(".icon_angle_double_up");
      button.innerHTML = icon.innerHTML;
      button.dataset.id = id;
      button.dataset.ignore = "true";
      button.addEventListener("click", onButtonClick);
      buttonContainer.prepend(button);
    }
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- angle double up icon -->

<template class="icon_angle_double_up">
  <!-- modified from: https://fontawesome.com/icons/angle-double-up -->
  <svg width="16" height="16" viewBox="0 0 320 512">
    <path
      fill="currentColor"
      d="M177 255.7l136 136c9.4 9.4 9.4 24.6 0 33.9l-22.6 22.6c-9.4 9.4-24.6 9.4-33.9 0L160 351.9l-96.4 96.4c-9.4 9.4-24.6 9.4-33.9 0L7 425.7c-9.4-9.4-9.4-24.6 0-33.9l136-136c9.4-9.5 24.6-9.5 34-.1zm-34-192L7 199.7c-9.4 9.4-9.4 24.6 0 33.9l22.6 22.6c9.4 9.4 24.6 9.4 33.9 0l96.4-96.4 96.4 96.4c9.4 9.4 24.6 9.4 33.9 0l22.6-22.6c9.4-9.4 9.4-24.6 0-33.9l-136-136c-9.2-9.4-24.4-9.4-33.8 0z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* jump button */
    .jump_arrow {
      position: relative;
      top: 0.125em;
      margin-right: 5px;
    }
  }

  /* always hide jump button on print */
  @media only print {
    .jump_arrow {
      display: none;
    }
  }
</style>
<!-- 
    Lightbox Plugin

    Makes it such that when a user clicks on an image, the image fills the
    screen and the user can pan/drag/zoom the image and navigate between other
    images in the document.
-->

<script type="module">
  // list of possible zoom/scale factors
  const zooms =
    "0.1, 0.25, 0.333333, 0.5, 0.666666, 0.75, 1, 1.25, 1.5, 1.75, 2, 2.5, 3, 3.5, 4, 5, 6, 7, 8";
  // whether to fit image to view ('fit'), display at 100% and shrink if
  // necessary ('shrink'), or always display at 100% ('100')
  const defaultZoom = "fit";
  // whether to zoom in/out toward center of view ('true') or mouse ('false')
  const centerZoom = "false";

  // start script
  function start() {
    // run through each <img> element
    const imgs = document.querySelectorAll("figure > img");
    let count = 1;
    for (const img of imgs) {
      img.classList.add("lightbox_document_img");
      img.dataset.number = count;
      img.dataset.total = imgs.length;
      img.addEventListener("click", openLightbox);
      count++;
    }

    // attach mouse and key listeners to window
    window.addEventListener("mousemove", onWindowMouseMove);
    window.addEventListener("keyup", onKeyUp);
  }

  // when mouse is moved anywhere in window
  function onWindowMouseMove(event) {
    window.mouseX = event.clientX;
    window.mouseY = event.clientY;
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    switch (event.key) {
      // trigger click of prev button
      case "ArrowLeft":
        const prevButton = document.getElementById("lightbox_prev_button");
        if (prevButton) prevButton.click();
        break;
      // trigger click of next button
      case "ArrowRight":
        const nextButton = document.getElementById("lightbox_next_button");
        if (nextButton) nextButton.click();
        break;
      // close on esc
      case "Escape":
        closeLightbox();
        break;
    }
  }

  // open lightbox
  function openLightbox() {
    const lightbox = makeLightbox(this);
    if (!lightbox) return;

    blurBody(lightbox);
    document.body.appendChild(lightbox);
  }

  // make lightbox
  function makeLightbox(img) {
    // delete lightbox if it exists, start fresh
    closeLightbox();

    // create screen overlay containing lightbox
    const overlay = document.createElement("div");
    overlay.id = "lightbox_overlay";

    // create image info boxes
    const numberInfo = document.createElement("div");
    const zoomInfo = document.createElement("div");
    numberInfo.id = "lightbox_number_info";
    zoomInfo.id = "lightbox_zoom_info";

    // create container for image
    const imageContainer = document.createElement("div");
    imageContainer.id = "lightbox_image_container";
    const lightboxImg = makeLightboxImg(
      img,
      imageContainer,
      numberInfo,
      zoomInfo
    );
    imageContainer.appendChild(lightboxImg);

    // create bottom container for caption and navigation buttons
    const bottomContainer = document.createElement("div");
    bottomContainer.id = "lightbox_bottom_container";
    const caption = makeCaption(img);
    const prevButton = makePrevButton(img);
    const nextButton = makeNextButton(img);
    bottomContainer.appendChild(prevButton);
    bottomContainer.appendChild(caption);
    bottomContainer.appendChild(nextButton);

    // attach top middle and bottom to overlay
    overlay.appendChild(numberInfo);
    overlay.appendChild(zoomInfo);
    overlay.appendChild(imageContainer);
    overlay.appendChild(bottomContainer);

    return overlay;
  }

  // make <img> object that is intuitively draggable and zoomable
  function makeLightboxImg(sourceImg, container, numberInfoBox, zoomInfoBox) {
    // create copy of source <img>
    const img = sourceImg.cloneNode(true);
    img.classList.remove("lightbox_document_img");
    img.removeAttribute("id");
    img.removeAttribute("width");
    img.removeAttribute("height");
    img.style.position = "unset";
    img.style.margin = "0";
    img.style.padding = "0";
    img.style.width = "";
    img.style.height = "";
    img.style.minWidth = "";
    img.style.minHeight = "";
    img.style.maxWidth = "";
    img.style.maxHeight = "";
    img.id = "lightbox_img";

    // build sorted list of zoomSteps
    const zoomSteps = zooms.split(/[^0-9.]/).map((step) => parseFloat(step));
    zoomSteps.sort((a, b) => a - b);

    // <img> object property variables
    let zoom = 1;
    let translateX = 0;
    let translateY = 0;
    let clickMouseX = undefined;
    let clickMouseY = undefined;
    let clickTranslateX = undefined;
    let clickTranslateY = undefined;

    updateNumberInfo();

    // update image numbers displayed in info box
    function updateNumberInfo() {
      numberInfoBox.innerHTML =
        sourceImg.dataset.number + " of " + sourceImg.dataset.total;
    }

    // update zoom displayed in info box
    function updateZoomInfo() {
      let zoomInfo = zoom * 100;
      if (!Number.isInteger(zoomInfo)) zoomInfo = zoomInfo.toFixed(2);
      zoomInfoBox.innerHTML = zoomInfo + "%";
    }

    // move to closest zoom step above current zoom
    const zoomIn = function () {
      for (const zoomStep of zoomSteps) {
        if (zoomStep > zoom) {
          zoom = zoomStep;
          break;
        }
      }
      updateTransform();
    };

    // move to closest zoom step above current zoom
    const zoomOut = function () {
      zoomSteps.reverse();
      for (const zoomStep of zoomSteps) {
        if (zoomStep < zoom) {
          zoom = zoomStep;
          break;
        }
      }
      zoomSteps.reverse();

      updateTransform();
    };

    // update display of <img> based on scale/translate properties
    const updateTransform = function () {
      // set transform
      img.style.transform =
        "translate(" +
        (translateX || 0) +
        "px," +
        (translateY || 0) +
        "px) scale(" +
        (zoom || 1) +
        ")";

      // get new width/height after scale
      const rect = img.getBoundingClientRect();
      // limit translate
      translateX = Math.max(translateX, -rect.width / 2);
      translateX = Math.min(translateX, rect.width / 2);
      translateY = Math.max(translateY, -rect.height / 2);
      translateY = Math.min(translateY, rect.height / 2);

      // set transform
      img.style.transform =
        "translate(" +
        (translateX || 0) +
        "px," +
        (translateY || 0) +
        "px) scale(" +
        (zoom || 1) +
        ")";

      updateZoomInfo();
    };

    // fit <img> to container
    const fit = function () {
      // no x/y offset, 100% zoom by default
      translateX = 0;
      translateY = 0;
      zoom = 1;

      // widths of <img> and container
      const imgWidth = img.naturalWidth;
      const imgHeight = img.naturalHeight;
      const containerWidth = parseFloat(
        window.getComputedStyle(container).width
      );
      const containerHeight = parseFloat(
        window.getComputedStyle(container).height
      );

      // how much zooming is needed to fit <img> to container
      const xRatio = imgWidth / containerWidth;
      const yRatio = imgHeight / containerHeight;
      const maxRatio = Math.max(xRatio, yRatio);
      const newZoom = 1 / maxRatio;

      // fit <img> to container according to option
      if (defaultZoom === "shrink") {
        if (maxRatio > 1) zoom = newZoom;
      } else if (defaultZoom === "fit") zoom = newZoom;

      updateTransform();
    };

    // when mouse wheel is rolled anywhere in container
    const onContainerWheel = function (event) {
      if (!event) return;

      // let ctrl + mouse wheel to zoom behave as normal
      if (event.ctrlKey) return;

      // prevent normal scroll behavior
      event.preventDefault();
      event.stopPropagation();

      // point around which to scale img
      const viewRect = container.getBoundingClientRect();
      const viewX = (viewRect.left + viewRect.right) / 2;
      const viewY = (viewRect.top + viewRect.bottom) / 2;
      const originX = centerZoom === "true" ? viewX : mouseX;
      const originY = centerZoom === "true" ? viewY : mouseY;

      // get point on image under origin
      const oldRect = img.getBoundingClientRect();
      const oldPercentX = (originX - oldRect.left) / oldRect.width;
      const oldPercentY = (originY - oldRect.top) / oldRect.height;

      // increment/decrement zoom
      if (event.deltaY < 0) zoomIn();
      if (event.deltaY > 0) zoomOut();

      // get offset between previous image point and origin
      const newRect = img.getBoundingClientRect();
      const offsetX = originX - (newRect.left + newRect.width * oldPercentX);
      const offsetY = originY - (newRect.top + newRect.height * oldPercentY);

      // translate image to keep image point under origin
      translateX += offsetX;
      translateY += offsetY;

      // perform translate
      updateTransform();
    };

    // when container is clicked
    function onContainerClick(event) {
      // if container itself is target of click, and not other
      // element above it
      if (event.target === this) closeLightbox();
    }

    // when mouse button is pressed on image
    const onImageMouseDown = function (event) {
      // store original mouse position relative to image
      clickMouseX = window.mouseX;
      clickMouseY = window.mouseY;
      clickTranslateX = translateX;
      clickTranslateY = translateY;
      event.stopPropagation();
      event.preventDefault();
    };

    // when mouse button is released anywhere in window
    const onWindowMouseUp = function (event) {
      // reset original mouse position
      clickMouseX = undefined;
      clickMouseY = undefined;
      clickTranslateX = undefined;
      clickTranslateY = undefined;

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("mouseup", onWindowMouseUp);
    };

    // when mouse is moved anywhere in window
    const onWindowMouseMove = function (event) {
      if (
        clickMouseX === undefined ||
        clickMouseY === undefined ||
        clickTranslateX === undefined ||
        clickTranslateY === undefined
      )
        return;

      // offset image based on original and current mouse position
      translateX = clickTranslateX + window.mouseX - clickMouseX;
      translateY = clickTranslateY + window.mouseY - clickMouseY;
      updateTransform();
      event.preventDefault();

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("mousemove", onWindowMouseMove);
    };

    // when window is resized
    const onWindowResize = function (event) {
      fit();

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("resize", onWindowResize);
    };

    // attach the necessary event listeners
    img.addEventListener("dblclick", fit);
    img.addEventListener("mousedown", onImageMouseDown);
    container.addEventListener("wheel", onContainerWheel);
    container.addEventListener("mousedown", onContainerClick);
    container.addEventListener("touchstart", onContainerClick);
    window.addEventListener("mouseup", onWindowMouseUp);
    window.addEventListener("mousemove", onWindowMouseMove);
    window.addEventListener("resize", onWindowResize);

    // run fit() after lightbox atttached to document and <img> Loaded
    // so needed container and img dimensions available
    img.addEventListener("load", fit);

    return img;
  }

  // make caption
  function makeCaption(img) {
    const caption = document.createElement("div");
    caption.id = "lightbox_caption";
    const captionSource = img.nextElementSibling;
    if (captionSource.tagName.toLowerCase() === "figcaption") {
      const captionCopy = makeCopy(captionSource);
      caption.innerHTML = captionCopy.innerHTML;
    }

    caption.addEventListener("touchstart", function (event) {
      event.stopPropagation();
    });

    return caption;
  }

  // make carbon copy of html dom element
  function makeCopy(source) {
    const sourceCopy = source.cloneNode(true);

    // delete elements marked with ignore (eg anchor and jump buttons)
    const deleteFromCopy = sourceCopy.querySelectorAll('[data-ignore="true"]');
    for (const element of deleteFromCopy) element.remove();

    // delete certain element attributes
    const attributes = [
      "id",
      "data-collapsed",
      "data-selected",
      "data-highlighted",
      "data-glow",
    ];
    for (const attribute of attributes) {
      sourceCopy.removeAttribute(attribute);
      const elements = sourceCopy.querySelectorAll("[" + attribute + "]");
      for (const element of elements) element.removeAttribute(attribute);
    }

    return sourceCopy;
  }

  // make button to jump to previous image in document
  function makePrevButton(img) {
    const prevButton = document.createElement("button");
    prevButton.id = "lightbox_prev_button";
    prevButton.title = "Jump to the previous image in the document [←]";
    prevButton.classList.add("icon_button", "lightbox_button");
    prevButton.innerHTML = document.querySelector(".icon_caret_left").innerHTML;

    // attach click listeners to button
    prevButton.addEventListener("click", function () {
      getPrevImg(img).click();
    });

    return prevButton;
  }

  // make button to jump to next image in document
  function makeNextButton(img) {
    const nextButton = document.createElement("button");
    nextButton.id = "lightbox_next_button";
    nextButton.title = "Jump to the next image in the document [→]";
    nextButton.classList.add("icon_button", "lightbox_button");
    nextButton.innerHTML = document.querySelector(
      ".icon_caret_right"
    ).innerHTML;

    // attach click listeners to button
    nextButton.addEventListener("click", function () {
      getNextImg(img).click();
    });

    return nextButton;
  }

  // get previous image in document
  function getPrevImg(img) {
    const imgs = document.querySelectorAll(".lightbox_document_img");

    // find index of provided img
    let index;
    for (index = 0; index < imgs.length; index++) {
      if (imgs[index] === img) break;
    }

    // wrap index to other side if < 1
    if (index - 1 >= 0) index--;
    else index = imgs.length - 1;
    return imgs[index];
  }

  // get next image in document
  function getNextImg(img) {
    const imgs = document.querySelectorAll(".lightbox_document_img");

    // find index of provided img
    let index;
    for (index = 0; index < imgs.length; index++) {
      if (imgs[index] === img) break;
    }

    // wrap index to other side if > total
    if (index + 1 <= imgs.length - 1) index++;
    else index = 0;
    return imgs[index];
  }

  // close lightbox
  function closeLightbox() {
    focusBody();

    const lightbox = document.getElementById("lightbox_overlay");
    if (lightbox) lightbox.remove();
  }

  // make all elements behind lightbox non-focusable
  function blurBody(overlay) {
    const all = document.querySelectorAll("*");
    for (const element of all) element.tabIndex = -1;
    document.body.classList.add("body_no_scroll");
  }

  // make all elements focusable again
  function focusBody() {
    const all = document.querySelectorAll("*");
    for (const element of all) element.removeAttribute("tabIndex");
    document.body.classList.remove("body_no_scroll");
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
  <!-- modified from: https://fontawesome.com/icons/caret-left -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
    ></path>
  </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
  <!-- modified from: https://fontawesome.com/icons/caret-right -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* regular <img> in document when hovered */
    img.lightbox_document_img:hover {
      cursor: pointer;
    }

    .body_no_scroll {
      overflow: hidden !important;
    }

    /* screen overlay */
    #lightbox_overlay {
      display: flex;
      flex-direction: column;
      position: fixed;
      left: 0;
      top: 0;
      right: 0;
      bottom: 0;
      background: rgba(0, 0, 0, 0.75);
      z-index: 3;
    }

    /* middle area containing lightbox image */
    #lightbox_image_container {
      flex-grow: 1;
      display: flex;
      justify-content: center;
      align-items: center;
      overflow: hidden;
      position: relative;
      padding: 20px;
    }

    /* bottom area containing caption */
    #lightbox_bottom_container {
      display: flex;
      justify-content: center;
      align-items: center;
      height: 100px;
      min-height: 100px;
      max-height: 100px;
      background: rgba(0, 0, 0, 0.5);
    }

    /* image number info text box */
    #lightbox_number_info {
      position: absolute;
      color: #ffffff;
      font-weight: 600;
      left: 2px;
      top: 0;
      z-index: 4;
    }

    /* zoom info text box */
    #lightbox_zoom_info {
      position: absolute;
      color: #ffffff;
      font-weight: 600;
      right: 2px;
      top: 0;
      z-index: 4;
    }

    /* copy of image caption */
    #lightbox_caption {
      box-sizing: border-box;
      display: inline-block;
      width: 100%;
      max-height: 100%;
      padding: 10px 0;
      text-align: center;
      overflow-y: auto;
      color: #ffffff;
    }

    /* navigation previous/next button */
    .lightbox_button {
      width: 100px;
      height: 100%;
      min-width: 100px;
      min-height: 100%;
      color: #ffffff;
    }

    /* navigation previous/next button when hovered */
    .lightbox_button:hover {
      background: none !important;
    }

    /* navigation button icon */
    .lightbox_button > svg {
      height: 25px;
    }

    /* figure auto-number */
    #lightbox_caption > span:first-of-type {
      font-weight: bold;
      margin-right: 5px;
    }

    /* lightbox image when hovered */
    #lightbox_img:hover {
      cursor: grab;
    }

    /* lightbox image when grabbed */
    #lightbox_img:active {
      cursor: grabbing;
    }
  }

  /* when on screen < 480px wide */
  @media only screen and (max-width: 480px) {
    /* make navigation buttons skinnier on small screens to make more room for caption text */
    .lightbox_button {
      width: 50px;
      min-width: 50px;
    }
  }

  /* always hide lightbox on print */
  @media only print {
    #lightbox_overlay {
      display: none;
    }
  }
</style>
<!-- 
  Link Highlight Plugin

  Makes it such that when a user hovers or focuses a link, other links that have
  the same target will be highlighted. It also makes it such that when clicking
  a link, the target of the link (eg reference, figure, table) is briefly
  highlighted.
-->

<script type="module">
  // whether to also highlight links that go to external urls
  const externalLinks = "false";
  // whether user must click off to unhighlight instead of just
  // un-hovering
  const clickUnhighlight = "false";
  // whether to also highlight links that are unique
  const highlightUnique = "true";

  // start script
  function start() {
    const links = getLinks();
    for (const link of links) {
      // attach mouse and focus listeners to link
      link.addEventListener("mouseenter", onLinkFocus);
      link.addEventListener("focus", onLinkFocus);
      link.addEventListener("mouseleave", onLinkUnhover);
    }

    // attach click and hash change listeners to window
    window.addEventListener("click", onClick);
    window.addEventListener("touchstart", onClick);
    window.addEventListener("hashchange", onHashChange);

    // run hash change on window load in case user has navigated
    // directly to hash
    onHashChange();
  }

  // when link is focused (tabbed to) or hovered
  function onLinkFocus() {
    highlight(this);
  }

  // when link is unhovered
  function onLinkUnhover() {
    if (clickUnhighlight !== "true") unhighlightAll();
  }

  // when the mouse is clicked anywhere in window
  function onClick(event) {
    unhighlightAll();
  }

  // when hash (eg manuscript.html#introduction) changes
  function onHashChange() {
    const target = getHashTarget();
    if (target) glowElement(target);
  }

  // start glow sequence on an element
  function glowElement(element) {
    const startGlow = function () {
      onGlowEnd();
      element.dataset.glow = "true";
      element.addEventListener("animationend", onGlowEnd);
    };
    const onGlowEnd = function () {
      element.removeAttribute("data-glow");
      element.removeEventListener("animationend", onGlowEnd);
    };
    startGlow();
  }

  // highlight link and all others with same target
  function highlight(link) {
    // force unhighlight all to start fresh
    unhighlightAll();

    // get links with same target
    if (!link) return;
    const sameLinks = getSameLinks(link);

    // if link unique and option is off, exit and don't highlight
    if (sameLinks.length <= 1 && highlightUnique !== "true") return;

    // highlight all same links, and "select" (special highlight) this
    // one
    for (const sameLink of sameLinks) {
      if (sameLink === link) sameLink.setAttribute("data-selected", "true");
      else sameLink.setAttribute("data-highlighted", "true");
    }
  }

  // unhighlight all links
  function unhighlightAll() {
    const links = getLinks();
    for (const link of links) {
      link.setAttribute("data-selected", "false");
      link.setAttribute("data-highlighted", "false");
    }
  }

  // get links with same target
  function getSameLinks(link) {
    const results = [];
    const links = getLinks();
    for (const otherLink of links) {
      if (otherLink.getAttribute("href") === link.getAttribute("href"))
        results.push(otherLink);
    }
    return results;
  }

  // get all links of types we wish to handle
  function getLinks() {
    let query = "a";
    if (externalLinks !== "true") query += '[href^="#"]';
    // exclude buttons, anchor links, toc links, etc
    query += ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    return document.querySelectorAll(query);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<style>
  @media only screen {
    /* anything with data-highlighted attribute true */
    [data-highlighted="true"] {
      background: #ffeb3b;
    }

    /* anything with data-selected attribute true */
    [data-selected="true"] {
      background: #ff8a65 !important;
    }

    /* animation definition for glow */
    @keyframes highlight_glow {
      0% {
        background: none;
      }
      10% {
        background: #bbdefb;
      }
      100% {
        background: none;
      }
    }

    /* anything with data-glow attribute true */
    [data-glow="true"] {
      animation: highlight_glow 2s;
    }
  }
</style>
<!--
  Table of Contents Plugin

  Provides a "table of contents" (toc) panel on the side of the document that
  allows the user to conveniently navigate between sections of the document.
-->

<script type="module">
  // which types of elements to add links for, in "document.querySelector" format
  const typesQuery = "h1, h2, h3";
  // whether toc starts open. use 'true' or 'false', or 'auto' to
  // use 'true' behavior when screen wide enough and 'false' when not
  const startOpen = "false";
  // whether toc closes when clicking on toc link. use 'true' or
  // 'false', or 'auto' to use 'false' behavior when screen wide
  // enough and 'true' when not
  const clickClose = "auto";
  // if list item is more than this many characters, text will be
  // truncated
  const charLimit = "50";
  // whether or not to show bullets next to each toc item
  const bullets = "false";

  // start script
  function start() {
    // make toc panel and populate with entries (links to document
    // sections)
    const panel = makePanel();
    if (!panel) return;
    makeEntries(panel);
    // attach panel to document after making entries, so 'toc' heading
    // in panel isn't included in toc
    document.body.insertBefore(panel, document.body.firstChild);

    // initial panel state
    if (startOpen === "true" || (startOpen === "auto" && !isSmallScreen()))
      openPanel();
    else closePanel();

    // attach click, scroll, and hash change listeners to window
    window.addEventListener("click", onClick);
    window.addEventListener("scroll", onScroll);
    window.addEventListener("hashchange", onScroll);
    window.addEventListener("keyup", onKeyUp);
    onScroll();

    // add class to push document body down out of way of toc button
    document.body.classList.add("toc_body_nudge");
  }

  // determine if screen wide enough to fit toc panel
  function isSmallScreen() {
    // in default theme:
    // 816px = 8.5in = width of "page" (<body>) element
    // 260px = min width of toc panel (*2 for both sides of <body>)
    return window.innerWidth < 816 + 260 * 2;
  }

  // when mouse is clicked anywhere in window
  function onClick() {
    if (isSmallScreen()) closePanel();
  }

  // when window is scrolled or hash changed
  function onScroll() {
    highlightViewed();
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    // close on esc
    if (event.key === "Escape") closePanel();
  }

  // find entry of currently viewed document section in toc and highlight
  function highlightViewed() {
    const firstId = getFirstInView(typesQuery);

    // get toc entries (links), unhighlight all, then highlight viewed
    const list = document.getElementById("toc_list");
    if (!firstId || !list) return;
    const links = list.querySelectorAll("a");
    for (const link of links) link.dataset.viewing = "false";
    const link = list.querySelector('a[href="#' + firstId + '"]');
    if (!link) return;
    link.dataset.viewing = "true";
  }

  // get first or previous toc listed element in top half of view
  function getFirstInView(query) {
    // get all elements matching query and with id
    const elements = document.querySelectorAll(query);
    const elementsWithIds = [];
    for (const element of elements) {
      if (element.id) elementsWithIds.push(element);
    }

    // get first or previous element in top half of view
    for (let i = 0; i < elementsWithIds.length; i++) {
      const element = elementsWithIds[i];
      const prevElement = elementsWithIds[Math.max(0, i - 1)];
      if (element.getBoundingClientRect().top >= 0) {
        if (element.getBoundingClientRect().top < window.innerHeight / 2)
          return element.id;
        else return prevElement.id;
      }
    }
  }

  // make panel
  function makePanel() {
    // create panel
    const panel = document.createElement("div");
    panel.id = "toc_panel";
    if (bullets === "true") panel.dataset.bullets = "true";

    // create header
    const header = document.createElement("div");
    header.id = "toc_header";

    // create toc button
    const button = document.createElement("button");
    button.id = "toc_button";
    button.innerHTML = document.querySelector(".icon_th_list").innerHTML;
    button.title = "Table of Contents";
    button.classList.add("icon_button");

    // create header text
    const text = document.createElement("h4");
    text.innerHTML = "Table of Contents";

    // create container for toc list
    const list = document.createElement("div");
    list.id = "toc_list";

    // attach click listeners
    panel.addEventListener("click", onPanelClick);
    header.addEventListener("click", onHeaderClick);
    button.addEventListener("click", onButtonClick);

    // attach elements
    header.appendChild(button);
    header.appendChild(text);
    panel.appendChild(header);
    panel.appendChild(list);

    return panel;
  }

  // create toc entries (links) to each element of the specified types
  function makeEntries(panel) {
    const elements = document.querySelectorAll(typesQuery);
    for (const element of elements) {
      // do not add link if element doesn't have assigned id
      if (!element.id) continue;

      // create link/list item
      const link = document.createElement("a");
      link.classList.add("toc_link");
      switch (element.tagName.toLowerCase()) {
        case "h1":
          link.dataset.level = "1";
          break;
        case "h2":
          link.dataset.level = "2";
          break;
        case "h3":
          link.dataset.level = "3";
          break;
        case "h4":
          link.dataset.level = "4";
          break;
      }
      link.title = element.innerText;
      let text = element.innerText;
      if (text.length > charLimit) text = text.slice(0, charLimit) + "...";
      link.innerHTML = text;
      link.href = "#" + element.id;
      link.addEventListener("click", onLinkClick);

      // attach link
      panel.querySelector("#toc_list").appendChild(link);
    }
  }

  // when panel is clicked
  function onPanelClick(event) {
    // stop click from propagating to window/document and closing panel
    event.stopPropagation();
  }

  // when header itself is clicked
  function onHeaderClick(event) {
    togglePanel();
  }

  // when button is clicked
  function onButtonClick(event) {
    togglePanel();
    // stop header underneath button from also being clicked
    event.stopPropagation();
  }

  // when link is clicked
  function onLinkClick(event) {
    if (clickClose === "true" || (clickClose === "auto" && isSmallScreen()))
      closePanel();
    else openPanel();
  }

  // open panel if closed, close if opened
  function togglePanel() {
    const panel = document.getElementById("toc_panel");
    if (!panel) return;

    if (panel.dataset.open === "true") closePanel();
    else openPanel();
  }

  // open panel
  function openPanel() {
    const panel = document.getElementById("toc_panel");
    if (panel) panel.dataset.open = "true";
  }

  // close panel
  function closePanel() {
    const panel = document.getElementById("toc_panel");
    if (panel) panel.dataset.open = "false";
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- th list icon -->

<template class="icon_th_list">
  <!-- modified from: https://fontawesome.com/icons/th-list -->
  <svg width="16" height="16" viewBox="0 0 512 512" tabindex="-1">
    <path
      fill="currentColor"
      d="M96 96c0 26.51-21.49 48-48 48S0 122.51 0 96s21.49-48 48-48 48 21.49 48 48zM48 208c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm0 160c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm96-236h352c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
      tabindex="-1"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* toc panel */
    #toc_panel {
      box-sizing: border-box;
      position: fixed;
      top: 0;
      left: 0;
      background: #ffffff;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      z-index: 2;
    }

    /* toc panel when closed */
    #toc_panel[data-open="false"] {
      min-width: 60px;
      width: 60px;
      height: 60px;
      border-right: solid 1px #bdbdbd;
      border-bottom: solid 1px #bdbdbd;
    }

    /* toc panel when open */
    #toc_panel[data-open="true"] {
      min-width: 260px;
      max-width: 480px;
      /* keep panel edge consistent distance away from "page" edge */
      width: calc(((100vw - 8.5in) / 2) - 30px - 40px);
      bottom: 0;
      border-right: solid 1px #bdbdbd;
    }

    /* toc panel header */
    #toc_header {
      box-sizing: border-box;
      display: flex;
      flex-direction: row;
      align-items: center;
      height: 60px;
      margin: 0;
      padding: 20px;
    }

    /* toc panel header when hovered */
    #toc_header:hover {
      cursor: pointer;
    }

    /* toc panel header when panel open */
    #toc_panel[data-open="true"] > #toc_header {
      border-bottom: solid 1px #bdbdbd;
    }

    /* toc open/close header button */
    #toc_button {
      margin-right: 20px;
    }

    /* hide toc list and header text when closed */
    #toc_panel[data-open="false"] > #toc_header > *:not(#toc_button),
    #toc_panel[data-open="false"] > #toc_list {
      display: none;
    }

    /* toc list of entries */
    #toc_list {
      box-sizing: border-box;
      width: 100%;
      padding: 20px;
      position: absolute;
      top: calc(60px + 1px);
      bottom: 0;
      overflow: auto;
    }

    /* toc entry, link to section in document */
    .toc_link {
      display: block;
      padding: 5px;
      position: relative;
      font-weight: 600;
      text-decoration: none;
    }

    /* toc entry when hovered or when "viewed" */
    .toc_link:hover,
    .toc_link[data-viewing="true"] {
      background: #f5f5f5;
    }

    /* toc entry, level 1 indentation */
    .toc_link[data-level="1"] {
      margin-left: 0;
    }

    /* toc entry, level 2 indentation */
    .toc_link[data-level="2"] {
      margin-left: 20px;
    }

    /* toc entry, level 3 indentation */
    .toc_link[data-level="3"] {
      margin-left: 40px;
    }

    /* toc entry, level 4 indentation */
    .toc_link[data-level="4"] {
      margin-left: 60px;
    }

    /* toc entry bullets */
    #toc_panel[data-bullets="true"] .toc_link[data-level]:before {
      position: absolute;
      left: -15px;
      top: -1px;
      font-size: 1.5em;
    }

    /* toc entry, level 2 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="2"]:before {
      content: "\2022";
    }

    /* toc entry, level 3 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="3"]:before {
      content: "\25AB";
    }

    /* toc entry, level 4 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="4"]:before {
      content: "-";
    }
  }

  /* when on screen < 8.5in wide */
  @media only screen and (max-width: 8.5in) {
    /* push <body> ("page") element down to make room for toc icon */
    .toc_body_nudge {
      padding-top: 60px;
    }

    /* toc icon when panel closed and not hovered */
    #toc_panel[data-open="false"]:not(:hover) {
      background: rgba(255, 255, 255, 0.75);
    }
  }

  /* always hide toc panel on print */
  @media only print {
    #toc_panel {
      display: none;
    }
  }
</style>
<!-- 
  Tooltips Plugin

  Makes it such that when the user hovers or focuses a link to a citation or
  figure, a tooltip appears with a preview of the reference content, along with
  arrows to navigate between instances of the same reference in the document.
-->

<script type="module">
  // whether user must click off to close tooltip instead of just un-hovering
  const clickClose = "false";
  // delay (in ms) between opening and closing tooltip
  const delay = "100";

  // start script
  function start() {
    const links = getLinks();
    for (const link of links) {
      // attach hover and focus listeners to link
      link.addEventListener("mouseover", onLinkHover);
      link.addEventListener("mouseleave", onLinkUnhover);
      link.addEventListener("focus", onLinkFocus);
      link.addEventListener("touchend", onLinkTouch);
    }

    // attach mouse, key, and resize listeners to window
    window.addEventListener("mousedown", onClick);
    window.addEventListener("touchstart", onClick);
    window.addEventListener("keyup", onKeyUp);
    window.addEventListener("resize", onResize);
  }

  // when link is hovered
  function onLinkHover() {
    // function to open tooltip
    const delayOpenTooltip = function () {
      openTooltip(this);
    }.bind(this);

    // run open function after delay
    this.openTooltipTimer = window.setTimeout(delayOpenTooltip, delay);
  }

  // when mouse leaves link
  function onLinkUnhover() {
    // cancel opening tooltip
    window.clearTimeout(this.openTooltipTimer);

    // don't close on unhover if option specifies
    if (clickClose === "true") return;

    // function to close tooltip
    const delayCloseTooltip = function () {
      // if tooltip open and if mouse isn't over tooltip, close
      const tooltip = document.getElementById("tooltip");
      if (tooltip && !tooltip.matches(":hover")) closeTooltip();
    };

    // run close function after delay
    this.closeTooltipTimer = window.setTimeout(delayCloseTooltip, delay);
  }

  // when link is focused (tabbed to)
  function onLinkFocus(event) {
    openTooltip(this);
  }

  // when link is touched on touch screen
  function onLinkTouch(event) {
    // attempt to force hover state on first tap always, and trigger
    // regular link click (and navigation) on second tap
    if (event.target === document.activeElement) event.target.click();
    else {
      document.activeElement.blur();
      event.target.focus();
    }
    if (event.cancelable) event.preventDefault();
    event.stopPropagation();
    return false;
  }

  // when mouse is clicked anywhere in window
  function onClick(event) {
    closeTooltip();
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    switch (event.key) {
      // trigger click of prev button
      case "ArrowLeft":
        const prevButton = document.getElementById("tooltip_prev_button");
        if (prevButton) prevButton.click();
        break;
      // trigger click of next button
      case "ArrowRight":
        const nextButton = document.getElementById("tooltip_next_button");
        if (nextButton) nextButton.click();
        break;
      // close on esc
      case "Escape":
        closeTooltip();
        break;
    }
  }

  // when window is resized or zoomed
  function onResize() {
    closeTooltip();
  }

  // get all links of types we wish to handle
  function getLinks() {
    const queries = [];
    // exclude buttons, anchor links, toc links, etc
    const exclude =
      ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    queries.push('a[href^="#ref-"]' + exclude); // citation links
    queries.push('a[href^="#fig:"]' + exclude); // figure links
    const query = queries.join(", ");
    return document.querySelectorAll(query);
  }

  // get links with same target, get index of link in set, get total
  // same links
  function getSameLinks(link) {
    const sameLinks = [];
    const links = getLinks();
    for (const otherLink of links) {
      if (otherLink.getAttribute("href") === link.getAttribute("href"))
        sameLinks.push(otherLink);
    }

    return {
      elements: sameLinks,
      index: sameLinks.indexOf(link),
      total: sameLinks.length,
    };
  }

  // open tooltip
  function openTooltip(link) {
    // delete tooltip if it exists, start fresh
    closeTooltip();

    // make tooltip element
    const tooltip = makeTooltip(link);

    // if source couldn't be found and tooltip not made, exit
    if (!tooltip) return;

    // make navbar elements
    const navBar = makeNavBar(link);
    if (navBar) tooltip.firstElementChild.appendChild(navBar);

    // attach tooltip to page
    document.body.appendChild(tooltip);

    // position tooltip
    const position = function () {
      positionTooltip(link);
    };
    position();

    // if tooltip contains images, position again after they've loaded
    const imgs = tooltip.querySelectorAll("img");
    for (const img of imgs) img.addEventListener("load", position);
  }

  // close (delete) tooltip
  function closeTooltip() {
    const tooltip = document.getElementById("tooltip");
    if (tooltip) tooltip.remove();
  }

  // make tooltip
  function makeTooltip(link) {
    // get target element that link points to
    const source = getSource(link);

    // if source can't be found, exit
    if (!source) return;

    // create new tooltip
    const tooltip = document.createElement("div");
    tooltip.id = "tooltip";
    const tooltipContent = document.createElement("div");
    tooltipContent.id = "tooltip_content";
    tooltip.appendChild(tooltipContent);

    // make copy of source node and put in tooltip
    const sourceCopy = makeCopy(source);
    tooltipContent.appendChild(sourceCopy);

    // attach mouse event listeners
    tooltip.addEventListener("click", onTooltipClick);
    tooltip.addEventListener("mousedown", onTooltipClick);
    tooltip.addEventListener("touchstart", onTooltipClick);
    tooltip.addEventListener("mouseleave", onTooltipUnhover);

    // (for interaction with lightbox plugin)
    // transfer click on tooltip copied img to original img
    const sourceImg = source.querySelector("img");
    const sourceCopyImg = sourceCopy.querySelector("img");
    if (sourceImg && sourceCopyImg) {
      const clickImg = function () {
        sourceImg.click();
        closeTooltip();
      };
      sourceCopyImg.addEventListener("click", clickImg);
    }

    return tooltip;
  }

  // make carbon copy of html dom element
  function makeCopy(source) {
    const sourceCopy = source.cloneNode(true);

    // delete elements marked with ignore (eg anchor and jump buttons)
    const deleteFromCopy = sourceCopy.querySelectorAll('[data-ignore="true"]');
    for (const element of deleteFromCopy) element.remove();

    // delete certain element attributes
    const attributes = [
      "id",
      "data-collapsed",
      "data-selected",
      "data-highlighted",
      "data-glow",
    ];
    for (const attribute of attributes) {
      sourceCopy.removeAttribute(attribute);
      const elements = sourceCopy.querySelectorAll("[" + attribute + "]");
      for (const element of elements) element.removeAttribute(attribute);
    }

    return sourceCopy;
  }

  // when tooltip is clicked
  function onTooltipClick(event) {
    // when user clicks on tooltip, stop click from transferring
    // outside of tooltip (eg, click off to close tooltip, or eg click
    // off to unhighlight same refs)
    event.stopPropagation();
  }

  // when tooltip is unhovered
  function onTooltipUnhover(event) {
    if (clickClose === "true") return;

    // make sure new mouse/touch/focus no longer over tooltip or any
    // element within it
    const tooltip = document.getElementById("tooltip");
    if (!tooltip) return;
    if (this.contains(event.relatedTarget)) return;

    closeTooltip();
  }

  // make nav bar to go betwen prev/next instances of same reference
  function makeNavBar(link) {
    // find other links to the same source
    const sameLinks = getSameLinks(link);

    // don't show nav bar when singular reference
    if (sameLinks.total <= 1) return;

    // find prev/next links with same target
    const prevLink = getPrevLink(link, sameLinks);
    const nextLink = getNextLink(link, sameLinks);

    // create nav bar
    const navBar = document.createElement("div");
    navBar.id = "tooltip_nav_bar";
    const text = sameLinks.index + 1 + " of " + sameLinks.total;

    // create nav bar prev/next buttons
    const prevButton = document.createElement("button");
    const nextButton = document.createElement("button");
    prevButton.id = "tooltip_prev_button";
    nextButton.id = "tooltip_next_button";
    prevButton.title =
      "Jump to the previous occurence of this item in the document [←]";
    nextButton.title =
      "Jump to the next occurence of this item in the document [→]";
    prevButton.classList.add("icon_button");
    nextButton.classList.add("icon_button");
    prevButton.innerHTML = document.querySelector(".icon_caret_left").innerHTML;
    nextButton.innerHTML = document.querySelector(
      ".icon_caret_right"
    ).innerHTML;
    navBar.appendChild(prevButton);
    navBar.appendChild(document.createTextNode(text));
    navBar.appendChild(nextButton);

    // attach click listeners to buttons
    prevButton.addEventListener("click", function () {
      onPrevNextClick(link, prevLink);
    });
    nextButton.addEventListener("click", function () {
      onPrevNextClick(link, nextLink);
    });

    return navBar;
  }

  // get previous link with same target
  function getPrevLink(link, sameLinks) {
    if (!sameLinks) sameLinks = getSameLinks(link);
    // wrap index to other side if < 1
    let index;
    if (sameLinks.index - 1 >= 0) index = sameLinks.index - 1;
    else index = sameLinks.total - 1;
    return sameLinks.elements[index];
  }

  // get next link with same target
  function getNextLink(link, sameLinks) {
    if (!sameLinks) sameLinks = getSameLinks(link);
    // wrap index to other side if > total
    let index;
    if (sameLinks.index + 1 <= sameLinks.total - 1) index = sameLinks.index + 1;
    else index = 0;
    return sameLinks.elements[index];
  }

  // get element that is target of link or url hash
  function getSource(link) {
    const hash = link ? link.hash : window.location.hash;
    const id = hash.slice(1);
    let target = document.querySelector('[id="' + id + '"]');
    if (!target) return;

    // if ref or figure, modify target to get expected element
    if (id.indexOf("ref-") === 0) target = target.querySelector("p");
    else if (id.indexOf("fig:") === 0) target = target.querySelector("figure");

    return target;
  }

  // when prev/next arrow button is clicked
  function onPrevNextClick(link, prevNextLink) {
    if (link && prevNextLink)
      goToElement(prevNextLink, window.innerHeight * 0.5);
  }

  // scroll to and focus element
  function goToElement(element, offset) {
    // expand accordion section if collapsed
    expandElement(element);
    const y =
      getRectInView(element).top -
      getRectInView(document.documentElement).top -
      (offset || 0);
    // trigger any function listening for "onscroll" event
    window.dispatchEvent(new Event("scroll"));
    window.scrollTo(0, y);
    document.activeElement.blur();
    element.focus();
  }

  // determine position to place tooltip based on link position in
  // viewport and tooltip size
  function positionTooltip(link, left, top) {
    const tooltipElement = document.getElementById("tooltip");
    if (!tooltipElement) return;

    // get convenient vars for position/dimensions of
    // link/tooltip/page/view
    link = getRectInPage(link);
    const tooltip = getRectInPage(tooltipElement);
    const view = getRectInPage();

    // horizontal positioning
    if (left)
      // use explicit value
      left = left;
    else if (link.left + tooltip.width < view.right)
      // fit tooltip to right of link
      left = link.left;
    else if (link.right - tooltip.width > view.left)
      // fit tooltip to left of link
      left = link.right - tooltip.width;
    // center tooltip in view
    else left = (view.right - view.left) / 2 - tooltip.width / 2;

    // vertical positioning
    if (top)
      // use explicit value
      top = top;
    else if (link.top - tooltip.height > view.top)
      // fit tooltip above link
      top = link.top - tooltip.height;
    else if (link.bottom + tooltip.height < view.bottom)
      // fit tooltip below link
      top = link.bottom;
    else {
      // center tooltip in view
      top = view.top + view.height / 2 - tooltip.height / 2;
      // nudge off of link to left/right if possible
      if (link.right + tooltip.width < view.right) left = link.right;
      else if (link.left - tooltip.width > view.left)
        left = link.left - tooltip.width;
    }

    tooltipElement.style.left = left + "px";
    tooltipElement.style.top = top + "px";
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
  <!-- modified from: https://fontawesome.com/icons/caret-left -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
    ></path>
  </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
  <!-- modified from: https://fontawesome.com/icons/caret-right -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* tooltip container */
    #tooltip {
      position: absolute;
      width: 50%;
      min-width: 240px;
      max-width: 75%;
      z-index: 1;
    }

    /* tooltip content */
    #tooltip_content {
      margin-bottom: 5px;
      padding: 20px;
      border-radius: 5px;
      border: solid 1px #bdbdbd;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      background: #ffffff;
      overflow-wrap: break-word;
    }

    /* tooltip copy of paragraphs and figures */
    #tooltip_content > p,
    #tooltip_content > figure {
      margin: 0;
      max-height: 320px;
      overflow-y: auto;
    }

    /* tooltip copy of <img> */
    #tooltip_content > figure > img,
    #tooltip_content > figure > svg {
      max-height: 260px;
    }

    /* navigation bar */
    #tooltip_nav_bar {
      margin-top: 10px;
      text-align: center;
    }

    /* navigation bar previous/next buton */
    #tooltip_nav_bar > .icon_button {
      position: relative;
      top: 3px;
    }

    /* navigation bar previous button */
    #tooltip_nav_bar > .icon_button:first-of-type {
      margin-right: 5px;
    }

    /* navigation bar next button */
    #tooltip_nav_bar > .icon_button:last-of-type {
      margin-left: 5px;
    }
  }

  /* always hide tooltip on print */
  @media only print {
    #tooltip {
      display: none;
    }
  }
</style>
<!--
  Analytics Plugin (third-party) 
  
  Copy and paste code from Google Analytics or similar service here.
-->
<!-- 
  Annotations Plugin

  Allows public annotation of the  manuscript. See https://web.hypothes.is/.
-->

<script>
  // configuration
  window.hypothesisConfig = function () {
    return {
      branding: {
        accentColor: "#2196f3",
        appBackgroundColor: "#f8f8f8",
        ctaBackgroundColor: "#f8f8f8",
        ctaTextColor: "#000000",
        selectionFontFamily: "Open Sans, Helvetica, sans serif",
        annotationFontFamily: "Open Sans, Helvetica, sans serif",
      },
    };
  };

  // hypothesis client script
  const embed = "https://hypothes.is/embed.js";
  // hypothesis annotation count query url
  const query = "https://api.hypothes.is/api/search?limit=0&url=";

  // start script
  function start() {
    const button = makeButton();
    document.body.insertBefore(button, document.body.firstChild);
    insertCount(button);
  }

  // make button
  function makeButton() {
    // create button
    const button = document.createElement("button");
    button.id = "hypothesis_button";
    button.innerHTML = document.querySelector(".icon_hypothesis").innerHTML;
    button.title = "Hypothesis annotations";
    button.classList.add("icon_button");

    function onClick(event) {
      onButtonClick(event, button);
    }

    // attach click listeners
    button.addEventListener("click", onClick);

    return button;
  }

  // insert annotations count
  async function insertCount(button) {
    // get annotation count from Hypothesis based on url
    let count = "-";
    try {
      const canonical = document.querySelector('link[rel="canonical"]');
      const location = window.location;
      const url = encodeURIComponent((canonical || location).href);
      const response = await fetch(query + url);
      const json = await response.json();
      count = json.total || "-";
    } catch (error) {
      console.log(error);
    }

    // put count into button
    const counter = document.createElement("span");
    counter.id = "hypothesis_count";
    counter.innerHTML = count;
    button.title = "View " + count + " Hypothesis annotations";
    button.append(counter);
  }

  // when button is clicked
  function onButtonClick(event, button) {
    const script = document.createElement("script");
    script.src = embed;
    document.body.append(script);
    button.remove();
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- hypothesis icon -->

<template class="icon_hypothesis">
  <!-- modified from: https://simpleicons.org/icons/hypothesis.svg / https://git.io/Jf1VB -->
  <svg width="16" height="16" viewBox="0 0 24 24" tabindex="-1">
    <path
      fill="currentColor"
      d="M3.43 0C2.5 0 1.72 .768 1.72 1.72V18.86C1.72 19.8 2.5 20.57 3.43 20.57H9.38L12 24L14.62 20.57H20.57C21.5 20.57 22.29 19.8 22.29 18.86V1.72C22.29 .77 21.5 0 20.57 0H3.43M5.14 3.43H7.72V9.43S8.58 7.72 10.28 7.72C12 7.72 13.74 8.57 13.74 11.24V17.14H11.16V12C11.16 10.61 10.28 10.07 9.43 10.29C8.57 10.5 7.72 11.41 7.72 13.29V17.14H5.14V3.43M18 13.72C18.95 13.72 19.72 14.5 19.72 15.42A1.71 1.71 0 0 1 18 17.13A1.71 1.71 0 0 1 16.29 15.42C16.29 14.5 17.05 13.71 18 13.71Z"
      tabindex="-1"
    ></path>
  </svg>
</template>

<style>
  /* hypothesis activation button */
  #hypothesis_button {
    box-sizing: border-box;
    position: fixed;
    top: 0;
    right: 0;
    width: 60px;
    height: 60px;
    background: #ffffff;
    border-radius: 0;
    border-left: solid 1px #bdbdbd;
    border-bottom: solid 1px #bdbdbd;
    box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
    z-index: 2;
  }

  /* hypothesis button svg */
  #hypothesis_button > svg {
    position: relative;
    top: -4px;
  }

  /* hypothesis annotation count */
  #hypothesis_count {
    position: absolute;
    left: 0;
    right: 0;
    bottom: 5px;
  }

  /* side panel */
  .annotator-frame {
    width: 280px !important;
  }

  /* match highlight color to rest of theme */
  .annotator-highlights-always-on .annotator-hl {
    background-color: #ffeb3b !important;
  }

  /* match focused color to rest of theme */
  .annotator-hl.annotator-hl-focused {
    background-color: #ff8a65 !important;
  }

  /* match bucket bar color to rest of theme */
  .annotator-bucket-bar {
    background: #f5f5f5 !important;
  }

  /* always hide button, toolbar, and tooltip on print */
  @media only print {
    #hypothesis_button {
      display: none;
    }

    .annotator-frame {
      display: none !important;
    }

    hypothesis-adder {
      display: none !important;
    }
  }
</style>
<!-- 
  Mathjax Plugin (third-party) 

  Allows the proper rendering of math/equations written in LaTeX.
  See https://www.mathjax.org/.
-->

<script type="text/x-mathjax-config">
  // configuration
  MathJax.Hub.Config({
    "CommonHTML": { linebreaks: { automatic: true } },
    "HTML-CSS": { linebreaks: { automatic: true } },
    "SVG": { linebreaks: { automatic: true } },
    "fast-preview": { disabled: true }
  });
</script>

<script
  src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"
  integrity="sha512-0doc9hKxR3PYwso42RD1p5ySZpzzuDiOwMrdCEh2WdJZCjcmFKc/wEnL+z8fBQrnHoiNWbo+3fiGkOYXBdQp4A=="
  crossorigin="anonymous"
></script>

<style>
  /* mathjax containers */
  .math.display > span:not(.MathJax_Preview) {
    /* turn inline element (no dimensions) into block (allows fixed width and thus scrolling) */
    display: flex !important;
    overflow-x: auto !important;
    overflow-y: hidden !important;
    justify-content: center;
    align-items: center;
    margin: 0 !important;
  }

  /* right click menu */
  .MathJax_Menu {
    border-radius: 5px !important;
    border: solid 1px #bdbdbd !important;
    box-shadow: none !important;
  }

  /* equation auto-number */
  span[id^="eq:"] > span.math.display + span {
    font-weight: 600;
  }

  /* equation */
  span[id^="eq:"] > span.math.display > span {
    /* nudge to make room for equation auto-number and anchor */
    margin-right: 60px !important;
  }
</style>
</body>
</html>
